kv/kvnemesis: TestKVNemesisSingleNode failed

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7596517?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7596517?buildTab=artifacts#/) on master @ [8357abb668a5adaff781343b394b162fb1b66c6e](https://github.com/cockroachdb/cockroach/commits/8357abb668a5adaff781343b394b162fb1b66c6e):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode3963958602
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [d]/Table/100/"08056413":1668847865.823418091,1-><nil> [s]/Table/100/"{102d0e5c"-66cda759"}:{0:[0,0, 0,0), gap:[<min>, 1668847865.588834720,1),[1668847865.793293449,0, <max>),[1668847865.680153892,1, 1668847865.792215515,0),[1668847865.603420672,1, 1668847865.680153892,1)}->[/Table/100/"2ab8e983":v-7] [w]/Table/100/"2ab8e983":1668847865.823418091,1->v-7 [rs]/Table/100/"{2ba47ac0"-c7658174"}:{0:[1668847865.823418091,0, <max>), gap:[<min>, 1668847865.588834720,0),[1668847865.588834720,0, 1668847865.588834720,1),[1668847865.795713494,0, <max>),[1668847865.793293449,0, 1668847865.795713494,0),[1668847865.680153892,1, 1668847865.743831142,0),[1668847865.603420672,1, 1668847865.680153892,1)}->[/Table/100/"9c5374b1":v-8]
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [d]/Table/100/"08056413":1668847865.823418091,1-><nil> [s]/Table/100/"{102d0e5c"-66cda759"}:{0:[0,0, 0,0), gap:[<min>, 1668847865.588834720,1),[1668847865.793293449,0, <max>),[1668847865.680153892,1, 1668847865.792215515,0),[1668847865.603420672,1, 1668847865.680153892,1)}->[/Table/100/"2ab8e983":v-7] [w]/Table/100/"2ab8e983":1668847865.823418091,1->v-7 [rs]/Table/100/"{2ba47ac0"-c7658174"}:{0:[1668847865.823418091,0, <max>), gap:[<min>, 1668847865.588834720,0),[1668847865.588834720,0, 1668847865.588834720,1),[1668847865.795713494,0, <max>),[1668847865.793293449,0, 1668847865.795713494,0),[1668847865.680153892,1, 1668847865.743831142,0),[1668847865.603420672,1, 1668847865.680153892,1)}->[/Table/100/"9c5374b1":v-8]
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode3963958602
--- FAIL: TestKVNemesisSingleNode (1.68s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21628

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7620188?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7620188?buildTab=artifacts#/) on master @ [f0554bc215e31ff53e644936ba645bd895ec0d15](https://github.com/cockroachdb/cockroach/commits/f0554bc215e31ff53e644936ba645bd895ec0d15):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2872175079
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [r]/Table/100/"84ed9808":[<min>, 1669020675.344435012,0),[1669020675.344435012,0, 1669020675.380563306,0)-><nil> [w]/Table/100/"25be7481":1669020675.378291802,1->v-9 [rs]/Table/100/"{76f506a5"-d4d44a56"}:{gap:[<min>, 1669020675.344435012,0),[1669020675.344435012,0, 1669020675.348339423,0)}->[] [d]/Table/100/"b135a3dd":1669020675.378291802,1-><nil> [d]/Table/100/"4506fec4":1669020675.378291802,1-><nil>
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [r]/Table/100/"84ed9808":[<min>, 1669020675.344435012,0),[1669020675.344435012,0, 1669020675.380563306,0)-><nil> [w]/Table/100/"25be7481":1669020675.378291802,1->v-9 [rs]/Table/100/"{76f506a5"-d4d44a56"}:{gap:[<min>, 1669020675.344435012,0),[1669020675.344435012,0, 1669020675.348339423,0)}->[] [d]/Table/100/"b135a3dd":1669020675.378291802,1-><nil> [d]/Table/100/"4506fec4":1669020675.378291802,1-><nil>
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2872175079
--- FAIL: TestKVNemesisSingleNode (1.23s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7639476?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7639476?buildTab=artifacts#/) on master @ [dca415eddac0d659ae6d76b4e3dfdf4076adbd34](https://github.com/cockroachdb/cockroach/commits/dca415eddac0d659ae6d76b4e3dfdf4076adbd34):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode4095949070
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [r]/Table/100/"ee077c6f":[<min>, <max>)-><nil> [rs]/Table/100/"{0fbd7ad7"-9ed48171"}:{0:[0,0, 0,0), gap:[<min>, 1669106927.475227880,0)}->[/Table/100/"64c80bdf":v-6] [d]/Table/100/"eee2bce9":1669106927.475036953,1-><nil> [w]/Table/100/"64c80bdf":1669106927.475036953,1->v-6
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [r]/Table/100/"ee077c6f":[<min>, <max>)-><nil> [rs]/Table/100/"{0fbd7ad7"-9ed48171"}:{0:[0,0, 0,0), gap:[<min>, 1669106927.475227880,0)}->[/Table/100/"64c80bdf":v-6] [d]/Table/100/"eee2bce9":1669106927.475036953,1-><nil> [w]/Table/100/"64c80bdf":1669106927.475036953,1->v-6
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode4095949070
--- FAIL: TestKVNemesisSingleNode (0.79s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7688447?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7688447?buildTab=artifacts#/) on master @ [b5be006bedd7d3cedc3fb3d2248df168e3d64be2](https://github.com/cockroachdb/cockroach/commits/b5be006bedd7d3cedc3fb3d2248df168e3d64be2):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1533383890
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [rs]/Table/100/"{704c00d4"-a0d6c1b8"}:{0:[1669365856.430646546,1, 1669365856.470810930,1), 1:[0,0, 0,0), gap:[<min>, <max>)}->[/Table/100/"9b31d5b5":v-0, /Table/100/"808fb884":v-1] [w]/Table/100/"808fb884":1669365856.431869167,1->v-1 [w]/Table/100/"dd677ed9":1669365856.431869167,1->v-2
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [rs]/Table/100/"{704c00d4"-a0d6c1b8"}:{0:[1669365856.430646546,1, 1669365856.470810930,1), 1:[0,0, 0,0), gap:[<min>, <max>)}->[/Table/100/"9b31d5b5":v-0, /Table/100/"808fb884":v-1] [w]/Table/100/"808fb884":1669365856.431869167,1->v-1 [w]/Table/100/"dd677ed9":1669365856.431869167,1->v-2
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1533383890
--- FAIL: TestKVNemesisSingleNode (1.29s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7711736?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7711736?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1074127392
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{8f955590"-c277a8c8"}:{gap:[<min>, 1669538192.096924691,0),[1669538192.096924691,0, 1669538192.286584912,0),[1669538192.286584912,0, 1669538192.380524394,0)}->[] [d]/Table/100/"da18e5b5":1669538192.377508801,1-><nil> [w]/Table/100/"9c486a3d":1669538192.377508801,1->v-4 [s]/Table/100/"{5569ed5f"-bdba839c"}:{0:[1669538192.377508801,1, 1669538192.380524394,0), gap:[<min>, 1669538192.286584912,0),[1669538192.286584912,0, 1669538192.380524394,0),[1669538192.380524394,0, 1669538192.417487483,0),[1669538192.417487483,0, <max>)}->[/Table/100/"9c486a3d":v-4] [rs]/Table/100/"{8f523fdb"-f458941f"}:{0:[1669538192.369070046,0, <max>), 1:[1669538192.377508801,1, 1669538192.380524394,0), gap:[<min>, 1669538192.109623049,0)}->[/Table/100/"d9700274":v-8, /Table/100/"9c486a3d":v-4] [d]/Table/100/"d2d4b0fb":1669538192.377508801,1-><nil> [d]/Table/100/"3a474691":1669538192.377508801,1-><nil>
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{8f955590"-c277a8c8"}:{gap:[<min>, 1669538192.096924691,0),[1669538192.096924691,0, 1669538192.286584912,0),[1669538192.286584912,0, 1669538192.380524394,0)}->[] [d]/Table/100/"da18e5b5":1669538192.377508801,1-><nil> [w]/Table/100/"9c486a3d":1669538192.377508801,1->v-4 [s]/Table/100/"{5569ed5f"-bdba839c"}:{0:[1669538192.377508801,1, 1669538192.380524394,0), gap:[<min>, 1669538192.286584912,0),[1669538192.286584912,0, 1669538192.380524394,0),[1669538192.380524394,0, 1669538192.417487483,0),[1669538192.417487483,0, <max>)}->[/Table/100/"9c486a3d":v-4] [rs]/Table/100/"{8f523fdb"-f458941f"}:{0:[1669538192.369070046,0, <max>), 1:[1669538192.377508801,1, 1669538192.380524394,0), gap:[<min>, 1669538192.109623049,0)}->[/Table/100/"d9700274":v-8, /Table/100/"9c486a3d":v-4] [d]/Table/100/"d2d4b0fb":1669538192.377508801,1-><nil> [d]/Table/100/"3a474691":1669538192.377508801,1-><nil>
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1074127392
--- FAIL: TestKVNemesisSingleNode (2.73s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7785875?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7785875?buildTab=artifacts#/) on master @ [db71e045cc3f2937bf20170d254dd7e54a620ffb](https://github.com/cockroachdb/cockroach/commits/db71e045cc3f2937bf20170d254dd7e54a620ffb):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode3760147621
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [w]/Table/100/"7884d6f4":1669886068.593403986,1->v-1526 [r]/Table/100/"7dd3299a":[<min>, 1669886066.505926981,0),[1669886066.505926981,0, <max>)-><nil> [w]/Table/100/"e7db2613":1669886068.593403986,1->v-1527 [s]/Table/100/"{14872ec7"-6ee7062e"}:{0:[0,0, 0,0), gap:[<min>, 1669885970.469447141,0),[1669885973.501189961,1, 1669885973.538622043,1),[1669886005.711914041,1, 1669886005.792582258,0),[1669886005.528332020,0, 1669886005.711914041,1),[1669886004.643010453,0, 1669886004.729842868,1),[1669886004.854093753,1, 1669886004.928708953,0),[1669886005.356519208,0, 1669886005.528332020,0),[1669886009.342747655,1, 1669886009.533952236,1),[1669886008.908347138,0, 1669886008.973817458,0),[1669886008.734058043,0, 1669886008.908347138,0),[1669886008.641999895,0, 1669886008.734058043,0),[1669885994.120991504,1, 1669885994.272505303,0),[1669885994.272505303,0, 1669885994.276155420,0),[1669885990.497578438,0, 1669885990.538992102,0),[1669885990.488086574,0, 1669885990.489401386,0),[1669885990.464342465,0, 1669885990.488086574,0),[1669885994.278957819,0, 1669885994.293831486,0),[1669885994.277940888,0, 1669885994.278957819,0),[1669885994.276155420,0, 1669885994.277940888,0),[1669885994.482108118,1, 1669885994.498743808,0),[1669885994.362764521,2, 1669885994.364372363,0),[1669885994.364372363,0, 1669885994.396512298,0),[1669886001.255818186,1, 1669886001.255818186,2),[1669886001.255818186,2, 1669886001.413840467,0),[1669886001.413840467,0, 1669886001.515976004,0),[1669886001.515976004,0, 1669886001.517416208,0),[1669886001.602765169,1, 1669886001.639252777,0),[1669885981.957210276,0, 1669885982.032637583,0),[1669885983.402710344,1, 1669885983.563049250,0),[1669885983.658080901,1, 1669885983.713397118,0),[1669885980.135146965,1, 1669885980.198355316,0),[1669885980.351410440,0, 1669885980.372161436,0),[1669885980.372161436,0, 1669885980.372161436,2),[1669885980.372161436,2, 1669885980.375608702,0),[1669885978.574496474,0, 1669885978.614399140,0),[1669885978.614399140,0, 1669885978.653870415,0),[1669885978.653870415,0, 1669885978.739209772,0),[1669886095.391231629,0, 1669886095.427021596,0),[1669886095.229550954,0, 1669886095.391231629,0),[1669886100.639682865,1, 1669886100.861343062,0),[1669886100.861343062,0, 1669886101.126447279,0),[1669886108.803661630,0, 1669886108.845541630,0),[1669886108.845541630,0, 1669886108.949914721,0),[1669886114.190562956,1, 1669886114.425914548,0),[1669886115.060428334,0, 1669886115.507732996,0),[1669886016.291044351,1, 1669886016.377988965,0),[1669886066.580211713,2, 1669886066.901576218,0),[1669886066.901576218,0, 1669886066.927639348,1),[1669886066.927639348,1, 1669886066.951520519,1),[1669886066.965429359,0, 1669886067.053111409,0),[1669886058.969178336,0, 1669886058.972860138,0),[1669886058.652793222,0, 1669886058.909737632,0),[1669886058.909737632,0, 1669886058.912044320,0),[1669886058.912044320,0, 1669886058.969178336,0),[1669886058.650451920,1, 1669886058.652793222,0),[1669886063.775197116,2, 1669886063.944611173,0),[1669886064.169638221,0, 1669886064.265860078,0),[1669886064.265860078,0, 1669886064.300696368,0),[1669886064.374074810,0, 1669886064.652052144,0),[1669886068.731932733,0, 1669886068.758905673,0),[1669886068.593403986,2, 1669886068.731932733,0),[1669886068.320490593,2, 1669886068.593403986,2),[1669886076.742023612,0, 1669886076.862668415,0),[1669886069.956117074,0, 1669886069.978569388,0),[1669886069.978569388,0, 1669886070.010160569,0),[1669886084.688439701,0, 1669886084.723410540,0),[1669886084.723410540,0, 1669886084.729925376,0),[1669886084.824686836,0, 1669886084.826154529,1),[1669886084.729925376,0, 1669886084.824686836,0),[1669886087.820594488,0, 1669886087.864652168,0),[1669886087.864652168,0, 1669886087.875469729,0),[1669886087.875469729,0, 1669886087.900396491,0),[1669886022.327599577,1, 1669886022.367806820,0),[1669886022.393066061,1, 1669886022.395362065,0),[1669886022.395362065,0, 1669886022.764668306,1),[1669886022.925544841,0, 1669886023.006685925,1),[1669886022.764668306,1, 1669886022.925544841,0),[1669886031.040326067,2, 1669886031.320834631,0),[1669886031.320834631,0, 1669886031.321802752,0),[1669886040.328836404,0, 1669886040.570290790,0),[1669886040.876861614,0, 1669886040.889162855,0),[1669886040.839049317,0, 1669886040.876861614,0),[1669886047.590882273,1, 1669886047.622895102,0)}->[/Table/100/"174767a2":v-1528] [w]/Table/100/"174767a2":1669886068.593403986,1->v-1528 [d]/Table/100/"85df1278":1669886068.593403986,1-><nil>
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [w]/Table/100/"7884d6f4":1669886068.593403986,1->v-1526 [r]/Table/100/"7dd3299a":[<min>, 1669886066.505926981,0),[1669886066.505926981,0, <max>)-><nil> [w]/Table/100/"e7db2613":1669886068.593403986,1->v-1527 [s]/Table/100/"{14872ec7"-6ee7062e"}:{0:[0,0, 0,0), gap:[<min>, 1669885970.469447141,0),[1669885973.501189961,1, 1669885973.538622043,1),[1669886005.711914041,1, 1669886005.792582258,0),[1669886005.528332020,0, 1669886005.711914041,1),[1669886004.643010453,0, 1669886004.729842868,1),[1669886004.854093753,1, 1669886004.928708953,0),[1669886005.356519208,0, 1669886005.528332020,0),[1669886009.342747655,1, 1669886009.533952236,1),[1669886008.908347138,0, 1669886008.973817458,0),[1669886008.734058043,0, 1669886008.908347138,0),[1669886008.641999895,0, 1669886008.734058043,0),[1669885994.120991504,1, 1669885994.272505303,0),[1669885994.272505303,0, 1669885994.276155420,0),[1669885990.497578438,0, 1669885990.538992102,0),[1669885990.488086574,0, 1669885990.489401386,0),[1669885990.464342465,0, 1669885990.488086574,0),[1669885994.278957819,0, 1669885994.293831486,0),[1669885994.277940888,0, 1669885994.278957819,0),[1669885994.276155420,0, 1669885994.277940888,0),[1669885994.482108118,1, 1669885994.498743808,0),[1669885994.362764521,2, 1669885994.364372363,0),[1669885994.364372363,0, 1669885994.396512298,0),[1669886001.255818186,1, 1669886001.255818186,2),[1669886001.255818186,2, 1669886001.413840467,0),[1669886001.413840467,0, 1669886001.515976004,0),[1669886001.515976004,0, 1669886001.517416208,0),[1669886001.602765169,1, 1669886001.639252777,0),[1669885981.957210276,0, 1669885982.032637583,0),[1669885983.402710344,1, 1669885983.563049250,0),[1669885983.658080901,1, 1669885983.713397118,0),[1669885980.135146965,1, 1669885980.198355316,0),[1669885980.351410440,0, 1669885980.372161436,0),[1669885980.372161436,0, 1669885980.372161436,2),[1669885980.372161436,2, 1669885980.375608702,0),[1669885978.574496474,0, 1669885978.614399140,0),[1669885978.614399140,0, 1669885978.653870415,0),[1669885978.653870415,0, 1669885978.739209772,0),[1669886095.391231629,0, 1669886095.427021596,0),[1669886095.229550954,0, 1669886095.391231629,0),[1669886100.639682865,1, 1669886100.861343062,0),[1669886100.861343062,0, 1669886101.126447279,0),[1669886108.803661630,0, 1669886108.845541630,0),[1669886108.845541630,0, 1669886108.949914721,0),[1669886114.190562956,1, 1669886114.425914548,0),[1669886115.060428334,0, 1669886115.507732996,0),[1669886016.291044351,1, 1669886016.377988965,0),[1669886066.580211713,2, 1669886066.901576218,0),[1669886066.901576218,0, 1669886066.927639348,1),[1669886066.927639348,1, 1669886066.951520519,1),[1669886066.965429359,0, 1669886067.053111409,0),[1669886058.969178336,0, 1669886058.972860138,0),[1669886058.652793222,0, 1669886058.909737632,0),[1669886058.909737632,0, 1669886058.912044320,0),[1669886058.912044320,0, 1669886058.969178336,0),[1669886058.650451920,1, 1669886058.652793222,0),[1669886063.775197116,2, 1669886063.944611173,0),[1669886064.169638221,0, 1669886064.265860078,0),[1669886064.265860078,0, 1669886064.300696368,0),[1669886064.374074810,0, 1669886064.652052144,0),[1669886068.731932733,0, 1669886068.758905673,0),[1669886068.593403986,2, 1669886068.731932733,0),[1669886068.320490593,2, 1669886068.593403986,2),[1669886076.742023612,0, 1669886076.862668415,0),[1669886069.956117074,0, 1669886069.978569388,0),[1669886069.978569388,0, 1669886070.010160569,0),[1669886084.688439701,0, 1669886084.723410540,0),[1669886084.723410540,0, 1669886084.729925376,0),[1669886084.824686836,0, 1669886084.826154529,1),[1669886084.729925376,0, 1669886084.824686836,0),[1669886087.820594488,0, 1669886087.864652168,0),[1669886087.864652168,0, 1669886087.875469729,0),[1669886087.875469729,0, 1669886087.900396491,0),[1669886022.327599577,1, 1669886022.367806820,0),[1669886022.393066061,1, 1669886022.395362065,0),[1669886022.395362065,0, 1669886022.764668306,1),[1669886022.925544841,0, 1669886023.006685925,1),[1669886022.764668306,1, 1669886022.925544841,0),[1669886031.040326067,2, 1669886031.320834631,0),[1669886031.320834631,0, 1669886031.321802752,0),[1669886040.328836404,0, 1669886040.570290790,0),[1669886040.876861614,0, 1669886040.889162855,0),[1669886040.839049317,0, 1669886040.876861614,0),[1669886047.590882273,1, 1669886047.622895102,0)}->[/Table/100/"174767a2":v-1528] [w]/Table/100/"174767a2":1669886068.593403986,1->v-1528 [d]/Table/100/"85df1278":1669886068.593403986,1-><nil>
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode3760147621
--- FAIL: TestKVNemesisSingleNode (218.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7802182?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7802182?buildTab=artifacts#/) on master @ [e7b15ebaed9c14668ade0a7827a5525aedef1ab0](https://github.com/cockroachdb/cockroach/commits/e7b15ebaed9c14668ade0a7827a5525aedef1ab0):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1451760855
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{052530e4"-b1fdf57b"}:{0:[1669976308.716441032,1, 1669976312.185638275,1), 1:[1669976308.716441032,0, 1669976312.185638275,1), 2:[1669976309.539350871,0, 1669976309.960944331,1), 3:[1669976309.429658560,0, 1669976309.960944331,1), 4:[1669976308.957780676,0, 1669976310.586382506,1), 5:[1669976309.418678077,0, 1669976310.586382506,1), 6:[0,0, 0,0), 7:[1669976309.397548084,0, 1669976310.586382506,1), 8:[1669976309.539350871,0, 1669976310.586382506,1), 9:[1669976308.688999380,0, 1669976310.586382506,1), 10:[1669976309.525298339,1, 1669976310.586382506,1), 11:[1669976309.467921224,0, 1669976309.948826655,1), 12:[1669976308.494618001,0, 1669976309.948826655,1), 13:[1669976308.619988033,0, 1669976309.948826655,1), gap:[<min>, 1669976268.225812480,0),[1669976268.227366412,1, 1669976268.229376148,0),[1669976268.229376148,0, 1669976268.232647056,1),[1669976268.225812480,0, 1669976268.227366412,1),[1669976289.368602787,0, 1669976289.372962388,0),[1669976309.428113027,0, 1669976309.494544495,0),[1669976309.112697638,0, 1669976309.415207249,0),[1669976309.415207249,0, 1669976309.428113027,0),[1669976309.104035745,0, 1669976309.112697638,0),[1669976309.030503582,0, 1669976309.104035745,0),[1669976308.957780676,0, 1669976309.030503582,0),[1669976308.716441032,0, 1669976308.818542585,0),[1669976308.818542585,0, 1669976308.957780676,0),[1669976308.619988033,0, 1669976308.716441032,0),[1669976309.545654225,0, 1669976309.738377406,0),[1669976309.494544495,0, 1669976309.545654225,0),[1669976309.738377406,0, 1669976309.758897760,0),[1669976307.955619517,1, 1669976308.205971517,0),[1669976302.142238629,0, 1669976302.146034817,0),[1669976302.004917305,1, 1669976302.142238629,0),[1669976300.872828766,1, 1669976300.905138721,0),[1669976300.803211681,1, 1669976300.870800584,0),[1669976300.906786342,0, 1669976300.908536379,0),[1669976300.905138721,0, 1669976300.906786342,0),[1669976296.569684308,1, 1669976296.718439241,0),[1669976342.982924813,1, 1669976343.089942365,0),[1669976343.530993137,0, 1669976343.629128779,0),[1669976343.629128779,0, 1669976343.933344018,0),[1669976399.900965747,0, 1669976399.924458684,0)}->[/Table/100/"16e4c753":v-743, /Table/100/"199d3c10":v-746, /Table/100/"2f578058":v-762, /Table/100/"340ffc92":v-758, /Table/100/"37f1d032":v-747, /Table/100/"4131eb0f":v-757, /Table/100/"48526632":v-764, /Table/100/"4e1b56ab":v-755, /Table/100/"4f08d403":v-763, /Table/100/"52c1ae73":v-745, /Table/100/"5aa11c86":v-759, /Table/100/"ac84d20c":v-760, /Table/100/"adb7b34c":v-739, /Table/100/"add77726":v-744] [d]/Table/100/"8b8b6d74":1669976309.653586837,1-><nil> [w]/Table/100/"48526632":1669976309.653586837,1->v-764
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{052530e4"-b1fdf57b"}:{0:[1669976308.716441032,1, 1669976312.185638275,1), 1:[1669976308.716441032,0, 1669976312.185638275,1), 2:[1669976309.539350871,0, 1669976309.960944331,1), 3:[1669976309.429658560,0, 1669976309.960944331,1), 4:[1669976308.957780676,0, 1669976310.586382506,1), 5:[1669976309.418678077,0, 1669976310.586382506,1), 6:[0,0, 0,0), 7:[1669976309.397548084,0, 1669976310.586382506,1), 8:[1669976309.539350871,0, 1669976310.586382506,1), 9:[1669976308.688999380,0, 1669976310.586382506,1), 10:[1669976309.525298339,1, 1669976310.586382506,1), 11:[1669976309.467921224,0, 1669976309.948826655,1), 12:[1669976308.494618001,0, 1669976309.948826655,1), 13:[1669976308.619988033,0, 1669976309.948826655,1), gap:[<min>, 1669976268.225812480,0),[1669976268.227366412,1, 1669976268.229376148,0),[1669976268.229376148,0, 1669976268.232647056,1),[1669976268.225812480,0, 1669976268.227366412,1),[1669976289.368602787,0, 1669976289.372962388,0),[1669976309.428113027,0, 1669976309.494544495,0),[1669976309.112697638,0, 1669976309.415207249,0),[1669976309.415207249,0, 1669976309.428113027,0),[1669976309.104035745,0, 1669976309.112697638,0),[1669976309.030503582,0, 1669976309.104035745,0),[1669976308.957780676,0, 1669976309.030503582,0),[1669976308.716441032,0, 1669976308.818542585,0),[1669976308.818542585,0, 1669976308.957780676,0),[1669976308.619988033,0, 1669976308.716441032,0),[1669976309.545654225,0, 1669976309.738377406,0),[1669976309.494544495,0, 1669976309.545654225,0),[1669976309.738377406,0, 1669976309.758897760,0),[1669976307.955619517,1, 1669976308.205971517,0),[1669976302.142238629,0, 1669976302.146034817,0),[1669976302.004917305,1, 1669976302.142238629,0),[1669976300.872828766,1, 1669976300.905138721,0),[1669976300.803211681,1, 1669976300.870800584,0),[1669976300.906786342,0, 1669976300.908536379,0),[1669976300.905138721,0, 1669976300.906786342,0),[1669976296.569684308,1, 1669976296.718439241,0),[1669976342.982924813,1, 1669976343.089942365,0),[1669976343.530993137,0, 1669976343.629128779,0),[1669976343.629128779,0, 1669976343.933344018,0),[1669976399.900965747,0, 1669976399.924458684,0)}->[/Table/100/"16e4c753":v-743, /Table/100/"199d3c10":v-746, /Table/100/"2f578058":v-762, /Table/100/"340ffc92":v-758, /Table/100/"37f1d032":v-747, /Table/100/"4131eb0f":v-757, /Table/100/"48526632":v-764, /Table/100/"4e1b56ab":v-755, /Table/100/"4f08d403":v-763, /Table/100/"52c1ae73":v-745, /Table/100/"5aa11c86":v-759, /Table/100/"ac84d20c":v-760, /Table/100/"adb7b34c":v-739, /Table/100/"add77726":v-744] [d]/Table/100/"8b8b6d74":1669976309.653586837,1-><nil> [w]/Table/100/"48526632":1669976309.653586837,1->v-764
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode1451760855
--- FAIL: TestKVNemesisSingleNode (201.21s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7818297?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7818297?buildTab=artifacts#/) on master @ [ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410](https://github.com/cockroachdb/cockroach/commits/ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode557267387
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [r]/Table/100/"c82e1188":[<min>, <max>)-><nil> [w]/Table/100/"b48058e5":1670062007.352459293,1->v-1349 [s]/Table/100/"{37f9a7be"-d7ac41f1"}:{0:[1670062004.223568436,0, 1670062009.415013503,1), 1:[1670062005.906464782,0, 1670062009.415013503,1), 2:[1670062007.187623873,0, 1670062009.415013503,1), 3:[1670062005.752376707,0, 1670062009.415013503,1), 4:[1670062004.408689465,0, 1670062009.415013503,1), 5:[1670062006.961254556,0, 1670062009.415013503,1), 6:[1670062005.943377703,0, 1670062009.415013503,1), 7:[1670062005.729582704,0, 1670062009.415013503,1), 8:[1670062005.191645098,0, 1670062009.415013503,1), 9:[1670062004.179446320,0, 1670062009.415013503,1), 10:[1670062004.261211489,0, 1670062009.415013503,1), 11:[1670062006.755733210,0, 1670062009.415013503,1), 12:[1670062006.317154274,0, 1670062009.415013503,1), 13:[1670062004.161334896,1, 1670062009.415013503,1), 14:[1670062004.269074868,0, 1670062009.415013503,1), 15:[1670062006.141828615,0, 1670062009.415013503,1), 16:[1670062004.200918022,0, 1670062009.415013503,1), 17:[1670062006.406540885,0, 1670062007.637092493,1), 18:[1670062007.352459293,1, 1670062007.637092493,1), 19:[1670062006.766085007,0, 1670062007.637092493,1), 20:[1670062006.354608458,0, 1670062007.637092493,1), 21:[1670062006.622106479,1, 1670062007.637092493,1), 22:[1670062006.344484765,0, 1670062007.637092493,1), gap:[<min>, 1670061925.184303846,0),[1670061925.187040229,0, 1670061925.200588762,0),[1670061925.184303846,0, 1670061925.187040229,0),[1670061949.746335616,0, 1670061949.749351003,0),[1670061949.200411571,0, 1670061949.241437798,0),[1670061937.677361763,1, 1670061937.694640889,0),[1670061940.157274394,0, 1670061940.159879238,0),[1670061940.159879238,0, 1670061940.182227655,0),[1670061940.124901231,0, 1670061940.157274394,0),[1670061940.112471822,0, 1670061940.117668745,0),[1670061940.117668745,0, 1670061940.124901231,0),[1670061940.068116508,1, 1670061940.112471822,0),[1670061936.156895582,1, 1670061936.162581735,1),[1670061930.037050546,1, 1670061930.076981662,1),[1670061959.820080173,0, 1670061959.821106416,0),[1670061959.603070012,0, 1670061959.814599245,0),[1670061959.814599245,0, 1670061959.814599245,1),[1670061959.814599245,1, 1670061959.817831665,0),[1670061959.817831665,0, 1670061959.820080173,0),[1670061959.410041174,1, 1670061959.603070012,0),[1670061960.645752223,1, 1670061960.771837843,0),[1670061960.771837843,0, 1670061960.776514634,0),[1670061971.141381264,1, 1670061971.204915897,0),[1670061971.961771709,1, 1670061972.084423998,1),[1670061972.084423998,1, 1670061972.115268295,0),[1670061972.115268295,0, 1670061972.148460829,0),[1670061972.169624968,0, 1670061972.180084308,0),[1670061972.154704379,0, 1670061972.169624968,0),[1670061972.148460829,0, 1670061972.154704379,0),[1670061974.327313512,1, 1670061974.356377539,0),[1670061974.496437959,0, 1670061974.499163025,0),[1670061974.452339865,0, 1670061974.496437959,0),[1670061975.894861205,0, 1670061976.128192785,0),[1670061976.159236084,0, 1670061976.168540808,0),[1670061976.149868719,0, 1670061976.159236084,0),[1670061976.128192785,0, 1670061976.149868719,0),[1670061979.039496186,1, 1670061979.071227951,0),[1670062006.306519380,0, 1670062006.336688664,0),[1670062006.262995309,0, 1670062006.306519380,0),[1670062006.442967686,0, 1670062006.478323667,0),[1670062006.478323667,0, 1670062006.479583796,0),[1670062006.479583796,0, 1670062006.663153171,0),[1670062007.216571030,0, 1670062007.636286650,0),[1670062006.679992083,0, 1670062006.694839111,0),[1670062006.663153171,0, 1670062006.679992083,0),[1670062006.694839111,0, 1670062007.216571030,0),[1670062006.418120936,0, 1670062006.442967686,0),[1670062006.397016135,0, 1670062006.418120936,0),[1670062006.344484765,0, 1670062006.397016135,0),[1670062006.336688664,1, 1670062006.344484765,0),[1670062006.336688664,0, 1670062006.336688664,1),[1670062007.637092493,1, 1670062007.791530288,0),[1670062007.636286650,0, 1670062007.637092493,1),[1670062009.730355411,0, 1670062009.733194519,0),[1670062009.415013503,1, 1670062009.730355411,0),[1670062011.973135674,1, 1670062012.068412705,0),[1670062012.068412705,0, 1670062012.071360336,0),[1670061996.862296802,0, 1670061996.928996069,0),[1670061996.928996069,0, 1670061996.930332009,0),[1670061992.509912313,0, 1670061992.640105524,1),[1670061992.640105524,1, 1670061992.658517318,0),[1670061992.658517318,0, 1670061992.688987934,0),[1670062013.406460904,0, 1670062013.576472446,1),[1670061985.252538791,0, 1670061985.268656834,0),[1670061985.222359877,0, 1670061985.252538791,0),[1670062062.276562800,0, 1670062062.287388300,1),[1670062062.224089971,1, 1670062062.276562800,0),[1670062062.287388300,1, 1670062062.367836308,0),[1670062060.178012908,0, 1670062060.206067694,0),[1670062060.206067694,0, 1670062060.214545264,0),[1670062066.405714206,0, 1670062066.473907370,0),[1670062018.365894969,0, 1670062018.439505492,0),[1670062018.468826842,0, 1670062018.553695040,0),[1670062053.153969533,0, 1670062053.180336107,0),[1670062053.031373345,0, 1670062053.153969533,0),[1670062053.203214372,0, 1670062053.237380635,0),[1670062059.982188470,0, 1670062060.046494317,0),[1670062060.046494317,0, 1670062060.117926294,0),[1670062060.117926294,0, 1670062060.178012908,0),[1670062043.538215721,0, 1670062043.717021560,0)}->[/Table/100/"40e03a52":v-1305, /Table/100/"452c3746":v-1321, /Table/100/"457169c3":v-1347, /Table/100/"4984fcbe":v-1319, /Table/100/"4eee12a7":v-1310, /Table/100/"4f9d9b19":v-1345, /Table/100/"5314755f":v-1324, /Table/100/"53e3bcc4":v-1318, /Table/100/"68b6175e":v-1315, /Table/100/"6c50e31f":v-1303, /Table/100/"6dd1cebd":v-1308, /Table/100/"6f41805b":v-1343, /Table/100/"6fec4a8c":v-1333, /Table/100/"7a2abe18":v-1300, /Table/100/"838a7167":v-1309, /Table/100/"9222d901":v-1328, /Table/100/"953d09d1":v-1304, /Table/100/"b2aaa961":v-1336, /Table/100/"b48058e5":v-1349, /Table/100/"c48c40ba":v-1344, /Table/100/"c5053c7e":v-1335, /Table/100/"ccac81a4":v-1340, /Table/100/"d24b59ee":v-1334] [s]/Table/100/"{6dd81807"-d345404d"}:{0:[1670062006.755733210,0, 1670062009.415013503,1), 1:[1670062006.317154274,0, 1670062009.415013503,1), 2:[1670062004.161334896,1, 1670062009.415013503,1), 3:[1670062004.269074868,0, 1670062009.415013503,1), 4:[1670062006.141828615,0, 1670062009.415013503,1), 5:[1670062004.200918022,0, 1670062009.415013503,1), 6:[0,0, 0,0), 7:[1670062006.406540885,0, 1670062007.637092493,1), 8:[1670062007.352459293,1, 1670062007.637092493,1), 9:[1670062006.766085007,0, 1670062007.637092493,1), 10:[1670062006.354608458,0, 1670062007.637092493,1), 11:[1670062006.622106479,1, 1670062007.637092493,1), 12:[1670062006.344484765,0, 1670062007.637092493,1), gap:[<min>, 1670061925.212851988,0),[1670061925.212851988,0, 1670061925.226757959,0),[1670061940.068116508,1, 1670061940.112471822,0),[1670061940.112471822,0, 1670061940.117668745,0),[1670061940.117668745,0, 1670061940.159879238,0),[1670061940.159879238,0, 1670061940.184230852,0),[1670061949.200411571,0, 1670061949.244058795,0),[1670061941.686952832,1, 1670061941.821494568,0),[1670061949.928363804,1, 1670061950.109020315,1),[1670061949.746335616,0, 1670061949.928363804,1),[1670061935.381886990,0, 1670061935.381886990,1),[1670061936.156895582,1, 1670061936.268690708,0),[1670061937.677361763,1, 1670061937.694640889,0),[1670061931.497811668,0, 1670061931.508579573,0),[1670061931.508579573,0, 1670061931.572968943,1),[1670061930.037050546,1, 1670061930.154383894,0),[1670061939.249176063,0, 1670061939.278010797,0),[1670061971.141381264,1, 1670061971.204915897,0),[1670061971.961771709,1, 1670061972.084423998,1),[1670061972.084423998,1, 1670061972.115268295,0),[1670061972.154704379,0, 1670061972.180084308,0),[1670061972.115268295,0, 1670061972.148460829,0),[1670061972.148460829,0, 1670061972.154704379,0),[1670061955.114978303,1, 1670061955.386388498,0),[1670061955.386388498,0, 1670061955.405269332,0),[1670061959.603070012,0, 1670061959.814599245,0),[1670061959.814599245,0, 1670061959.814599245,1),[1670061959.814599245,1, 1670061959.817831665,0),[1670061959.410041174,1, 1670061959.603070012,0),[1670061958.228740742,1, 1670061958.383049169,0),[1670061960.645752223,1, 1670061960.771837843,0),[1670061960.771837843,0, 1670061960.865010849,0),[1670061959.916368704,0, 1670061959.917382820,0),[1670061959.839629949,0, 1670061959.916368704,0),[1670061959.820080173,0, 1670061959.839629949,0),[1670061959.917382820,0, 1670061959.917382820,1),[1670061959.817831665,0, 1670061959.820080173,0),[1670061969.281549834,0, 1670061969.289628505,0),[1670061969.182189237,1, 1670061969.281549834,0),[1670061967.109653540,0, 1670061967.311903477,0),[1670061967.311903477,0, 1670061967.384963389,0),[1670061974.499163025,0, 1670061974.546094545,0),[1670061975.894861205,0, 1670061976.128192785,0),[1670061976.159236084,0, 1670061976.168540808,0),[1670061976.149868719,0, 1670061976.159236084,0),[1670061976.128192785,0, 1670061976.149868719,0),[1670061979.039496186,1, 1670061979.071227951,0),[1670061974.452339865,0, 1670061974.499163025,0),[1670061974.327313512,1, 1670061974.412333430,0),[1670061985.252538791,0, 1670061985.268656834,0),[1670061985.222359877,0, 1670061985.252538791,0),[1670061985.331269467,1, 1670061985.350094564,0),[1670061984.198578203,1, 1670061984.316370372,1),[1670061992.509912313,0, 1670061992.640105524,1),[1670061992.640105524,1, 1670061992.658517318,0),[1670061992.658517318,0, 1670061992.688987934,0),[1670062013.373207218,0, 1670062013.404274597,0),[1670062013.404274597,0, 1670062013.576472446,1),[1670062011.973135674,0, 1670062012.068412705,0),[1670062012.068412705,0, 1670062012.071360336,0),[1670062012.115794597,0, 1670062012.124826352,0),[1670062009.415013503,1, 1670062009.733194519,0),[1670062013.273054211,0, 1670062013.373207218,0),[1670062006.442967686,0, 1670062006.478323667,0),[1670062006.478323667,0, 1670062006.479583796,0),[1670062006.397016135,0, 1670062006.442967686,0),[1670062007.637092493,1, 1670062007.791530288,0),[1670062006.479583796,0, 1670062006.663153171,0),[1670062006.679992083,0, 1670062006.694839111,0),[1670062006.663153171,0, 1670062006.679992083,0),[1670062007.216571030,0, 1670062007.637092493,1),[1670062006.694839111,0, 1670062007.216571030,0),[1670062006.344484765,0, 1670062006.397016135,0),[1670062006.336688664,1, 1670062006.344484765,0),[1670062006.262995309,0, 1670062006.336688664,1),[1670062019.411235173,0, 1670062019.432301369,0),[1670062018.819667239,1, 1670062019.005559825,0),[1670062018.365894969,0, 1670062018.553695040,0),[1670062017.786565696,1, 1670062017.799014982,0),[1670062021.653220215,0, 1670062021.696071408,0),[1670062021.077272400,1, 1670062021.109296648,0),[1670062028.674961627,1, 1670062028.688204709,0),[1670062028.716861811,0, 1670062028.726442763,0),[1670062028.688204709,0, 1670062028.699508395,0),[1670062029.741624235,1, 1670062029.841059854,0),[1670062001.680797075,1, 1670062001.720176150,2),[1670061996.862296802,0, 1670061996.930332009,0),[1670061994.154184256,1, 1670061994.189205982,0),[1670062040.393992219,2, 1670062040.868292680,0),[1670062043.538215721,0, 1670062043.717021560,0),[1670062053.756683336,2, 1670062053.860176444,0),[1670062053.264993073,0, 1670062053.397679229,0),[1670062053.203214372,0, 1670062053.264993073,0),[1670062050.185326004,0, 1670062050.280840440,0),[1670062050.171727271,0, 1670062050.185326004,0),[1670062053.153969533,0, 1670062053.180336107,0),[1670062053.031373345,0, 1670062053.153969533,0),[1670062056.561341466,1, 1670062056.745992799,0),[1670062062.224089971,1, 1670062062.369094673,0),[1670062063.740031340,1, 1670062064.055163262,1),[1670062064.055163262,1, 1670062064.190443006,0),[1670062064.190443006,0, 1670062064.190941141,0),[1670062059.982188470,0, 1670062060.046494317,0),[1670062060.046494317,0, 1670062060.206067694,0),[1670062060.206067694,0, 1670062060.214545264,0),[1670062066.405714206,0, 1670062066.473907370,0),[1670062068.743793978,2, 1670062068.852000803,0),[1670062068.852000803,0, 1670062069.087694562,0),[1670062069.087694562,0, 1670062069.087694562,1)}->[/Table/100/"6f41805b":v-1343, /Table/100/"6fec4a8c":v-1333, /Table/100/"7a2abe18":v-1300, /Table/100/"838a7167":v-1309, /Table/100/"9222d901":v-1328, /Table/100/"953d09d1":v-1304, /Table/100/"ae1ab0a2":v-1351, /Table/100/"b2aaa961":v-1336, /Table/100/"b48058e5":v-1349, /Table/100/"c48c40ba":v-1344, /Table/100/"c5053c7e":v-1335, /Table/100/"ccac81a4":v-1340, /Table/100/"d24b59ee":v-1334] [w]/Table/100/"6fdd3926":1670062007.352459293,1->v-1350 [w]/Table/100/"ae1ab0a2":1670062007.352459293,1->v-1351
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [r]/Table/100/"c82e1188":[<min>, <max>)-><nil> [w]/Table/100/"b48058e5":1670062007.352459293,1->v-1349 [s]/Table/100/"{37f9a7be"-d7ac41f1"}:{0:[1670062004.223568436,0, 1670062009.415013503,1), 1:[1670062005.906464782,0, 1670062009.415013503,1), 2:[1670062007.187623873,0, 1670062009.415013503,1), 3:[1670062005.752376707,0, 1670062009.415013503,1), 4:[1670062004.408689465,0, 1670062009.415013503,1), 5:[1670062006.961254556,0, 1670062009.415013503,1), 6:[1670062005.943377703,0, 1670062009.415013503,1), 7:[1670062005.729582704,0, 1670062009.415013503,1), 8:[1670062005.191645098,0, 1670062009.415013503,1), 9:[1670062004.179446320,0, 1670062009.415013503,1), 10:[1670062004.261211489,0, 1670062009.415013503,1), 11:[1670062006.755733210,0, 1670062009.415013503,1), 12:[1670062006.317154274,0, 1670062009.415013503,1), 13:[1670062004.161334896,1, 1670062009.415013503,1), 14:[1670062004.269074868,0, 1670062009.415013503,1), 15:[1670062006.141828615,0, 1670062009.415013503,1), 16:[1670062004.200918022,0, 1670062009.415013503,1), 17:[1670062006.406540885,0, 1670062007.637092493,1), 18:[1670062007.352459293,1, 1670062007.637092493,1), 19:[1670062006.766085007,0, 1670062007.637092493,1), 20:[1670062006.354608458,0, 1670062007.637092493,1), 21:[1670062006.622106479,1, 1670062007.637092493,1), 22:[1670062006.344484765,0, 1670062007.637092493,1), gap:[<min>, 1670061925.184303846,0),[1670061925.187040229,0, 1670061925.200588762,0),[1670061925.184303846,0, 1670061925.187040229,0),[1670061949.746335616,0, 1670061949.749351003,0),[1670061949.200411571,0, 1670061949.241437798,0),[1670061937.677361763,1, 1670061937.694640889,0),[1670061940.157274394,0, 1670061940.159879238,0),[1670061940.159879238,0, 1670061940.182227655,0),[1670061940.124901231,0, 1670061940.157274394,0),[1670061940.112471822,0, 1670061940.117668745,0),[1670061940.117668745,0, 1670061940.124901231,0),[1670061940.068116508,1, 1670061940.112471822,0),[1670061936.156895582,1, 1670061936.162581735,1),[1670061930.037050546,1, 1670061930.076981662,1),[1670061959.820080173,0, 1670061959.821106416,0),[1670061959.603070012,0, 1670061959.814599245,0),[1670061959.814599245,0, 1670061959.814599245,1),[1670061959.814599245,1, 1670061959.817831665,0),[1670061959.817831665,0, 1670061959.820080173,0),[1670061959.410041174,1, 1670061959.603070012,0),[1670061960.645752223,1, 1670061960.771837843,0),[1670061960.771837843,0, 1670061960.776514634,0),[1670061971.141381264,1, 1670061971.204915897,0),[1670061971.961771709,1, 1670061972.084423998,1),[1670061972.084423998,1, 1670061972.115268295,0),[1670061972.115268295,0, 1670061972.148460829,0),[1670061972.169624968,0, 1670061972.180084308,0),[1670061972.154704379,0, 1670061972.169624968,0),[1670061972.148460829,0, 1670061972.154704379,0),[1670061974.327313512,1, 1670061974.356377539,0),[1670061974.496437959,0, 1670061974.499163025,0),[1670061974.452339865,0, 1670061974.496437959,0),[1670061975.894861205,0, 1670061976.128192785,0),[1670061976.159236084,0, 1670061976.168540808,0),[1670061976.149868719,0, 1670061976.159236084,0),[1670061976.128192785,0, 1670061976.149868719,0),[1670061979.039496186,1, 1670061979.071227951,0),[1670062006.306519380,0, 1670062006.336688664,0),[1670062006.262995309,0, 1670062006.306519380,0),[1670062006.442967686,0, 1670062006.478323667,0),[1670062006.478323667,0, 1670062006.479583796,0),[1670062006.479583796,0, 1670062006.663153171,0),[1670062007.216571030,0, 1670062007.636286650,0),[1670062006.679992083,0, 1670062006.694839111,0),[1670062006.663153171,0, 1670062006.679992083,0),[1670062006.694839111,0, 1670062007.216571030,0),[1670062006.418120936,0, 1670062006.442967686,0),[1670062006.397016135,0, 1670062006.418120936,0),[1670062006.344484765,0, 1670062006.397016135,0),[1670062006.336688664,1, 1670062006.344484765,0),[1670062006.336688664,0, 1670062006.336688664,1),[1670062007.637092493,1, 1670062007.791530288,0),[1670062007.636286650,0, 1670062007.637092493,1),[1670062009.730355411,0, 1670062009.733194519,0),[1670062009.415013503,1, 1670062009.730355411,0),[1670062011.973135674,1, 1670062012.068412705,0),[1670062012.068412705,0, 1670062012.071360336,0),[1670061996.862296802,0, 1670061996.928996069,0),[1670061996.928996069,0, 1670061996.930332009,0),[1670061992.509912313,0, 1670061992.640105524,1),[1670061992.640105524,1, 1670061992.658517318,0),[1670061992.658517318,0, 1670061992.688987934,0),[1670062013.406460904,0, 1670062013.576472446,1),[1670061985.252538791,0, 1670061985.268656834,0),[1670061985.222359877,0, 1670061985.252538791,0),[1670062062.276562800,0, 1670062062.287388300,1),[1670062062.224089971,1, 1670062062.276562800,0),[1670062062.287388300,1, 1670062062.367836308,0),[1670062060.178012908,0, 1670062060.206067694,0),[1670062060.206067694,0, 1670062060.214545264,0),[1670062066.405714206,0, 1670062066.473907370,0),[1670062018.365894969,0, 1670062018.439505492,0),[1670062018.468826842,0, 1670062018.553695040,0),[1670062053.153969533,0, 1670062053.180336107,0),[1670062053.031373345,0, 1670062053.153969533,0),[1670062053.203214372,0, 1670062053.237380635,0),[1670062059.982188470,0, 1670062060.046494317,0),[1670062060.046494317,0, 1670062060.117926294,0),[1670062060.117926294,0, 1670062060.178012908,0),[1670062043.538215721,0, 1670062043.717021560,0)}->[/Table/100/"40e03a52":v-1305, /Table/100/"452c3746":v-1321, /Table/100/"457169c3":v-1347, /Table/100/"4984fcbe":v-1319, /Table/100/"4eee12a7":v-1310, /Table/100/"4f9d9b19":v-1345, /Table/100/"5314755f":v-1324, /Table/100/"53e3bcc4":v-1318, /Table/100/"68b6175e":v-1315, /Table/100/"6c50e31f":v-1303, /Table/100/"6dd1cebd":v-1308, /Table/100/"6f41805b":v-1343, /Table/100/"6fec4a8c":v-1333, /Table/100/"7a2abe18":v-1300, /Table/100/"838a7167":v-1309, /Table/100/"9222d901":v-1328, /Table/100/"953d09d1":v-1304, /Table/100/"b2aaa961":v-1336, /Table/100/"b48058e5":v-1349, /Table/100/"c48c40ba":v-1344, /Table/100/"c5053c7e":v-1335, /Table/100/"ccac81a4":v-1340, /Table/100/"d24b59ee":v-1334] [s]/Table/100/"{6dd81807"-d345404d"}:{0:[1670062006.755733210,0, 1670062009.415013503,1), 1:[1670062006.317154274,0, 1670062009.415013503,1), 2:[1670062004.161334896,1, 1670062009.415013503,1), 3:[1670062004.269074868,0, 1670062009.415013503,1), 4:[1670062006.141828615,0, 1670062009.415013503,1), 5:[1670062004.200918022,0, 1670062009.415013503,1), 6:[0,0, 0,0), 7:[1670062006.406540885,0, 1670062007.637092493,1), 8:[1670062007.352459293,1, 1670062007.637092493,1), 9:[1670062006.766085007,0, 1670062007.637092493,1), 10:[1670062006.354608458,0, 1670062007.637092493,1), 11:[1670062006.622106479,1, 1670062007.637092493,1), 12:[1670062006.344484765,0, 1670062007.637092493,1), gap:[<min>, 1670061925.212851988,0),[1670061925.212851988,0, 1670061925.226757959,0),[1670061940.068116508,1, 1670061940.112471822,0),[1670061940.112471822,0, 1670061940.117668745,0),[1670061940.117668745,0, 1670061940.159879238,0),[1670061940.159879238,0, 1670061940.184230852,0),[1670061949.200411571,0, 1670061949.244058795,0),[1670061941.686952832,1, 1670061941.821494568,0),[1670061949.928363804,1, 1670061950.109020315,1),[1670061949.746335616,0, 1670061949.928363804,1),[1670061935.381886990,0, 1670061935.381886990,1),[1670061936.156895582,1, 1670061936.268690708,0),[1670061937.677361763,1, 1670061937.694640889,0),[1670061931.497811668,0, 1670061931.508579573,0),[1670061931.508579573,0, 1670061931.572968943,1),[1670061930.037050546,1, 1670061930.154383894,0),[1670061939.249176063,0, 1670061939.278010797,0),[1670061971.141381264,1, 1670061971.204915897,0),[1670061971.961771709,1, 1670061972.084423998,1),[1670061972.084423998,1, 1670061972.115268295,0),[1670061972.154704379,0, 1670061972.180084308,0),[1670061972.115268295,0, 1670061972.148460829,0),[1670061972.148460829,0, 1670061972.154704379,0),[1670061955.114978303,1, 1670061955.386388498,0),[1670061955.386388498,0, 1670061955.405269332,0),[1670061959.603070012,0, 1670061959.814599245,0),[1670061959.814599245,0, 1670061959.814599245,1),[1670061959.814599245,1, 1670061959.817831665,0),[1670061959.410041174,1, 1670061959.603070012,0),[1670061958.228740742,1, 1670061958.383049169,0),[1670061960.645752223,1, 1670061960.771837843,0),[1670061960.771837843,0, 1670061960.865010849,0),[1670061959.916368704,0, 1670061959.917382820,0),[1670061959.839629949,0, 1670061959.916368704,0),[1670061959.820080173,0, 1670061959.839629949,0),[1670061959.917382820,0, 1670061959.917382820,1),[1670061959.817831665,0, 1670061959.820080173,0),[1670061969.281549834,0, 1670061969.289628505,0),[1670061969.182189237,1, 1670061969.281549834,0),[1670061967.109653540,0, 1670061967.311903477,0),[1670061967.311903477,0, 1670061967.384963389,0),[1670061974.499163025,0, 1670061974.546094545,0),[1670061975.894861205,0, 1670061976.128192785,0),[1670061976.159236084,0, 1670061976.168540808,0),[1670061976.149868719,0, 1670061976.159236084,0),[1670061976.128192785,0, 1670061976.149868719,0),[1670061979.039496186,1, 1670061979.071227951,0),[1670061974.452339865,0, 1670061974.499163025,0),[1670061974.327313512,1, 1670061974.412333430,0),[1670061985.252538791,0, 1670061985.268656834,0),[1670061985.222359877,0, 1670061985.252538791,0),[1670061985.331269467,1, 1670061985.350094564,0),[1670061984.198578203,1, 1670061984.316370372,1),[1670061992.509912313,0, 1670061992.640105524,1),[1670061992.640105524,1, 1670061992.658517318,0),[1670061992.658517318,0, 1670061992.688987934,0),[1670062013.373207218,0, 1670062013.404274597,0),[1670062013.404274597,0, 1670062013.576472446,1),[1670062011.973135674,0, 1670062012.068412705,0),[1670062012.068412705,0, 1670062012.071360336,0),[1670062012.115794597,0, 1670062012.124826352,0),[1670062009.415013503,1, 1670062009.733194519,0),[1670062013.273054211,0, 1670062013.373207218,0),[1670062006.442967686,0, 1670062006.478323667,0),[1670062006.478323667,0, 1670062006.479583796,0),[1670062006.397016135,0, 1670062006.442967686,0),[1670062007.637092493,1, 1670062007.791530288,0),[1670062006.479583796,0, 1670062006.663153171,0),[1670062006.679992083,0, 1670062006.694839111,0),[1670062006.663153171,0, 1670062006.679992083,0),[1670062007.216571030,0, 1670062007.637092493,1),[1670062006.694839111,0, 1670062007.216571030,0),[1670062006.344484765,0, 1670062006.397016135,0),[1670062006.336688664,1, 1670062006.344484765,0),[1670062006.262995309,0, 1670062006.336688664,1),[1670062019.411235173,0, 1670062019.432301369,0),[1670062018.819667239,1, 1670062019.005559825,0),[1670062018.365894969,0, 1670062018.553695040,0),[1670062017.786565696,1, 1670062017.799014982,0),[1670062021.653220215,0, 1670062021.696071408,0),[1670062021.077272400,1, 1670062021.109296648,0),[1670062028.674961627,1, 1670062028.688204709,0),[1670062028.716861811,0, 1670062028.726442763,0),[1670062028.688204709,0, 1670062028.699508395,0),[1670062029.741624235,1, 1670062029.841059854,0),[1670062001.680797075,1, 1670062001.720176150,2),[1670061996.862296802,0, 1670061996.930332009,0),[1670061994.154184256,1, 1670061994.189205982,0),[1670062040.393992219,2, 1670062040.868292680,0),[1670062043.538215721,0, 1670062043.717021560,0),[1670062053.756683336,2, 1670062053.860176444,0),[1670062053.264993073,0, 1670062053.397679229,0),[1670062053.203214372,0, 1670062053.264993073,0),[1670062050.185326004,0, 1670062050.280840440,0),[1670062050.171727271,0, 1670062050.185326004,0),[1670062053.153969533,0, 1670062053.180336107,0),[1670062053.031373345,0, 1670062053.153969533,0),[1670062056.561341466,1, 1670062056.745992799,0),[1670062062.224089971,1, 1670062062.369094673,0),[1670062063.740031340,1, 1670062064.055163262,1),[1670062064.055163262,1, 1670062064.190443006,0),[1670062064.190443006,0, 1670062064.190941141,0),[1670062059.982188470,0, 1670062060.046494317,0),[1670062060.046494317,0, 1670062060.206067694,0),[1670062060.206067694,0, 1670062060.214545264,0),[1670062066.405714206,0, 1670062066.473907370,0),[1670062068.743793978,2, 1670062068.852000803,0),[1670062068.852000803,0, 1670062069.087694562,0),[1670062069.087694562,0, 1670062069.087694562,1)}->[/Table/100/"6f41805b":v-1343, /Table/100/"6fec4a8c":v-1333, /Table/100/"7a2abe18":v-1300, /Table/100/"838a7167":v-1309, /Table/100/"9222d901":v-1328, /Table/100/"953d09d1":v-1304, /Table/100/"ae1ab0a2":v-1351, /Table/100/"b2aaa961":v-1336, /Table/100/"b48058e5":v-1349, /Table/100/"c48c40ba":v-1344, /Table/100/"c5053c7e":v-1335, /Table/100/"ccac81a4":v-1340, /Table/100/"d24b59ee":v-1334] [w]/Table/100/"6fdd3926":1670062007.352459293,1->v-1350 [w]/Table/100/"ae1ab0a2":1670062007.352459293,1->v-1351
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode557267387
--- FAIL: TestKVNemesisSingleNode (201.16s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7818301?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7818301?buildTab=artifacts#/) on master @ [ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410](https://github.com/cockroachdb/cockroach/commits/ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode4173209639
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [w]/Table/100/"cb7a3113":1670062030.669450279,1->v-1176 [s]/Table/100/"{65c851c0"-9e1c550a"}:{0:[1670062026.270264388,0, 1670062037.571279388,1), 1:[1670062026.885150437,0, 1670062037.571279388,1), 2:[1670062024.549051602,0, 1670062037.571279388,1), gap:[<min>, 1670061857.280052462,1),[1670061857.579843134,0, 1670061857.805271279,0),[1670061857.539049140,1, 1670061857.579843134,0),[1670061857.805271279,0, 1670061858.132036208,1),[1670061858.132036208,1, 1670061858.203661134,0),[1670061858.214921323,0, 1670061858.245389934,0),[1670061858.203661134,0, 1670061858.214921323,0),[1670061857.280052462,1, 1670061857.526100177,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061861.898327299,0, 1670061862.064037691,0),[1670061861.805067540,1, 1670061861.898327299,0),[1670061861.636420839,3, 1670061861.789009491,0),[1670061862.064037691,0, 1670061862.113834163,0),[1670061858.711297257,1, 1670061858.786698623,0),[1670061930.742802379,1, 1670061931.230066137,1),[1670061938.140281406,0, 1670061938.393299592,1),[1670061938.393299592,1, 1670061938.527417677,0),[1670061938.527417677,0, 1670061939.281735031,0),[1670061939.281735031,0, 1670061939.303732224,0),[1670061939.303732224,0, 1670061940.051168132,0),[1670061956.060228519,0, 1670061957.319586164,0),[1670061942.839030492,1, 1670061943.991608439,0),[1670061946.085309439,1, 1670061946.381911962,1),[1670061946.381911962,1, 1670061946.487341168,0),[1670061946.487341168,0, 1670061946.857082025,1),[1670061946.857082025,1, 1670061948.261703755,0),[1670061926.603850939,1, 1670061926.920406111,0),[1670061919.456098482,0, 1670061919.651938114,0),[1670061918.985743395,1, 1670061919.456098482,0),[1670061920.465730480,1, 1670061921.057619668,0),[1670061919.661057115,1, 1670061919.796277560,1),[1670061921.187517227,1, 1670061921.515390910,0),[1670061922.197830688,1, 1670061922.960141419,0),[1670061923.488658316,0, 1670061923.522481538,0),[1670061923.109591031,0, 1670061923.488658316,0),[1670061922.960141419,0, 1670061923.109591031,0),[1670061970.532813388,0, 1670061971.155140416,0),[1670061970.169329812,0, 1670061970.532813388,0),[1670061965.735414624,1, 1670061965.975975183,0),[1670061968.926972352,0, 1670061970.169329812,0),[1670061966.654378005,0, 1670061966.953635835,0),[1670061973.300680311,0, 1670061973.520340990,0),[1670061973.520340990,0, 1670061974.034410199,0),[1670061974.034410199,0, 1670061974.266365092,0),[1670061974.266365092,0, 1670061974.388551101,0),[1670061984.836084680,2, 1670061985.377260412,0),[1670061980.679951979,1, 1670061981.984994521,0),[1670061987.759230312,1, 1670061988.314574010,0),[1670062005.745450679,0, 1670062007.007789057,0),[1670061994.576554089,1, 1670061995.563571184,0),[1670061995.563571184,0, 1670061995.563571184,1),[1670061877.963712367,0, 1670061877.985454294,0),[1670061877.985454294,0, 1670061877.988869671,0),[1670061877.865264133,1, 1670061877.963712367,0),[1670061898.212693521,0, 1670061898.298630328,0),[1670061898.019145006,1, 1670061898.193340471,0),[1670061898.193340471,0, 1670061898.212693521,0),[1670061892.371748009,0, 1670061892.741652092,0),[1670061893.119141682,0, 1670061893.119141682,1),[1670061891.461271331,0, 1670061891.614074257,0),[1670061886.454221343,1, 1670061886.583275716,1),[1670061909.847274898,1, 1670061911.258040125,0),[1670061906.449575439,1, 1670061906.927953227,0),[1670061904.437157190,1, 1670061904.543561168,0),[1670061900.915273937,1, 1670061901.135641253,1),[1670061903.134069045,1, 1670061903.165660905,0),[1670061902.632054488,0, 1670061902.752399989,0),[1670061903.298325717,1, 1670061903.501014133,0),[1670061903.845210535,1, 1670061903.863138856,0),[1670061899.369235956,0, 1670061899.583416502,0),[1670061898.790792192,1, 1670061899.369235956,0),[1670061898.593791477,0, 1670061898.612679303,0),[1670061898.612679303,0, 1670061898.780832467,0),[1670061898.780832467,0, 1670061898.790792192,1),[1670061868.474015989,1, 1670061868.748030881,0),[1670062038.938092015,0, 1670062039.191124722,1),[1670062039.191124722,1, 1670062041.574569085,0),[1670062041.574569085,0, 1670062043.743531375,0),[1670062037.571279388,1, 1670062038.938092015,0),[1670062044.357901857,1, 1670062044.519054116,0),[1670062044.519054116,0, 1670062044.641205730,1),[1670062030.288421000,0, 1670062031.842767806,0),[1670062031.842767806,0, 1670062031.861707458,0),[1670062012.642863490,0, 1670062013.190533561,0),[1670062023.095515845,1, 1670062024.549051602,0),[1670062024.549051602,0, 1670062024.818440090,1),[1670062024.818440090,1, 1670062025.739699110,1),[1670062025.777132161,0, 1670062027.299108410,0),[1670062027.299108410,0, 1670062028.138347506,0),[1670062028.138347506,0, 1670062028.450734850,1),[1670062025.739699110,1, 1670062025.777132161,0),[1670062015.135019699,0, 1670062015.441850144,0),[1670062028.450734850,1, 1670062030.288421000,0),[1670062107.983363656,1, 1670062108.175304937,1),[1670062118.810564766,0, 1670062119.051388277,0),[1670062113.812707820,2, 1670062115.318204206,0),[1670062120.118653097,2, 1670062120.993786933,0),[1670062119.954198063,0, 1670062120.118653097,2),[1670062119.784917849,1, 1670062119.954198063,0),[1670062127.141203521,1, 1670062127.210518481,0),[1670062128.962649107,1, 1670062129.453215993,1),[1670062130.671284563,1, 1670062130.857027076,0),[1670062126.614333198,1, 1670062127.141203521,1),[1670062121.898050012,0, 1670062122.611136421,0),[1670062121.601965450,1, 1670062121.898050012,0),[1670062120.993786933,0, 1670062121.022814231,0),[1670062132.865675802,1, 1670062132.921669089,0),[1670062132.481274227,0, 1670062132.497194293,0),[1670062132.497194293,0, 1670062132.865675802,1),[1670062132.082481973,1, 1670062132.481274227,0),[1670062133.493404973,0, 1670062133.503403263,0),[1670062132.921669089,0, 1670062133.493404973,0),[1670062148.494497491,1, 1670062148.902168499,0),[1670062102.285271310,0, 1670062103.310800441,0),[1670062103.310800441,0, 1670062103.427436906,0),[1670062107.813746503,0, 1670062107.983363656,1),[1670062103.427436906,0, 1670062103.888463742,0),[1670062105.913259401,1, 1670062107.813746503,0),[1670062101.975804668,1, 1670062102.086221090,0),[1670062102.086221090,0, 1670062102.285271310,0),[1670062092.565444699,0, 1670062092.883786154,1),[1670062092.883786154,1, 1670062093.032525469,0),[1670062074.339643520,0, 1670062074.810374369,0),[1670062082.362615741,1, 1670062082.960368150,1),[1670062071.352616871,3, 1670062072.170618729,0),[1670062072.606443247,0, 1670062073.089816533,0),[1670062063.093729214,1, 1670062063.169968952,0),[1670062067.439293684,1, 1670062067.753815380,1),[1670062067.753815380,1, 1670062067.843235518,0),[1670062051.852209236,1, 1670062051.907746235,0),[1670062050.052286571,1, 1670062050.162096788,0),[1670062217.830301543,0, 1670062220.071688060,0),[1670062215.129322710,0, 1670062217.830301543,0),[1670062222.628129156,0, 1670062222.741682111,0),[1670062221.537265713,0, 1670062222.628129156,0),[1670062222.741682111,0, 1670062223.920501022,0),[1670062223.920501022,0, 1670062224.458701614,0),[1670062229.070261614,0, 1670062229.924245767,0),[1670062189.943865240,1, 1670062190.505563332,0),[1670062234.523069710,1, 1670062234.721440073,0),[1670062234.721440073,0, 1670062234.776287869,0),[1670062234.776287869,0, 1670062234.987603964,0),[1670062234.994758746,0, 1670062235.062220589,0),[1670062233.534691208,0, 1670062233.714867505,0),[1670062233.984188531,0, 1670062234.523069710,1),[1670062233.714867505,0, 1670062233.984188531,0),[1670062232.870887124,0, 1670062233.250140158,1),[1670062231.400602820,0, 1670062232.870887124,0),[1670062233.250140158,1, 1670062233.534691208,0),[1670062235.139716977,0, 1670062235.346153864,0),[1670062235.359167347,1, 1670062235.432294486,0),[1670062183.756767349,1, 1670062184.221946578,0),[1670062185.131652345,0, 1670062186.874543093,0),[1670062160.149383157,1, 1670062160.498300406,0),[1670062170.721533730,0, 1670062171.123624674,0),[1670062171.123624674,0, 1670062171.339275667,0),[1670062171.339275667,0, 1670062171.871575685,0),[1670062175.534010069,1, 1670062176.763870318,1),[1670062179.190902447,0, 1670062181.045854603,0),[1670062166.800454856,0, 1670062168.676629124,0),[1670062169.791861591,1, 1670062170.721533730,0),[1670062243.880077193,1, 1670062244.050907984,0),[1670062244.050907984,0, 1670062244.121045725,0),[1670062245.576644998,0, 1670062245.806102383,0),[1670062245.806102383,0, 1670062245.838726703,1)}->[/Table/100/"689237fd":v-1150, /Table/100/"6ba4af04":v-1156, /Table/100/"762f36d6":v-1141] [s]/Table/100/"{c2258db0"-fa6796ed"}:{0:[1670062025.294561448,0, 1670062033.557201152,0), 1:[1670062027.694852638,0, 1670062033.557201152,0), 2:[1670062026.442921745,0, 1670062033.557201152,0), 3:[1670062023.834752262,0, 1670062033.557201152,0), 4:[1670062030.669450279,1, 1670062031.615636237,0), 5:[1670062022.523856746,0, 1670062033.557201152,0), 6:[1670062016.004068736,0, 1670062033.557201152,0), 7:[1670062027.778182888,0, 1670062033.557201152,0), 8:[1670062022.427231995,0, 1670062033.557201152,0), 9:[1670062027.946986602,0, 1670062033.557201152,0), 10:[1670062015.949455653,0, 1670062033.557201152,0), 11:[1670062026.535835967,0, 1670062033.557201152,0), 12:[1670062027.946986602,0, 1670062033.557201152,0), 13:[1670062021.252554842,0, 1670062033.557201152,0), 14:[1670062011.503792762,0, 1670062033.557201152,0), 15:[1670062028.138347506,0, 1670062033.557201152,0), 16:[1670062025.739699110,1, 1670062033.557201152,0), 17:[1670062021.478281968,0, 1670062033.557201152,0), 18:[1670062018.714089381,1, 1670062033.557201152,0), 19:[1670062020.958659876,0, 1670062033.557201152,0), 20:[1670062010.228667037,0, 1670062033.557201152,0), 21:[1670062015.949455653,0, 1670062033.557201152,0), gap:[<min>, 1670061857.527400704,0),[1670061857.527400704,0, 1670061857.543795132,0),[1670061898.298630328,1, 1670061898.302651492,1),[1670061898.019145006,1, 1670061898.298630328,1),[1670061898.593791477,0, 1670061899.142558360,0),[1670061899.142558360,0, 1670061899.583416502,0),[1670061902.632054488,0, 1670061902.964382111,0),[1670061892.954452642,0, 1670061897.319170962,0),[1670061902.964382111,0, 1670061903.390296126,0),[1670061904.098578948,0, 1670061904.543561168,0),[1670061904.543561168,0, 1670061904.738052868,0),[1670061904.800372830,0, 1670061906.688957803,0),[1670061904.738052868,0, 1670061904.800372830,0),[1670061878.839971994,0, 1670061879.239230097,0),[1670061880.325994152,0, 1670061880.515396921,0),[1670061879.510442878,0, 1670061880.325994152,0),[1670061879.239230097,0, 1670061879.504240487,1),[1670061879.504240487,1, 1670061879.510442878,0),[1670061881.098200170,1, 1670061881.572780355,0),[1670061858.282412726,0, 1670061858.342143033,0),[1670061858.342143033,0, 1670061858.529722557,0),[1670061858.529722557,0, 1670061858.780406793,0),[1670061919.461603410,0, 1670061919.484080953,0),[1670061919.175621618,0, 1670061919.461603410,0),[1670061918.985743395,1, 1670061919.175621618,0),[1670061921.187517227,1, 1670061921.739484358,0),[1670061930.783439563,0, 1670061931.353584911,0),[1670061933.319601784,2, 1670061934.142780063,1),[1670061933.319601784,1, 1670061933.319601784,2),[1670061935.134675798,0, 1670061935.227223413,0),[1670061935.227223413,0, 1670061935.365746334,0),[1670061935.365746334,0, 1670061936.319818497,2),[1670061936.319818497,2, 1670061936.993652098,1),[1670061938.140281406,0, 1670061938.386068620,2),[1670061942.839030492,1, 1670061943.684757651,0),[1670061943.684757651,0, 1670061945.412426017,1),[1670061923.592948541,0, 1670061923.789042774,1),[1670061922.197830688,1, 1670061922.396625981,0),[1670061923.789042774,1, 1670061924.130255630,0),[1670062225.889127936,0, 1670062226.241222012,0),[1670062234.994758746,0, 1670062235.251011047,0),[1670062235.251011047,0, 1670062235.313920359,0),[1670061966.654378005,0, 1670061967.148871380,0),[1670061967.148871380,0, 1670061967.981009832,0),[1670061978.247164145,1, 1670061979.729025043,2),[1670061979.729025043,2, 1670061980.083617405,0),[1670061984.138387238,0, 1670061984.435129733,1),[1670061984.435129733,1, 1670061985.185525882,0),[1670061976.961017141,0, 1670061977.699208866,0),[1670061976.488359125,1, 1670061976.961017141,0),[1670061977.699208866,0, 1670061978.021777531,1),[1670062005.745450679,0, 1670062006.814489235,0),[1670062036.960834728,2, 1670062037.307043337,1),[1670062007.068296261,0, 1670062008.198199421,0),[1670062006.814489235,0, 1670062007.068296261,0),[1670062011.503792762,0, 1670062011.721937415,0),[1670062028.063558707,0, 1670062028.085776561,1),[1670062023.095515845,2, 1670062024.357462674,0),[1670062021.805583032,0, 1670062023.095515845,2),[1670062018.109476152,0, 1670062021.805583032,0),[1670062017.846194420,1, 1670062018.109476152,0),[1670062016.173656339,0, 1670062017.846194420,1),[1670062015.675674708,1, 1670062016.173656339,0),[1670062024.357462674,0, 1670062026.029387820,0),[1670062026.029387820,0, 1670062028.063558707,0),[1670062028.085776561,1, 1670062028.133244593,0),[1670062028.133244593,0, 1670062028.748172415,0),[1670062028.748172415,0, 1670062030.669450279,2),[1670062033.557201152,0, 1670062035.352138608,0),[1670062035.352138608,0, 1670062036.152633176,0),[1670062036.152633176,0, 1670062036.960834728,2),[1670062074.881269597,0, 1670062076.198379727,0),[1670062074.740140975,0, 1670062074.881269597,0),[1670062074.339643520,0, 1670062074.740140975,0),[1670062038.793115187,0, 1670062038.902779843,0),[1670062041.964166233,1, 1670062042.223504206,0),[1670062042.223504206,0, 1670062042.468721325,1),[1670062044.137423551,0, 1670062044.177946536,0),[1670062043.795415287,0, 1670062044.137423551,0),[1670062044.987639004,0, 1670062045.908465064,0),[1670062044.177946536,0, 1670062044.987639004,0),[1670062045.908465064,0, 1670062046.181141981,0),[1670062038.187131806,1, 1670062038.793115187,0),[1670062037.571279388,1, 1670062038.187131806,1),[1670062096.225576232,1, 1670062097.141146412,1),[1670062098.877380116,0, 1670062099.853580664,0),[1670062107.567953236,1, 1670062108.175304937,0),[1670062101.975804668,1, 1670062102.421370878,0),[1670062099.853580664,0, 1670062099.898142047,0),[1670062099.898142047,0, 1670062100.854145016,0),[1670062130.671284563,1, 1670062131.805964996,0),[1670062131.805964996,0, 1670062132.483731337,1)}->[/Table/100/"c361509a":v-1143, /Table/100/"c42c19df":v-1152, /Table/100/"c4652e27":v-1153, /Table/100/"c6a5917f":v-1142, /Table/100/"cb7a3113":v-1176, /Table/100/"d7fede39":v-1136, /Table/100/"d8cf130f":v-1108, /Table/100/"d9d8c8f5":v-1159, /Table/100/"da7ca4b2":v-1135, /Table/100/"de705bbf":v-1160, /Table/100/"e1bc8050":v-1105, /Table/100/"e8f2cff3":v-1155, /Table/100/"f02eb3af":v-1161, /Table/100/"f0355d18":v-1131, /Table/100/"f1af37c6":v-1087, /Table/100/"f2beaf83":v-1165, /Table/100/"f2c8e52b":v-1145, /Table/100/"f335c7a0":v-1109, /Table/100/"f4d04920":v-1117, /Table/100/"f6c9558b":v-1126, /Table/100/"f75944c4":v-1077, /Table/100/"f775023c":v-1106] [s]/Table/100/"{26a2f0a5"-ba0bc4bd"}:{0:[1670062027.299108410,1, 1670062037.571279388,1), 1:[1670062023.797853404,0, 1670062037.571279388,1), 2:[1670062025.739699110,1, 1670062037.571279388,1), 3:[1670062025.739699110,1, 1670062037.571279388,1), 4:[1670062023.763727993,1, 1670062034.861346539,1), 5:[1670062023.095515845,2, 1670062037.571279388,1), 6:[1670062026.029387820,0, 1670062037.571279388,1), 7:[1670062026.270264388,0, 1670062037.571279388,1), 8:[1670062026.885150437,0, 1670062037.571279388,1), 9:[1670062024.549051602,0, 1670062037.571279388,1), 10:[0,0, 0,0), 11:[1670062028.614136674,0, 1670062033.557201152,0), 12:[1670062020.468897757,0, 1670062033.557201152,0), 13:[1670062016.274660758,1, 1670062031.536365407,0), 14:[1670062023.364641172,0, 1670062033.557201152,0), 15:[1670062028.046115783,0, 1670062033.557201152,0), 16:[1670062028.040824681,0, 1670062033.557201152,0), 17:[1670062021.478281968,0, 1670062033.557201152,0), 18:[1670062027.778182888,0, 1670062033.557201152,0), 19:[1670062016.796065736,0, 1670062033.557201152,0), 20:[1670062028.458494205,0, 1670062033.557201152,0), gap:[<min>, 1670061857.280052462,1),[1670061857.579843134,0, 1670061857.703817830,1),[1670061857.539049140,1, 1670061857.579843134,0),[1670061857.703817830,1, 1670061857.805271279,0),[1670061858.018136853,0, 1670061858.042431195,0),[1670061857.805271279,0, 1670061857.805271279,1),[1670061857.280052462,1, 1670061857.391457975,0),[1670061857.391457975,0, 1670061857.526100177,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061861.636420839,3, 1670061861.697384339,0),[1670061861.697384339,0, 1670061861.789009491,0),[1670061861.898327299,0, 1670061861.912549111,0),[1670061861.805067540,1, 1670061861.898327299,0),[1670062233.534691208,0, 1670062233.642013356,1),[1670062233.250140158,1, 1670062233.534691208,0),[1670062243.880077193,1, 1670062244.050907984,0),[1670062169.978532948,0, 1670062170.003421577,0),[1670062169.791861591,1, 1670062169.978532948,0),[1670061980.837619069,1, 1670061981.402406603,0),[1670061980.679951979,1, 1670061980.837619069,1),[1670061984.836084680,2, 1670061985.005894334,0),[1670061965.735414624,1, 1670061965.975975183,0),[1670061994.576554089,1, 1670061995.116998909,0),[1670062038.387217034,0, 1670062038.511079740,0),[1670062038.511079740,0, 1670062038.667435281,0),[1670062037.571279388,1, 1670062037.934979515,0),[1670062012.642863490,0, 1670062013.190533561,0),[1670062015.135019699,0, 1670062015.441850144,0),[1670062027.946986602,0, 1670062028.138347506,0),[1670062027.863769092,1, 1670062027.946986602,0),[1670062028.138347506,0, 1670062028.450734850,0),[1670062028.450734850,0, 1670062028.450734850,1),[1670062028.450734850,1, 1670062028.632119074,0),[1670062029.543324829,0, 1670062029.915624436,0),[1670062028.632119074,0, 1670062029.543324829,0),[1670062023.095515845,1, 1670062023.095515845,2),[1670062025.805044706,0, 1670062027.299108410,0),[1670062025.777132161,0, 1670062025.805044706,0),[1670062024.818440090,0, 1670062024.818440090,1),[1670062024.818440090,1, 1670062025.739699110,1),[1670062025.739699110,1, 1670062025.777132161,0),[1670062023.095515845,2, 1670062024.549051602,0),[1670062024.549051602,0, 1670062024.818440090,0),[1670062027.299108410,0, 1670062027.863769092,1),[1670062030.288421000,0, 1670062031.536365407,1),[1670062029.915624436,0, 1670062030.288421000,0),[1670062031.536365407,1, 1670062031.842767806,0),[1670062031.842767806,0, 1670062031.848036361,0),[1670062129.231775320,0, 1670062129.453215993,1),[1670062130.671284563,1, 1670062130.857027076,0),[1670062121.601965450,1, 1670062121.725045450,0),[1670062063.093729214,1, 1670062063.169968952,0),[1670062072.606443247,0, 1670062073.089816533,0),[1670062082.362615741,1, 1670062082.960368150,1),[1670062101.975804668,1, 1670062102.086221090,0),[1670062102.086221090,0, 1670062102.145595303,0),[1670062107.983363656,1, 1670062108.175304937,1),[1670062107.813746503,0, 1670062107.983363656,1),[1670062105.913259401,1, 1670062107.006904862,0),[1670062107.567953236,1, 1670062107.813746503,0),[1670061926.603850939,1, 1670061926.841973671,0),[1670061926.841973671,0, 1670061926.920406111,0),[1670061903.298325717,1, 1670061903.501014133,0),[1670061903.134069045,1, 1670061903.137517452,0),[1670061902.632054488,0, 1670061902.752399989,0),[1670061906.449575439,1, 1670061906.659567402,0),[1670061906.659567402,0, 1670061906.688957803,0),[1670061904.437157190,1, 1670061904.495775142,1),[1670061904.495775142,1, 1670061904.543561168,0),[1670061919.507280622,0, 1670061919.651938114,0),[1670061919.456098482,0, 1670061919.507280622,0),[1670061920.903379705,0, 1670061921.057619668,0),[1670061921.187517227,1, 1670061921.515390910,0),[1670061920.465730480,1, 1670061920.903379705,0),[1670061919.661057115,1, 1670061919.796277560,1),[1670061922.197830688,1, 1670061922.585252762,0),[1670061918.985743395,1, 1670061919.456098482,0),[1670061893.119141682,0, 1670061893.119141682,1)}->[/Table/100/"302b4d51":v-1154, /Table/100/"39a78e5c":v-1140, /Table/100/"3c9feae9":v-1147, /Table/100/"44ce1e33":v-1146, /Table/100/"4c452c79":v-1138, /Table/100/"55780ad4":v-1137, /Table/100/"65055b05":v-1148, /Table/100/"689237fd":v-1150, /Table/100/"6ba4af04":v-1156, /Table/100/"762f36d6":v-1141, /Table/100/"7f0359c7":v-1177, /Table/100/"a7e441e2":v-1167, /Table/100/"a8bb4f76":v-1128, /Table/100/"aa5339ff":v-1113, /Table/100/"aab792b5":v-1139, /Table/100/"aae5a0e8":v-1163, /Table/100/"ab90f38a":v-1162, /Table/100/"ad3ed452":v-1110, /Table/100/"b08d822f":v-1158, /Table/100/"b131faff":v-1118, /Table/100/"b326a4b5":v-1166] [w]/Table/100/"7f0359c7":1670062030.669450279,1->v-1177 [d]/Table/100/"95ff4a51":1670062030.669450279,1-><nil>
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [w]/Table/100/"cb7a3113":1670062030.669450279,1->v-1176 [s]/Table/100/"{65c851c0"-9e1c550a"}:{0:[1670062026.270264388,0, 1670062037.571279388,1), 1:[1670062026.885150437,0, 1670062037.571279388,1), 2:[1670062024.549051602,0, 1670062037.571279388,1), gap:[<min>, 1670061857.280052462,1),[1670061857.579843134,0, 1670061857.805271279,0),[1670061857.539049140,1, 1670061857.579843134,0),[1670061857.805271279,0, 1670061858.132036208,1),[1670061858.132036208,1, 1670061858.203661134,0),[1670061858.214921323,0, 1670061858.245389934,0),[1670061858.203661134,0, 1670061858.214921323,0),[1670061857.280052462,1, 1670061857.526100177,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061861.898327299,0, 1670061862.064037691,0),[1670061861.805067540,1, 1670061861.898327299,0),[1670061861.636420839,3, 1670061861.789009491,0),[1670061862.064037691,0, 1670061862.113834163,0),[1670061858.711297257,1, 1670061858.786698623,0),[1670061930.742802379,1, 1670061931.230066137,1),[1670061938.140281406,0, 1670061938.393299592,1),[1670061938.393299592,1, 1670061938.527417677,0),[1670061938.527417677,0, 1670061939.281735031,0),[1670061939.281735031,0, 1670061939.303732224,0),[1670061939.303732224,0, 1670061940.051168132,0),[1670061956.060228519,0, 1670061957.319586164,0),[1670061942.839030492,1, 1670061943.991608439,0),[1670061946.085309439,1, 1670061946.381911962,1),[1670061946.381911962,1, 1670061946.487341168,0),[1670061946.487341168,0, 1670061946.857082025,1),[1670061946.857082025,1, 1670061948.261703755,0),[1670061926.603850939,1, 1670061926.920406111,0),[1670061919.456098482,0, 1670061919.651938114,0),[1670061918.985743395,1, 1670061919.456098482,0),[1670061920.465730480,1, 1670061921.057619668,0),[1670061919.661057115,1, 1670061919.796277560,1),[1670061921.187517227,1, 1670061921.515390910,0),[1670061922.197830688,1, 1670061922.960141419,0),[1670061923.488658316,0, 1670061923.522481538,0),[1670061923.109591031,0, 1670061923.488658316,0),[1670061922.960141419,0, 1670061923.109591031,0),[1670061970.532813388,0, 1670061971.155140416,0),[1670061970.169329812,0, 1670061970.532813388,0),[1670061965.735414624,1, 1670061965.975975183,0),[1670061968.926972352,0, 1670061970.169329812,0),[1670061966.654378005,0, 1670061966.953635835,0),[1670061973.300680311,0, 1670061973.520340990,0),[1670061973.520340990,0, 1670061974.034410199,0),[1670061974.034410199,0, 1670061974.266365092,0),[1670061974.266365092,0, 1670061974.388551101,0),[1670061984.836084680,2, 1670061985.377260412,0),[1670061980.679951979,1, 1670061981.984994521,0),[1670061987.759230312,1, 1670061988.314574010,0),[1670062005.745450679,0, 1670062007.007789057,0),[1670061994.576554089,1, 1670061995.563571184,0),[1670061995.563571184,0, 1670061995.563571184,1),[1670061877.963712367,0, 1670061877.985454294,0),[1670061877.985454294,0, 1670061877.988869671,0),[1670061877.865264133,1, 1670061877.963712367,0),[1670061898.212693521,0, 1670061898.298630328,0),[1670061898.019145006,1, 1670061898.193340471,0),[1670061898.193340471,0, 1670061898.212693521,0),[1670061892.371748009,0, 1670061892.741652092,0),[1670061893.119141682,0, 1670061893.119141682,1),[1670061891.461271331,0, 1670061891.614074257,0),[1670061886.454221343,1, 1670061886.583275716,1),[1670061909.847274898,1, 1670061911.258040125,0),[1670061906.449575439,1, 1670061906.927953227,0),[1670061904.437157190,1, 1670061904.543561168,0),[1670061900.915273937,1, 1670061901.135641253,1),[1670061903.134069045,1, 1670061903.165660905,0),[1670061902.632054488,0, 1670061902.752399989,0),[1670061903.298325717,1, 1670061903.501014133,0),[1670061903.845210535,1, 1670061903.863138856,0),[1670061899.369235956,0, 1670061899.583416502,0),[1670061898.790792192,1, 1670061899.369235956,0),[1670061898.593791477,0, 1670061898.612679303,0),[1670061898.612679303,0, 1670061898.780832467,0),[1670061898.780832467,0, 1670061898.790792192,1),[1670061868.474015989,1, 1670061868.748030881,0),[1670062038.938092015,0, 1670062039.191124722,1),[1670062039.191124722,1, 1670062041.574569085,0),[1670062041.574569085,0, 1670062043.743531375,0),[1670062037.571279388,1, 1670062038.938092015,0),[1670062044.357901857,1, 1670062044.519054116,0),[1670062044.519054116,0, 1670062044.641205730,1),[1670062030.288421000,0, 1670062031.842767806,0),[1670062031.842767806,0, 1670062031.861707458,0),[1670062012.642863490,0, 1670062013.190533561,0),[1670062023.095515845,1, 1670062024.549051602,0),[1670062024.549051602,0, 1670062024.818440090,1),[1670062024.818440090,1, 1670062025.739699110,1),[1670062025.777132161,0, 1670062027.299108410,0),[1670062027.299108410,0, 1670062028.138347506,0),[1670062028.138347506,0, 1670062028.450734850,1),[1670062025.739699110,1, 1670062025.777132161,0),[1670062015.135019699,0, 1670062015.441850144,0),[1670062028.450734850,1, 1670062030.288421000,0),[1670062107.983363656,1, 1670062108.175304937,1),[1670062118.810564766,0, 1670062119.051388277,0),[1670062113.812707820,2, 1670062115.318204206,0),[1670062120.118653097,2, 1670062120.993786933,0),[1670062119.954198063,0, 1670062120.118653097,2),[1670062119.784917849,1, 1670062119.954198063,0),[1670062127.141203521,1, 1670062127.210518481,0),[1670062128.962649107,1, 1670062129.453215993,1),[1670062130.671284563,1, 1670062130.857027076,0),[1670062126.614333198,1, 1670062127.141203521,1),[1670062121.898050012,0, 1670062122.611136421,0),[1670062121.601965450,1, 1670062121.898050012,0),[1670062120.993786933,0, 1670062121.022814231,0),[1670062132.865675802,1, 1670062132.921669089,0),[1670062132.481274227,0, 1670062132.497194293,0),[1670062132.497194293,0, 1670062132.865675802,1),[1670062132.082481973,1, 1670062132.481274227,0),[1670062133.493404973,0, 1670062133.503403263,0),[1670062132.921669089,0, 1670062133.493404973,0),[1670062148.494497491,1, 1670062148.902168499,0),[1670062102.285271310,0, 1670062103.310800441,0),[1670062103.310800441,0, 1670062103.427436906,0),[1670062107.813746503,0, 1670062107.983363656,1),[1670062103.427436906,0, 1670062103.888463742,0),[1670062105.913259401,1, 1670062107.813746503,0),[1670062101.975804668,1, 1670062102.086221090,0),[1670062102.086221090,0, 1670062102.285271310,0),[1670062092.565444699,0, 1670062092.883786154,1),[1670062092.883786154,1, 1670062093.032525469,0),[1670062074.339643520,0, 1670062074.810374369,0),[1670062082.362615741,1, 1670062082.960368150,1),[1670062071.352616871,3, 1670062072.170618729,0),[1670062072.606443247,0, 1670062073.089816533,0),[1670062063.093729214,1, 1670062063.169968952,0),[1670062067.439293684,1, 1670062067.753815380,1),[1670062067.753815380,1, 1670062067.843235518,0),[1670062051.852209236,1, 1670062051.907746235,0),[1670062050.052286571,1, 1670062050.162096788,0),[1670062217.830301543,0, 1670062220.071688060,0),[1670062215.129322710,0, 1670062217.830301543,0),[1670062222.628129156,0, 1670062222.741682111,0),[1670062221.537265713,0, 1670062222.628129156,0),[1670062222.741682111,0, 1670062223.920501022,0),[1670062223.920501022,0, 1670062224.458701614,0),[1670062229.070261614,0, 1670062229.924245767,0),[1670062189.943865240,1, 1670062190.505563332,0),[1670062234.523069710,1, 1670062234.721440073,0),[1670062234.721440073,0, 1670062234.776287869,0),[1670062234.776287869,0, 1670062234.987603964,0),[1670062234.994758746,0, 1670062235.062220589,0),[1670062233.534691208,0, 1670062233.714867505,0),[1670062233.984188531,0, 1670062234.523069710,1),[1670062233.714867505,0, 1670062233.984188531,0),[1670062232.870887124,0, 1670062233.250140158,1),[1670062231.400602820,0, 1670062232.870887124,0),[1670062233.250140158,1, 1670062233.534691208,0),[1670062235.139716977,0, 1670062235.346153864,0),[1670062235.359167347,1, 1670062235.432294486,0),[1670062183.756767349,1, 1670062184.221946578,0),[1670062185.131652345,0, 1670062186.874543093,0),[1670062160.149383157,1, 1670062160.498300406,0),[1670062170.721533730,0, 1670062171.123624674,0),[1670062171.123624674,0, 1670062171.339275667,0),[1670062171.339275667,0, 1670062171.871575685,0),[1670062175.534010069,1, 1670062176.763870318,1),[1670062179.190902447,0, 1670062181.045854603,0),[1670062166.800454856,0, 1670062168.676629124,0),[1670062169.791861591,1, 1670062170.721533730,0),[1670062243.880077193,1, 1670062244.050907984,0),[1670062244.050907984,0, 1670062244.121045725,0),[1670062245.576644998,0, 1670062245.806102383,0),[1670062245.806102383,0, 1670062245.838726703,1)}->[/Table/100/"689237fd":v-1150, /Table/100/"6ba4af04":v-1156, /Table/100/"762f36d6":v-1141] [s]/Table/100/"{c2258db0"-fa6796ed"}:{0:[1670062025.294561448,0, 1670062033.557201152,0), 1:[1670062027.694852638,0, 1670062033.557201152,0), 2:[1670062026.442921745,0, 1670062033.557201152,0), 3:[1670062023.834752262,0, 1670062033.557201152,0), 4:[1670062030.669450279,1, 1670062031.615636237,0), 5:[1670062022.523856746,0, 1670062033.557201152,0), 6:[1670062016.004068736,0, 1670062033.557201152,0), 7:[1670062027.778182888,0, 1670062033.557201152,0), 8:[1670062022.427231995,0, 1670062033.557201152,0), 9:[1670062027.946986602,0, 1670062033.557201152,0), 10:[1670062015.949455653,0, 1670062033.557201152,0), 11:[1670062026.535835967,0, 1670062033.557201152,0), 12:[1670062027.946986602,0, 1670062033.557201152,0), 13:[1670062021.252554842,0, 1670062033.557201152,0), 14:[1670062011.503792762,0, 1670062033.557201152,0), 15:[1670062028.138347506,0, 1670062033.557201152,0), 16:[1670062025.739699110,1, 1670062033.557201152,0), 17:[1670062021.478281968,0, 1670062033.557201152,0), 18:[1670062018.714089381,1, 1670062033.557201152,0), 19:[1670062020.958659876,0, 1670062033.557201152,0), 20:[1670062010.228667037,0, 1670062033.557201152,0), 21:[1670062015.949455653,0, 1670062033.557201152,0), gap:[<min>, 1670061857.527400704,0),[1670061857.527400704,0, 1670061857.543795132,0),[1670061898.298630328,1, 1670061898.302651492,1),[1670061898.019145006,1, 1670061898.298630328,1),[1670061898.593791477,0, 1670061899.142558360,0),[1670061899.142558360,0, 1670061899.583416502,0),[1670061902.632054488,0, 1670061902.964382111,0),[1670061892.954452642,0, 1670061897.319170962,0),[1670061902.964382111,0, 1670061903.390296126,0),[1670061904.098578948,0, 1670061904.543561168,0),[1670061904.543561168,0, 1670061904.738052868,0),[1670061904.800372830,0, 1670061906.688957803,0),[1670061904.738052868,0, 1670061904.800372830,0),[1670061878.839971994,0, 1670061879.239230097,0),[1670061880.325994152,0, 1670061880.515396921,0),[1670061879.510442878,0, 1670061880.325994152,0),[1670061879.239230097,0, 1670061879.504240487,1),[1670061879.504240487,1, 1670061879.510442878,0),[1670061881.098200170,1, 1670061881.572780355,0),[1670061858.282412726,0, 1670061858.342143033,0),[1670061858.342143033,0, 1670061858.529722557,0),[1670061858.529722557,0, 1670061858.780406793,0),[1670061919.461603410,0, 1670061919.484080953,0),[1670061919.175621618,0, 1670061919.461603410,0),[1670061918.985743395,1, 1670061919.175621618,0),[1670061921.187517227,1, 1670061921.739484358,0),[1670061930.783439563,0, 1670061931.353584911,0),[1670061933.319601784,2, 1670061934.142780063,1),[1670061933.319601784,1, 1670061933.319601784,2),[1670061935.134675798,0, 1670061935.227223413,0),[1670061935.227223413,0, 1670061935.365746334,0),[1670061935.365746334,0, 1670061936.319818497,2),[1670061936.319818497,2, 1670061936.993652098,1),[1670061938.140281406,0, 1670061938.386068620,2),[1670061942.839030492,1, 1670061943.684757651,0),[1670061943.684757651,0, 1670061945.412426017,1),[1670061923.592948541,0, 1670061923.789042774,1),[1670061922.197830688,1, 1670061922.396625981,0),[1670061923.789042774,1, 1670061924.130255630,0),[1670062225.889127936,0, 1670062226.241222012,0),[1670062234.994758746,0, 1670062235.251011047,0),[1670062235.251011047,0, 1670062235.313920359,0),[1670061966.654378005,0, 1670061967.148871380,0),[1670061967.148871380,0, 1670061967.981009832,0),[1670061978.247164145,1, 1670061979.729025043,2),[1670061979.729025043,2, 1670061980.083617405,0),[1670061984.138387238,0, 1670061984.435129733,1),[1670061984.435129733,1, 1670061985.185525882,0),[1670061976.961017141,0, 1670061977.699208866,0),[1670061976.488359125,1, 1670061976.961017141,0),[1670061977.699208866,0, 1670061978.021777531,1),[1670062005.745450679,0, 1670062006.814489235,0),[1670062036.960834728,2, 1670062037.307043337,1),[1670062007.068296261,0, 1670062008.198199421,0),[1670062006.814489235,0, 1670062007.068296261,0),[1670062011.503792762,0, 1670062011.721937415,0),[1670062028.063558707,0, 1670062028.085776561,1),[1670062023.095515845,2, 1670062024.357462674,0),[1670062021.805583032,0, 1670062023.095515845,2),[1670062018.109476152,0, 1670062021.805583032,0),[1670062017.846194420,1, 1670062018.109476152,0),[1670062016.173656339,0, 1670062017.846194420,1),[1670062015.675674708,1, 1670062016.173656339,0),[1670062024.357462674,0, 1670062026.029387820,0),[1670062026.029387820,0, 1670062028.063558707,0),[1670062028.085776561,1, 1670062028.133244593,0),[1670062028.133244593,0, 1670062028.748172415,0),[1670062028.748172415,0, 1670062030.669450279,2),[1670062033.557201152,0, 1670062035.352138608,0),[1670062035.352138608,0, 1670062036.152633176,0),[1670062036.152633176,0, 1670062036.960834728,2),[1670062074.881269597,0, 1670062076.198379727,0),[1670062074.740140975,0, 1670062074.881269597,0),[1670062074.339643520,0, 1670062074.740140975,0),[1670062038.793115187,0, 1670062038.902779843,0),[1670062041.964166233,1, 1670062042.223504206,0),[1670062042.223504206,0, 1670062042.468721325,1),[1670062044.137423551,0, 1670062044.177946536,0),[1670062043.795415287,0, 1670062044.137423551,0),[1670062044.987639004,0, 1670062045.908465064,0),[1670062044.177946536,0, 1670062044.987639004,0),[1670062045.908465064,0, 1670062046.181141981,0),[1670062038.187131806,1, 1670062038.793115187,0),[1670062037.571279388,1, 1670062038.187131806,1),[1670062096.225576232,1, 1670062097.141146412,1),[1670062098.877380116,0, 1670062099.853580664,0),[1670062107.567953236,1, 1670062108.175304937,0),[1670062101.975804668,1, 1670062102.421370878,0),[1670062099.853580664,0, 1670062099.898142047,0),[1670062099.898142047,0, 1670062100.854145016,0),[1670062130.671284563,1, 1670062131.805964996,0),[1670062131.805964996,0, 1670062132.483731337,1)}->[/Table/100/"c361509a":v-1143, /Table/100/"c42c19df":v-1152, /Table/100/"c4652e27":v-1153, /Table/100/"c6a5917f":v-1142, /Table/100/"cb7a3113":v-1176, /Table/100/"d7fede39":v-1136, /Table/100/"d8cf130f":v-1108, /Table/100/"d9d8c8f5":v-1159, /Table/100/"da7ca4b2":v-1135, /Table/100/"de705bbf":v-1160, /Table/100/"e1bc8050":v-1105, /Table/100/"e8f2cff3":v-1155, /Table/100/"f02eb3af":v-1161, /Table/100/"f0355d18":v-1131, /Table/100/"f1af37c6":v-1087, /Table/100/"f2beaf83":v-1165, /Table/100/"f2c8e52b":v-1145, /Table/100/"f335c7a0":v-1109, /Table/100/"f4d04920":v-1117, /Table/100/"f6c9558b":v-1126, /Table/100/"f75944c4":v-1077, /Table/100/"f775023c":v-1106] [s]/Table/100/"{26a2f0a5"-ba0bc4bd"}:{0:[1670062027.299108410,1, 1670062037.571279388,1), 1:[1670062023.797853404,0, 1670062037.571279388,1), 2:[1670062025.739699110,1, 1670062037.571279388,1), 3:[1670062025.739699110,1, 1670062037.571279388,1), 4:[1670062023.763727993,1, 1670062034.861346539,1), 5:[1670062023.095515845,2, 1670062037.571279388,1), 6:[1670062026.029387820,0, 1670062037.571279388,1), 7:[1670062026.270264388,0, 1670062037.571279388,1), 8:[1670062026.885150437,0, 1670062037.571279388,1), 9:[1670062024.549051602,0, 1670062037.571279388,1), 10:[0,0, 0,0), 11:[1670062028.614136674,0, 1670062033.557201152,0), 12:[1670062020.468897757,0, 1670062033.557201152,0), 13:[1670062016.274660758,1, 1670062031.536365407,0), 14:[1670062023.364641172,0, 1670062033.557201152,0), 15:[1670062028.046115783,0, 1670062033.557201152,0), 16:[1670062028.040824681,0, 1670062033.557201152,0), 17:[1670062021.478281968,0, 1670062033.557201152,0), 18:[1670062027.778182888,0, 1670062033.557201152,0), 19:[1670062016.796065736,0, 1670062033.557201152,0), 20:[1670062028.458494205,0, 1670062033.557201152,0), gap:[<min>, 1670061857.280052462,1),[1670061857.579843134,0, 1670061857.703817830,1),[1670061857.539049140,1, 1670061857.579843134,0),[1670061857.703817830,1, 1670061857.805271279,0),[1670061858.018136853,0, 1670061858.042431195,0),[1670061857.805271279,0, 1670061857.805271279,1),[1670061857.280052462,1, 1670061857.391457975,0),[1670061857.391457975,0, 1670061857.526100177,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061861.636420839,3, 1670061861.697384339,0),[1670061861.697384339,0, 1670061861.789009491,0),[1670061861.898327299,0, 1670061861.912549111,0),[1670061861.805067540,1, 1670061861.898327299,0),[1670062233.534691208,0, 1670062233.642013356,1),[1670062233.250140158,1, 1670062233.534691208,0),[1670062243.880077193,1, 1670062244.050907984,0),[1670062169.978532948,0, 1670062170.003421577,0),[1670062169.791861591,1, 1670062169.978532948,0),[1670061980.837619069,1, 1670061981.402406603,0),[1670061980.679951979,1, 1670061980.837619069,1),[1670061984.836084680,2, 1670061985.005894334,0),[1670061965.735414624,1, 1670061965.975975183,0),[1670061994.576554089,1, 1670061995.116998909,0),[1670062038.387217034,0, 1670062038.511079740,0),[1670062038.511079740,0, 1670062038.667435281,0),[1670062037.571279388,1, 1670062037.934979515,0),[1670062012.642863490,0, 1670062013.190533561,0),[1670062015.135019699,0, 1670062015.441850144,0),[1670062027.946986602,0, 1670062028.138347506,0),[1670062027.863769092,1, 1670062027.946986602,0),[1670062028.138347506,0, 1670062028.450734850,0),[1670062028.450734850,0, 1670062028.450734850,1),[1670062028.450734850,1, 1670062028.632119074,0),[1670062029.543324829,0, 1670062029.915624436,0),[1670062028.632119074,0, 1670062029.543324829,0),[1670062023.095515845,1, 1670062023.095515845,2),[1670062025.805044706,0, 1670062027.299108410,0),[1670062025.777132161,0, 1670062025.805044706,0),[1670062024.818440090,0, 1670062024.818440090,1),[1670062024.818440090,1, 1670062025.739699110,1),[1670062025.739699110,1, 1670062025.777132161,0),[1670062023.095515845,2, 1670062024.549051602,0),[1670062024.549051602,0, 1670062024.818440090,0),[1670062027.299108410,0, 1670062027.863769092,1),[1670062030.288421000,0, 1670062031.536365407,1),[1670062029.915624436,0, 1670062030.288421000,0),[1670062031.536365407,1, 1670062031.842767806,0),[1670062031.842767806,0, 1670062031.848036361,0),[1670062129.231775320,0, 1670062129.453215993,1),[1670062130.671284563,1, 1670062130.857027076,0),[1670062121.601965450,1, 1670062121.725045450,0),[1670062063.093729214,1, 1670062063.169968952,0),[1670062072.606443247,0, 1670062073.089816533,0),[1670062082.362615741,1, 1670062082.960368150,1),[1670062101.975804668,1, 1670062102.086221090,0),[1670062102.086221090,0, 1670062102.145595303,0),[1670062107.983363656,1, 1670062108.175304937,1),[1670062107.813746503,0, 1670062107.983363656,1),[1670062105.913259401,1, 1670062107.006904862,0),[1670062107.567953236,1, 1670062107.813746503,0),[1670061926.603850939,1, 1670061926.841973671,0),[1670061926.841973671,0, 1670061926.920406111,0),[1670061903.298325717,1, 1670061903.501014133,0),[1670061903.134069045,1, 1670061903.137517452,0),[1670061902.632054488,0, 1670061902.752399989,0),[1670061906.449575439,1, 1670061906.659567402,0),[1670061906.659567402,0, 1670061906.688957803,0),[1670061904.437157190,1, 1670061904.495775142,1),[1670061904.495775142,1, 1670061904.543561168,0),[1670061919.507280622,0, 1670061919.651938114,0),[1670061919.456098482,0, 1670061919.507280622,0),[1670061920.903379705,0, 1670061921.057619668,0),[1670061921.187517227,1, 1670061921.515390910,0),[1670061920.465730480,1, 1670061920.903379705,0),[1670061919.661057115,1, 1670061919.796277560,1),[1670061922.197830688,1, 1670061922.585252762,0),[1670061918.985743395,1, 1670061919.456098482,0),[1670061893.119141682,0, 1670061893.119141682,1)}->[/Table/100/"302b4d51":v-1154, /Table/100/"39a78e5c":v-1140, /Table/100/"3c9feae9":v-1147, /Table/100/"44ce1e33":v-1146, /Table/100/"4c452c79":v-1138, /Table/100/"55780ad4":v-1137, /Table/100/"65055b05":v-1148, /Table/100/"689237fd":v-1150, /Table/100/"6ba4af04":v-1156, /Table/100/"762f36d6":v-1141, /Table/100/"7f0359c7":v-1177, /Table/100/"a7e441e2":v-1167, /Table/100/"a8bb4f76":v-1128, /Table/100/"aa5339ff":v-1113, /Table/100/"aab792b5":v-1139, /Table/100/"aae5a0e8":v-1163, /Table/100/"ab90f38a":v-1162, /Table/100/"ad3ed452":v-1110, /Table/100/"b08d822f":v-1158, /Table/100/"b131faff":v-1118, /Table/100/"b326a4b5":v-1166] [w]/Table/100/"7f0359c7":1670062030.669450279,1->v-1177 [d]/Table/100/"95ff4a51":1670062030.669450279,1-><nil>
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{19a353c2"-6e9037b3"}:{0:[1670062172.943229979,0, 1670062197.773220257,1), 1:[1670062171.681702909,0, 1670062179.190902447,0), 2:[1670062172.575995711,1, 1670062179.190902447,0), 3:[1670062170.871425910,0, 1670062179.190902447,0), 4:[0,0, 0,0), 5:[1670062173.018937226,1, 1670062179.190902447,0), gap:[<min>, 1670061857.526100177,0),[1670061857.579843134,0, 1670061857.703817830,1),[1670061857.526100177,0, 1670061857.579843134,0),[1670061857.805271279,1, 1670061858.042431195,0),[1670061857.703817830,1, 1670061857.805271279,1),[1670061858.711297257,1, 1670061858.780406793,0),[1670061861.636420839,3, 1670061861.789009491,0),[1670061861.789009491,0, 1670061861.912549111,0),[1670061867.082673187,1, 1670061867.329977708,0),[1670061868.285898367,0, 1670061868.452025248,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061880.886871550,1, 1670061880.969935355,0),[1670061880.969935355,0, 1670061881.391953786,0),[1670061903.134069045,1, 1670061903.137517452,0),[1670061902.632054488,0, 1670061902.819772568,0),[1670061904.437157190,1, 1670061904.495775142,1),[1670061904.495775142,1, 1670061904.543561168,0),[1670061906.449575439,1, 1670061906.659567402,0),[1670061906.659567402,0, 1670061906.688957803,0),[1670061910.135208459,1, 1670061910.450860376,1),[1670061909.847274898,1, 1670061910.135208459,1),[1670061897.057079348,0, 1670061897.523983671,0),[1670061924.480447740,0, 1670061925.482362313,0),[1670061925.482362313,0, 1670061926.395463236,1),[1670061926.395463236,1, 1670061926.603850939,1),[1670061926.899129144,0, 1670061926.920406111,0),[1670061926.603850939,1, 1670061926.899129144,0),[1670061920.465730480,1, 1670061922.017686826,0),[1670061922.017686826,0, 1670061922.396625981,0),[1670061919.507280622,0, 1670061919.796277560,1),[1670061918.985743395,1, 1670061919.507280622,0),[1670061930.742802379,1, 1670061930.745367785,0),[1670061930.745367785,0, 1670061930.783439563,0),[1670062063.396786407,0, 1670062063.716456095,0),[1670062063.093729214,1, 1670062063.396786407,0),[1670062038.387217034,0, 1670062038.511079740,0),[1670062038.511079740,0, 1670062038.667435281,0),[1670062037.571279388,1, 1670062037.934979515,0),[1670062067.439293684,1, 1670062067.602548520,0),[1670062067.852619666,0, 1670062067.910588416,1),[1670062067.753815380,0, 1670062067.852619666,0),[1670062068.335762419,0, 1670062068.338617766,0),[1670062068.321134676,0, 1670062068.335762419,0),[1670062172.575995711,1, 1670062173.573429087,0),[1670062169.978532948,0, 1670062170.027071831,0),[1670062169.791861591,1, 1670062169.978532948,0),[1670062170.027071831,0, 1670062170.831879782,0),[1670062170.831879782,0, 1670062171.681702909,0),[1670062171.681702909,0, 1670062172.575995711,1),[1670062215.129322710,0, 1670062215.512312822,0),[1670062212.924892675,0, 1670062213.167393159,0),[1670062233.250140158,1, 1670062233.642013356,1),[1670062243.880077193,1, 1670062244.050907984,0),[1670062105.913259401,1, 1670062107.006904862,0),[1670062107.567953236,1, 1670062107.769284736,0),[1670062101.975804668,1, 1670062102.955708634,0),[1670062102.955708634,0, 1670062103.278238671,0),[1670062121.601965450,1, 1670062121.725045450,0),[1670062122.693435096,0, 1670062124.046173714,1),[1670062150.827006808,0, 1670062151.103697633,0),[1670062156.527714490,1, 1670062157.493794987,1),[1670062157.493794987,1, 1670062157.582329744,0),[1670062157.582329744,0, 1670062157.614886892,0),[1670062130.857027076,0, 1670062131.779630742,0),[1670062130.220846510,0, 1670062130.857027076,0),[1670062128.962649107,1, 1670062130.220846510,0),[1670062131.779630742,0, 1670062131.840320318,0),[1670062079.229023108,0, 1670062079.568069929,0),[1670062076.840652671,0, 1670062076.936558056,0),[1670062076.936558056,0, 1670062076.946171755,0),[1670062076.946171755,0, 1670062077.646807606,1),[1670061994.576554089,1, 1670061994.628007753,0),[1670062012.642863490,0, 1670062014.050134337,0),[1670062014.050134337,0, 1670062014.140564372,0),[1670062015.675674708,0, 1670062015.675674708,1),[1670062014.140564372,0, 1670062015.675674708,0),[1670062020.468897757,1, 1670062020.958659876,0),[1670061946.940790159,1, 1670061948.161604565,0),[1670061949.161640621,0, 1670061949.544345551,1),[1670061965.163736664,2, 1670061965.735414624,0),[1670061967.005907286,1, 1670061967.098039088,0),[1670061967.098039088,0, 1670061967.386977852,0),[1670061967.386977852,0, 1670061967.483638851,0),[1670061965.735414624,0, 1670061966.078881424,0),[1670061969.370102355,1, 1670061969.732256629,0),[1670061968.926972352,0, 1670061969.370102355,1),[1670061970.573993336,0, 1670061970.674755571,0),[1670061969.732315458,0, 1670061969.765337622,0),[1670061970.858068303,0, 1670061970.864998640,0),[1670061970.674755571,0, 1670061970.858068303,0),[1670061980.679951979,1, 1670061981.057410039,0),[1670061984.836084680,2, 1670061985.325399152,0)}->[/Table/100/"20bfd42a":v-1881, /Table/100/"2ae1aba8":v-1877, /Table/100/"3570d962":v-1878, /Table/100/"375aee0f":v-1873, /Table/100/"465ecf3b":v-1880, /Table/100/"479be315":v-1876] [w]/Table/100/"465ecf3b":1670062173.338070704,1->v-1880 [d]/Table/100/"ba728422":1670062173.338070704,1-><nil>
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{19a353c2"-6e9037b3"}:{0:[1670062172.943229979,0, 1670062197.773220257,1), 1:[1670062171.681702909,0, 1670062179.190902447,0), 2:[1670062172.575995711,1, 1670062179.190902447,0), 3:[1670062170.871425910,0, 1670062179.190902447,0), 4:[0,0, 0,0), 5:[1670062173.018937226,1, 1670062179.190902447,0), gap:[<min>, 1670061857.526100177,0),[1670061857.579843134,0, 1670061857.703817830,1),[1670061857.526100177,0, 1670061857.579843134,0),[1670061857.805271279,1, 1670061858.042431195,0),[1670061857.703817830,1, 1670061857.805271279,1),[1670061858.711297257,1, 1670061858.780406793,0),[1670061861.636420839,3, 1670061861.789009491,0),[1670061861.789009491,0, 1670061861.912549111,0),[1670061867.082673187,1, 1670061867.329977708,0),[1670061868.285898367,0, 1670061868.452025248,0),[1670061861.636420839,1, 1670061861.636420839,3),[1670061880.886871550,1, 1670061880.969935355,0),[1670061880.969935355,0, 1670061881.391953786,0),[1670061903.134069045,1, 1670061903.137517452,0),[1670061902.632054488,0, 1670061902.819772568,0),[1670061904.437157190,1, 1670061904.495775142,1),[1670061904.495775142,1, 1670061904.543561168,0),[1670061906.449575439,1, 1670061906.659567402,0),[1670061906.659567402,0, 1670061906.688957803,0),[1670061910.135208459,1, 1670061910.450860376,1),[1670061909.847274898,1, 1670061910.135208459,1),[1670061897.057079348,0, 1670061897.523983671,0),[1670061924.480447740,0, 1670061925.482362313,0),[1670061925.482362313,0, 1670061926.395463236,1),[1670061926.395463236,1, 1670061926.603850939,1),[1670061926.899129144,0, 1670061926.920406111,0),[1670061926.603850939,1, 1670061926.899129144,0),[1670061920.465730480,1, 1670061922.017686826,0),[1670061922.017686826,0, 1670061922.396625981,0),[1670061919.507280622,0, 1670061919.796277560,1),[1670061918.985743395,1, 1670061919.507280622,0),[1670061930.742802379,1, 1670061930.745367785,0),[1670061930.745367785,0, 1670061930.783439563,0),[1670062063.396786407,0, 1670062063.716456095,0),[1670062063.093729214,1, 1670062063.396786407,0),[1670062038.387217034,0, 1670062038.511079740,0),[1670062038.511079740,0, 1670062038.667435281,0),[1670062037.571279388,1, 1670062037.934979515,0),[1670062067.439293684,1, 1670062067.602548520,0),[1670062067.852619666,0, 1670062067.910588416,1),[1670062067.753815380,0, 1670062067.852619666,0),[1670062068.335762419,0, 1670062068.338617766,0),[1670062068.321134676,0, 1670062068.335762419,0),[1670062172.575995711,1, 1670062173.573429087,0),[1670062169.978532948,0, 1670062170.027071831,0),[1670062169.791861591,1, 1670062169.978532948,0),[1670062170.027071831,0, 1670062170.831879782,0),[1670062170.831879782,0, 1670062171.681702909,0),[1670062171.681702909,0, 1670062172.575995711,1),[1670062215.129322710,0, 1670062215.512312822,0),[1670062212.924892675,0, 1670062213.167393159,0),[1670062233.250140158,1, 1670062233.642013356,1),[1670062243.880077193,1, 1670062244.050907984,0),[1670062105.913259401,1, 1670062107.006904862,0),[1670062107.567953236,1, 1670062107.769284736,0),[1670062101.975804668,1, 1670062102.955708634,0),[1670062102.955708634,0, 1670062103.278238671,0),[1670062121.601965450,1, 1670062121.725045450,0),[1670062122.693435096,0, 1670062124.046173714,1),[1670062150.827006808,0, 1670062151.103697633,0),[1670062156.527714490,1, 1670062157.493794987,1),[1670062157.493794987,1, 1670062157.582329744,0),[1670062157.582329744,0, 1670062157.614886892,0),[1670062130.857027076,0, 1670062131.779630742,0),[1670062130.220846510,0, 1670062130.857027076,0),[1670062128.962649107,1, 1670062130.220846510,0),[1670062131.779630742,0, 1670062131.840320318,0),[1670062079.229023108,0, 1670062079.568069929,0),[1670062076.840652671,0, 1670062076.936558056,0),[1670062076.936558056,0, 1670062076.946171755,0),[1670062076.946171755,0, 1670062077.646807606,1),[1670061994.576554089,1, 1670061994.628007753,0),[1670062012.642863490,0, 1670062014.050134337,0),[1670062014.050134337,0, 1670062014.140564372,0),[1670062015.675674708,0, 1670062015.675674708,1),[1670062014.140564372,0, 1670062015.675674708,0),[1670062020.468897757,1, 1670062020.958659876,0),[1670061946.940790159,1, 1670061948.161604565,0),[1670061949.161640621,0, 1670061949.544345551,1),[1670061965.163736664,2, 1670061965.735414624,0),[1670061967.005907286,1, 1670061967.098039088,0),[1670061967.098039088,0, 1670061967.386977852,0),[1670061967.386977852,0, 1670061967.483638851,0),[1670061965.735414624,0, 1670061966.078881424,0),[1670061969.370102355,1, 1670061969.732256629,0),[1670061968.926972352,0, 1670061969.370102355,1),[1670061970.573993336,0, 1670061970.674755571,0),[1670061969.732315458,0, 1670061969.765337622,0),[1670061970.858068303,0, 1670061970.864998640,0),[1670061970.674755571,0, 1670061970.858068303,0),[1670061980.679951979,1, 1670061981.057410039,0),[1670061984.836084680,2, 1670061985.325399152,0)}->[/Table/100/"20bfd42a":v-1881, /Table/100/"2ae1aba8":v-1877, /Table/100/"3570d962":v-1878, /Table/100/"375aee0f":v-1873, /Table/100/"465ecf3b":v-1880, /Table/100/"479be315":v-1876] [w]/Table/100/"465ecf3b":1670062173.338070704,1->v-1880 [d]/Table/100/"ba728422":1670062173.338070704,1-><nil>
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode4173209639
--- FAIL: TestKVNemesisSingleNode (454.23s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7827992?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7827992?buildTab=artifacts#/) on master @ [859e1e1f26d79af43cce16519f9ba984baaa495c](https://github.com/cockroachdb/cockroach/commits/859e1e1f26d79af43cce16519f9ba984baaa495c):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2260819905
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{8960d268"-db5ca6a7"}:{0:[1670148064.319069247,1, 1670148065.288286826,0), 1:[1670148064.313232227,0, 1670148065.288286826,0), 2:[1670148064.424665249,2, 1670148065.288286826,0), 3:[1670148064.137549029,0, 1670148065.288286826,0), gap:[<min>, 1670148044.136551161,0),[1670148044.160601581,0, 1670148044.162450623,0),[1670148046.158686195,1, 1670148046.167398972,0),[1670148046.170396603,0, 1670148046.180909575,0),[1670148046.180909575,0, 1670148046.351200192,0),[1670148046.167398972,0, 1670148046.170396603,0),[1670148044.162450623,0, 1670148044.191321143,0),[1670148048.715227825,0, 1670148048.742182004,0),[1670148048.742182004,0, 1670148048.750451315,0),[1670148048.863495159,0, 1670148048.873910822,0),[1670148048.750451315,0, 1670148048.788159938,0),[1670148048.873910822,0, 1670148049.010677285,1),[1670148055.371159819,0, 1670148055.469476265,0),[1670148055.365291617,1, 1670148055.371159819,0),[1670148056.875221781,1, 1670148057.289430330,0),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148056.612736904,0, 1670148056.635827011,1),[1670148057.289430330,0, 1670148057.398484383,0),[1670148064.024402656,1, 1670148064.131032094,0),[1670148063.152530974,1, 1670148063.155584284,0),[1670148066.473113678,0, 1670148066.586243779,0),[1670148069.386349470,1, 1670148069.404094198,0),[1670148068.072032888,0, 1670148068.101512246,1),[1670148067.965763699,1, 1670148068.069914281,0),[1670148068.069914281,0, 1670148068.072032888,0),[1670148064.319069247,0, 1670148064.377203965,0),[1670148064.377203965,0, 1670148064.424665249,2),[1670148064.187994768,0, 1670148064.319069247,0),[1670148064.131032094,0, 1670148064.187994768,0),[1670148064.424665249,2, 1670148064.536806718,0),[1670148077.728966034,0, 1670148077.902976706,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148080.095995211,0, 1670148080.202770129,0),[1670148091.008424405,1, 1670148091.160695248,0),[1670148091.240302979,0, 1670148091.273220960,0),[1670148091.160695248,0, 1670148091.183408086,1),[1670148091.183408086,1, 1670148091.240302979,0),[1670148088.106366212,0, 1670148088.207670082,0),[1670148086.314505849,0, 1670148086.315270191,0),[1670148086.315270191,0, 1670148086.402588331,0),[1670148086.402588331,0, 1670148086.994305417,1),[1670148086.994305417,1, 1670148087.419368202,0),[1670148086.166533351,1, 1670148086.314505849,0),[1670148102.526985881,1, 1670148102.598590895,0),[1670148100.907093948,1, 1670148101.168952945,0),[1670148101.190740416,0, 1670148101.200765152,1),[1670148080.765766730,1, 1670148080.776598577,0),[1670148080.554675139,0, 1670148080.765766730,1),[1670148085.282333009,1, 1670148085.456614377,0),[1670148085.797344361,0, 1670148085.955867028,0),[1670148112.567211021,1, 1670148112.601553391,0),[1670148112.601553391,0, 1670148112.619516948,0),[1670148112.842956239,0, 1670148112.854064166,0),[1670148112.819613390,0, 1670148112.842956239,0),[1670148147.520217768,0, 1670148147.657104961,0),[1670148146.725612179,1, 1670148147.056220234,0),[1670148158.228680930,1, 1670148158.322502155,0),[1670148158.322502155,0, 1670148158.324172969,0),[1670148154.884224787,0, 1670148154.987181855,0),[1670148155.010363264,0, 1670148155.010363264,1),[1670148154.987181855,0, 1670148155.010363264,0),[1670148155.010363264,1, 1670148155.099600705,0),[1670148144.624972085,0, 1670148144.683419709,1),[1670148144.606792751,1, 1670148144.624972085,0),[1670148144.539289814,1, 1670148144.606792751,1),[1670148144.683419709,1, 1670148144.693358372,0),[1670148118.838311689,1, 1670148119.240712520,0),[1670148119.841611024,0, 1670148119.914911923,0),[1670148119.500118962,1, 1670148119.841611024,0),[1670148130.854888027,0, 1670148130.899805539,0),[1670148139.732801991,1, 1670148140.080181748,0),[1670148216.404592462,0, 1670148216.557175095,1),[1670148214.795892281,1, 1670148214.955329917,1),[1670148188.238181075,1, 1670148188.357301546,0),[1670148182.449488971,0, 1670148183.097835683,0),[1670148180.283149878,0, 1670148180.629269665,0),[1670148180.227159186,0, 1670148180.283149878,0),[1670148164.446723008,0, 1670148164.595112183,0),[1670148166.892167245,1, 1670148166.982516684,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148175.201489366,1, 1670148175.536295590,0),[1670148195.761488448,1, 1670148195.929507473,0),[1670148190.543308708,1, 1670148190.697848614,0),[1670148208.842949594,0, 1670148208.846558789,0),[1670148204.477999586,0, 1670148204.617022629,0),[1670148204.896883027,1, 1670148204.985717712,0),[1670148208.833142765,0, 1670148208.842949594,0),[1670148208.458785287,0, 1670148208.589078057,0),[1670148208.017500408,1, 1670148208.458785287,0),[1670148208.657606068,1, 1670148208.833142765,0),[1670148195.930828370,0, 1670148195.993670609,0),[1670148195.929507473,0, 1670148195.930828370,0),[1670148196.907235910,0, 1670148196.912498006,0),[1670148196.283226274,0, 1670148196.907235910,0),[1670148163.886120198,1, 1670148164.094968246,0),[1670148164.094968246,0, 1670148164.243567243,0)}->[/Table/100/"8c80afd4":v-375, /Table/100/"a7bf767b":v-383, /Table/100/"b0bb9f54":v-386, /Table/100/"c43a6b0d":v-378] [rs]/Table/100/"{11da045a"-b2d5f22f"}:{0:[1670148064.424665249,2, 1670148065.288286826,0), 1:[1670148064.313232227,0, 1670148065.288286826,0), 2:[1670148064.319069247,1, 1670148065.288286826,0), 3:[1670148064.421950277,0, 1670148065.288286826,0), 4:[1670148064.145455694,0, 1670148065.288286826,0), 5:[0,0, 0,0), 6:[1670148064.024402656,1, 1670148066.284741440,1), 7:[1670148064.319069247,0, 1670148066.284741440,1), 8:[1670148064.319069247,0, 1670148066.284741440,1), 9:[1670148064.107129357,0, 1670148066.284741440,1), 10:[1670148064.424665249,0, 1670148066.284741440,1), 11:[1670148064.187994768,0, 1670148066.284741440,1), 12:[1670148064.424665249,2, 1670148066.284741440,1), 13:[1670148064.145455694,1, 1670148066.284741440,1), 14:[1670148064.293646216,0, 1670148066.284741440,1), gap:[<min>, 1670148044.136551161,0),[1670148044.136551161,0, 1670148044.137566597,0),[1670148044.137566597,0, 1670148044.138125794,0),[1670148046.170396603,0, 1670148046.360933679,0),[1670148045.469047737,0, 1670148045.678396455,0),[1670148045.453539419,0, 1670148045.469047737,0),[1670148045.678396455,0, 1670148045.692002088,0),[1670148044.142906641,0, 1670148044.160345709,0),[1670148046.970153418,0, 1670148046.972508048,0),[1670148046.894403223,1, 1670148046.970153418,0),[1670148046.972508048,0, 1670148046.972508048,1),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148060.168404275,0, 1670148060.169498092,0),[1670148064.319069247,1, 1670148064.377203965,0),[1670148064.319069247,0, 1670148064.319069247,1),[1670148064.286369199,0, 1670148064.319069247,0),[1670148064.024402656,1, 1670148064.107129357,0),[1670148064.107129357,0, 1670148064.131032094,0),[1670148064.187994768,0, 1670148064.241409095,0),[1670148064.241409095,0, 1670148064.286369199,0),[1670148064.145455694,0, 1670148064.145455694,1),[1670148064.131032094,0, 1670148064.145455694,0),[1670148064.145455694,1, 1670148064.187994768,0),[1670148064.424665249,2, 1670148064.557792225,0),[1670148064.424665249,1, 1670148064.424665249,2),[1670148064.377203965,0, 1670148064.424665249,1),[1670148080.095995211,0, 1670148080.201290557,0),[1670148080.554675139,0, 1670148080.649982979,0),[1670148080.649982979,0, 1670148080.723578373,0),[1670148066.584884009,0, 1670148066.586243779,0),[1670148066.566248185,0, 1670148066.584884009,0),[1670148066.379314038,0, 1670148066.471016220,0),[1670148066.471016220,0, 1670148066.566248185,0),[1670148066.284741440,1, 1670148066.379314038,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148072.920825570,1, 1670148073.003096705,1),[1670148073.003096705,1, 1670148073.018211244,0),[1670148110.897243446,0, 1670148111.330504626,1),[1670148102.526985881,1, 1670148102.553632760,0),[1670148102.927630719,0, 1670148102.985132072,0),[1670148102.985132072,0, 1670148103.011682271,0),[1670148103.011682271,0, 1670148103.012967622,0),[1670148085.139336944,1, 1670148085.222251157,0),[1670148085.222251157,0, 1670148085.223785736,0),[1670148085.797344361,0, 1670148085.940354836,0),[1670148085.282333009,1, 1670148085.351163986,0),[1670148215.756167664,1, 1670148215.846020243,0),[1670148215.846020243,0, 1670148215.942349036,0),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.233640688,0, 1670148205.233640688,1),[1670148204.896883027,1, 1670148205.142594540,0),[1670148139.732801991,1, 1670148139.968108530,0),[1670148139.968108530,0, 1670148140.036904364,0),[1670148158.228680930,1, 1670148158.313223550,0),[1670148166.968282339,0, 1670148166.982516684,0),[1670148166.892167245,1, 1670148166.968282339,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148190.619030093,0, 1670148190.697848614,0),[1670148190.543308708,1, 1670148190.619030093,0),[1670148193.886458757,0, 1670148193.918976756,1),[1670148196.683112888,0, 1670148196.905429187,0),[1670148195.929507473,0, 1670148195.993670609,0),[1670148195.761488448,1, 1670148195.929507473,0)}->[/Table/100/"b0bb9f54":v-386, /Table/100/"a7bf767b":v-383, /Table/100/"8c80afd4":v-375, /Table/100/"82f9a3f1":v-388, /Table/100/"7dce1af8":v-379, /Table/100/"77684da4":v-391, /Table/100/"72645aea":v-372, /Table/100/"72267024":v-385, /Table/100/"64f1c4e7":v-384, /Table/100/"604023d3":v-377, /Table/100/"572681ed":v-389, /Table/100/"49e78099":v-380, /Table/100/"487f893d":v-387, /Table/100/"486c0250":v-370, /Table/100/"32419333":v-382] [w]/Table/100/"37761a6e":1670148064.520070668,1->v-390 [w]/Table/100/"77684da4":1670148064.520070668,1->v-391
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{8960d268"-db5ca6a7"}:{0:[1670148064.319069247,1, 1670148065.288286826,0), 1:[1670148064.313232227,0, 1670148065.288286826,0), 2:[1670148064.424665249,2, 1670148065.288286826,0), 3:[1670148064.137549029,0, 1670148065.288286826,0), gap:[<min>, 1670148044.136551161,0),[1670148044.160601581,0, 1670148044.162450623,0),[1670148046.158686195,1, 1670148046.167398972,0),[1670148046.170396603,0, 1670148046.180909575,0),[1670148046.180909575,0, 1670148046.351200192,0),[1670148046.167398972,0, 1670148046.170396603,0),[1670148044.162450623,0, 1670148044.191321143,0),[1670148048.715227825,0, 1670148048.742182004,0),[1670148048.742182004,0, 1670148048.750451315,0),[1670148048.863495159,0, 1670148048.873910822,0),[1670148048.750451315,0, 1670148048.788159938,0),[1670148048.873910822,0, 1670148049.010677285,1),[1670148055.371159819,0, 1670148055.469476265,0),[1670148055.365291617,1, 1670148055.371159819,0),[1670148056.875221781,1, 1670148057.289430330,0),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148056.612736904,0, 1670148056.635827011,1),[1670148057.289430330,0, 1670148057.398484383,0),[1670148064.024402656,1, 1670148064.131032094,0),[1670148063.152530974,1, 1670148063.155584284,0),[1670148066.473113678,0, 1670148066.586243779,0),[1670148069.386349470,1, 1670148069.404094198,0),[1670148068.072032888,0, 1670148068.101512246,1),[1670148067.965763699,1, 1670148068.069914281,0),[1670148068.069914281,0, 1670148068.072032888,0),[1670148064.319069247,0, 1670148064.377203965,0),[1670148064.377203965,0, 1670148064.424665249,2),[1670148064.187994768,0, 1670148064.319069247,0),[1670148064.131032094,0, 1670148064.187994768,0),[1670148064.424665249,2, 1670148064.536806718,0),[1670148077.728966034,0, 1670148077.902976706,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148080.095995211,0, 1670148080.202770129,0),[1670148091.008424405,1, 1670148091.160695248,0),[1670148091.240302979,0, 1670148091.273220960,0),[1670148091.160695248,0, 1670148091.183408086,1),[1670148091.183408086,1, 1670148091.240302979,0),[1670148088.106366212,0, 1670148088.207670082,0),[1670148086.314505849,0, 1670148086.315270191,0),[1670148086.315270191,0, 1670148086.402588331,0),[1670148086.402588331,0, 1670148086.994305417,1),[1670148086.994305417,1, 1670148087.419368202,0),[1670148086.166533351,1, 1670148086.314505849,0),[1670148102.526985881,1, 1670148102.598590895,0),[1670148100.907093948,1, 1670148101.168952945,0),[1670148101.190740416,0, 1670148101.200765152,1),[1670148080.765766730,1, 1670148080.776598577,0),[1670148080.554675139,0, 1670148080.765766730,1),[1670148085.282333009,1, 1670148085.456614377,0),[1670148085.797344361,0, 1670148085.955867028,0),[1670148112.567211021,1, 1670148112.601553391,0),[1670148112.601553391,0, 1670148112.619516948,0),[1670148112.842956239,0, 1670148112.854064166,0),[1670148112.819613390,0, 1670148112.842956239,0),[1670148147.520217768,0, 1670148147.657104961,0),[1670148146.725612179,1, 1670148147.056220234,0),[1670148158.228680930,1, 1670148158.322502155,0),[1670148158.322502155,0, 1670148158.324172969,0),[1670148154.884224787,0, 1670148154.987181855,0),[1670148155.010363264,0, 1670148155.010363264,1),[1670148154.987181855,0, 1670148155.010363264,0),[1670148155.010363264,1, 1670148155.099600705,0),[1670148144.624972085,0, 1670148144.683419709,1),[1670148144.606792751,1, 1670148144.624972085,0),[1670148144.539289814,1, 1670148144.606792751,1),[1670148144.683419709,1, 1670148144.693358372,0),[1670148118.838311689,1, 1670148119.240712520,0),[1670148119.841611024,0, 1670148119.914911923,0),[1670148119.500118962,1, 1670148119.841611024,0),[1670148130.854888027,0, 1670148130.899805539,0),[1670148139.732801991,1, 1670148140.080181748,0),[1670148216.404592462,0, 1670148216.557175095,1),[1670148214.795892281,1, 1670148214.955329917,1),[1670148188.238181075,1, 1670148188.357301546,0),[1670148182.449488971,0, 1670148183.097835683,0),[1670148180.283149878,0, 1670148180.629269665,0),[1670148180.227159186,0, 1670148180.283149878,0),[1670148164.446723008,0, 1670148164.595112183,0),[1670148166.892167245,1, 1670148166.982516684,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148175.201489366,1, 1670148175.536295590,0),[1670148195.761488448,1, 1670148195.929507473,0),[1670148190.543308708,1, 1670148190.697848614,0),[1670148208.842949594,0, 1670148208.846558789,0),[1670148204.477999586,0, 1670148204.617022629,0),[1670148204.896883027,1, 1670148204.985717712,0),[1670148208.833142765,0, 1670148208.842949594,0),[1670148208.458785287,0, 1670148208.589078057,0),[1670148208.017500408,1, 1670148208.458785287,0),[1670148208.657606068,1, 1670148208.833142765,0),[1670148195.930828370,0, 1670148195.993670609,0),[1670148195.929507473,0, 1670148195.930828370,0),[1670148196.907235910,0, 1670148196.912498006,0),[1670148196.283226274,0, 1670148196.907235910,0),[1670148163.886120198,1, 1670148164.094968246,0),[1670148164.094968246,0, 1670148164.243567243,0)}->[/Table/100/"8c80afd4":v-375, /Table/100/"a7bf767b":v-383, /Table/100/"b0bb9f54":v-386, /Table/100/"c43a6b0d":v-378] [rs]/Table/100/"{11da045a"-b2d5f22f"}:{0:[1670148064.424665249,2, 1670148065.288286826,0), 1:[1670148064.313232227,0, 1670148065.288286826,0), 2:[1670148064.319069247,1, 1670148065.288286826,0), 3:[1670148064.421950277,0, 1670148065.288286826,0), 4:[1670148064.145455694,0, 1670148065.288286826,0), 5:[0,0, 0,0), 6:[1670148064.024402656,1, 1670148066.284741440,1), 7:[1670148064.319069247,0, 1670148066.284741440,1), 8:[1670148064.319069247,0, 1670148066.284741440,1), 9:[1670148064.107129357,0, 1670148066.284741440,1), 10:[1670148064.424665249,0, 1670148066.284741440,1), 11:[1670148064.187994768,0, 1670148066.284741440,1), 12:[1670148064.424665249,2, 1670148066.284741440,1), 13:[1670148064.145455694,1, 1670148066.284741440,1), 14:[1670148064.293646216,0, 1670148066.284741440,1), gap:[<min>, 1670148044.136551161,0),[1670148044.136551161,0, 1670148044.137566597,0),[1670148044.137566597,0, 1670148044.138125794,0),[1670148046.170396603,0, 1670148046.360933679,0),[1670148045.469047737,0, 1670148045.678396455,0),[1670148045.453539419,0, 1670148045.469047737,0),[1670148045.678396455,0, 1670148045.692002088,0),[1670148044.142906641,0, 1670148044.160345709,0),[1670148046.970153418,0, 1670148046.972508048,0),[1670148046.894403223,1, 1670148046.970153418,0),[1670148046.972508048,0, 1670148046.972508048,1),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148060.168404275,0, 1670148060.169498092,0),[1670148064.319069247,1, 1670148064.377203965,0),[1670148064.319069247,0, 1670148064.319069247,1),[1670148064.286369199,0, 1670148064.319069247,0),[1670148064.024402656,1, 1670148064.107129357,0),[1670148064.107129357,0, 1670148064.131032094,0),[1670148064.187994768,0, 1670148064.241409095,0),[1670148064.241409095,0, 1670148064.286369199,0),[1670148064.145455694,0, 1670148064.145455694,1),[1670148064.131032094,0, 1670148064.145455694,0),[1670148064.145455694,1, 1670148064.187994768,0),[1670148064.424665249,2, 1670148064.557792225,0),[1670148064.424665249,1, 1670148064.424665249,2),[1670148064.377203965,0, 1670148064.424665249,1),[1670148080.095995211,0, 1670148080.201290557,0),[1670148080.554675139,0, 1670148080.649982979,0),[1670148080.649982979,0, 1670148080.723578373,0),[1670148066.584884009,0, 1670148066.586243779,0),[1670148066.566248185,0, 1670148066.584884009,0),[1670148066.379314038,0, 1670148066.471016220,0),[1670148066.471016220,0, 1670148066.566248185,0),[1670148066.284741440,1, 1670148066.379314038,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148072.920825570,1, 1670148073.003096705,1),[1670148073.003096705,1, 1670148073.018211244,0),[1670148110.897243446,0, 1670148111.330504626,1),[1670148102.526985881,1, 1670148102.553632760,0),[1670148102.927630719,0, 1670148102.985132072,0),[1670148102.985132072,0, 1670148103.011682271,0),[1670148103.011682271,0, 1670148103.012967622,0),[1670148085.139336944,1, 1670148085.222251157,0),[1670148085.222251157,0, 1670148085.223785736,0),[1670148085.797344361,0, 1670148085.940354836,0),[1670148085.282333009,1, 1670148085.351163986,0),[1670148215.756167664,1, 1670148215.846020243,0),[1670148215.846020243,0, 1670148215.942349036,0),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.233640688,0, 1670148205.233640688,1),[1670148204.896883027,1, 1670148205.142594540,0),[1670148139.732801991,1, 1670148139.968108530,0),[1670148139.968108530,0, 1670148140.036904364,0),[1670148158.228680930,1, 1670148158.313223550,0),[1670148166.968282339,0, 1670148166.982516684,0),[1670148166.892167245,1, 1670148166.968282339,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148190.619030093,0, 1670148190.697848614,0),[1670148190.543308708,1, 1670148190.619030093,0),[1670148193.886458757,0, 1670148193.918976756,1),[1670148196.683112888,0, 1670148196.905429187,0),[1670148195.929507473,0, 1670148195.993670609,0),[1670148195.761488448,1, 1670148195.929507473,0)}->[/Table/100/"b0bb9f54":v-386, /Table/100/"a7bf767b":v-383, /Table/100/"8c80afd4":v-375, /Table/100/"82f9a3f1":v-388, /Table/100/"7dce1af8":v-379, /Table/100/"77684da4":v-391, /Table/100/"72645aea":v-372, /Table/100/"72267024":v-385, /Table/100/"64f1c4e7":v-384, /Table/100/"604023d3":v-377, /Table/100/"572681ed":v-389, /Table/100/"49e78099":v-380, /Table/100/"487f893d":v-387, /Table/100/"486c0250":v-370, /Table/100/"32419333":v-382] [w]/Table/100/"37761a6e":1670148064.520070668,1->v-390 [w]/Table/100/"77684da4":1670148064.520070668,1->v-391
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{31ca24ae"-cd587630"}:{0:[0,0, 0,0), gap:[<min>, 1670148044.136551161,0),[1670148046.170396603,0, 1670148046.180909575,0),[1670148046.180909575,0, 1670148046.360933679,0),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148066.584884009,0, 1670148066.586243779,0),[1670148066.473113678,0, 1670148066.584884009,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148080.095995211,0, 1670148080.201290557,0),[1670148085.139336944,1, 1670148085.222251157,0),[1670148085.222251157,0, 1670148085.223785736,0),[1670148085.282333009,1, 1670148085.351163986,0),[1670148085.797344361,0, 1670148085.940354836,0),[1670148080.554675139,0, 1670148080.649982979,0),[1670148080.649982979,0, 1670148080.723578373,0),[1670148102.526985881,1, 1670148102.553632760,0),[1670148102.927630719,0, 1670148102.984518396,0),[1670148216.404592462,0, 1670148216.517635024,0),[1670148216.517635024,0, 1670148216.557175095,1),[1670148205.673883734,0, 1670148205.911284316,0),[1670148208.017500408,1, 1670148208.331352126,0),[1670148204.896883027,1, 1670148205.033646087,1),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.033646087,1, 1670148205.142594540,0),[1670148205.630835898,0, 1670148205.673883734,0),[1670148205.233640688,0, 1670148205.630835898,0),[1670148212.218227113,1, 1670148212.283233013,0),[1670148212.015908464,1, 1670148212.218227113,1),[1670148139.732801991,1, 1670148139.968108530,0),[1670148139.968108530,0, 1670148140.036904364,0),[1670148158.228680930,1, 1670148158.313223550,0),[1670148163.886120198,1, 1670148164.001182118,0),[1670148164.001182118,0, 1670148164.094968246,0),[1670148164.094968246,0, 1670148164.097202794,0),[1670148166.968282339,0, 1670148166.982516684,0),[1670148166.892167245,1, 1670148166.968282339,0),[1670148164.446723008,0, 1670148164.527006307,0),[1670148175.201489366,1, 1670148175.511123631,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148188.238181075,1, 1670148188.357301546,0),[1670148190.543308708,1, 1670148190.697848614,0),[1670148193.886458757,0, 1670148193.895948914,0),[1670148196.283226274,0, 1670148196.905429187,0),[1670148193.970546448,0, 1670148193.972834587,0),[1670148193.972834587,0, 1670148194.006779328,0),[1670148193.895948914,0, 1670148193.970546448,0),[1670148195.761488448,1, 1670148195.929507473,0),[1670148195.930828370,0, 1670148195.993670609,0),[1670148195.929507473,0, 1670148195.930828370,0)}->[/Table/100/"998431f2":v-2179] [rs]/Table/100/"{05d9bddc"-9b59cd9c"}:{gap:[<min>, 1670148044.136551161,0),[1670148044.136551161,0, 1670148044.137566597,0),[1670148044.137566597,0, 1670148044.138125794,0),[1670148045.469047737,0, 1670148045.678396455,0),[1670148045.453539419,0, 1670148045.469047737,0),[1670148045.678396455,0, 1670148045.692002088,0),[1670148044.142906641,0, 1670148044.160345709,0),[1670148046.970153418,0, 1670148046.972508048,0),[1670148046.972508048,0, 1670148046.972508048,1),[1670148046.972508048,1, 1670148046.982593107,0),[1670148046.894403223,1, 1670148046.970153418,0),[1670148046.982593107,0, 1670148046.991474129,0),[1670148055.749860248,1, 1670148055.842153433,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148072.920825570,1, 1670148073.003096705,1),[1670148073.003096705,1, 1670148073.022348877,0),[1670148110.897243446,0, 1670148111.296463372,1),[1670148102.439380449,0, 1670148102.553632760,0),[1670148102.424349881,1, 1670148102.439380449,0),[1670148102.927630719,0, 1670148102.985132072,0),[1670148102.985132072,0, 1670148103.011682271,0),[1670148103.011682271,0, 1670148103.012967622,0),[1670148205.673883734,0, 1670148205.911284316,0),[1670148205.630835898,0, 1670148205.673883734,0),[1670148205.237140080,1, 1670148205.630835898,0),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.233640688,0, 1670148205.237140080,1),[1670148204.896883027,1, 1670148205.142594540,0)}->[] [w]/Table/100/"998431f2":1670148205.233640688,1->v-2179
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{31ca24ae"-cd587630"}:{0:[0,0, 0,0), gap:[<min>, 1670148044.136551161,0),[1670148046.170396603,0, 1670148046.180909575,0),[1670148046.180909575,0, 1670148046.360933679,0),[1670148055.749860248,1, 1670148055.769825713,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148066.584884009,0, 1670148066.586243779,0),[1670148066.473113678,0, 1670148066.584884009,0),[1670148076.047827306,1, 1670148076.073150993,0),[1670148080.095995211,0, 1670148080.201290557,0),[1670148085.139336944,1, 1670148085.222251157,0),[1670148085.222251157,0, 1670148085.223785736,0),[1670148085.282333009,1, 1670148085.351163986,0),[1670148085.797344361,0, 1670148085.940354836,0),[1670148080.554675139,0, 1670148080.649982979,0),[1670148080.649982979,0, 1670148080.723578373,0),[1670148102.526985881,1, 1670148102.553632760,0),[1670148102.927630719,0, 1670148102.984518396,0),[1670148216.404592462,0, 1670148216.517635024,0),[1670148216.517635024,0, 1670148216.557175095,1),[1670148205.673883734,0, 1670148205.911284316,0),[1670148208.017500408,1, 1670148208.331352126,0),[1670148204.896883027,1, 1670148205.033646087,1),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.033646087,1, 1670148205.142594540,0),[1670148205.630835898,0, 1670148205.673883734,0),[1670148205.233640688,0, 1670148205.630835898,0),[1670148212.218227113,1, 1670148212.283233013,0),[1670148212.015908464,1, 1670148212.218227113,1),[1670148139.732801991,1, 1670148139.968108530,0),[1670148139.968108530,0, 1670148140.036904364,0),[1670148158.228680930,1, 1670148158.313223550,0),[1670148163.886120198,1, 1670148164.001182118,0),[1670148164.001182118,0, 1670148164.094968246,0),[1670148164.094968246,0, 1670148164.097202794,0),[1670148166.968282339,0, 1670148166.982516684,0),[1670148166.892167245,1, 1670148166.968282339,0),[1670148164.446723008,0, 1670148164.527006307,0),[1670148175.201489366,1, 1670148175.511123631,0),[1670148180.123310160,1, 1670148180.227159186,0),[1670148188.238181075,1, 1670148188.357301546,0),[1670148190.543308708,1, 1670148190.697848614,0),[1670148193.886458757,0, 1670148193.895948914,0),[1670148196.283226274,0, 1670148196.905429187,0),[1670148193.970546448,0, 1670148193.972834587,0),[1670148193.972834587,0, 1670148194.006779328,0),[1670148193.895948914,0, 1670148193.970546448,0),[1670148195.761488448,1, 1670148195.929507473,0),[1670148195.930828370,0, 1670148195.993670609,0),[1670148195.929507473,0, 1670148195.930828370,0)}->[/Table/100/"998431f2":v-2179] [rs]/Table/100/"{05d9bddc"-9b59cd9c"}:{gap:[<min>, 1670148044.136551161,0),[1670148044.136551161,0, 1670148044.137566597,0),[1670148044.137566597,0, 1670148044.138125794,0),[1670148045.469047737,0, 1670148045.678396455,0),[1670148045.453539419,0, 1670148045.469047737,0),[1670148045.678396455,0, 1670148045.692002088,0),[1670148044.142906641,0, 1670148044.160345709,0),[1670148046.970153418,0, 1670148046.972508048,0),[1670148046.972508048,0, 1670148046.972508048,1),[1670148046.972508048,1, 1670148046.982593107,0),[1670148046.894403223,1, 1670148046.970153418,0),[1670148046.982593107,0, 1670148046.991474129,0),[1670148055.749860248,1, 1670148055.842153433,0),[1670148056.558592346,1, 1670148056.612736904,0),[1670148072.920825570,1, 1670148073.003096705,1),[1670148073.003096705,1, 1670148073.022348877,0),[1670148110.897243446,0, 1670148111.296463372,1),[1670148102.439380449,0, 1670148102.553632760,0),[1670148102.424349881,1, 1670148102.439380449,0),[1670148102.927630719,0, 1670148102.985132072,0),[1670148102.985132072,0, 1670148103.011682271,0),[1670148103.011682271,0, 1670148103.012967622,0),[1670148205.673883734,0, 1670148205.911284316,0),[1670148205.630835898,0, 1670148205.673883734,0),[1670148205.237140080,1, 1670148205.630835898,0),[1670148205.142594540,0, 1670148205.233640688,0),[1670148205.233640688,0, 1670148205.237140080,1),[1670148204.896883027,1, 1670148205.142594540,0)}->[] [w]/Table/100/"998431f2":1670148205.233640688,1->v-2179
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2260819905
--- FAIL: TestKVNemesisSingleNode (241.39s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7827993?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7827993?buildTab=artifacts#/) on master @ [859e1e1f26d79af43cce16519f9ba984baaa495c](https://github.com/cockroachdb/cockroach/commits/859e1e1f26d79af43cce16519f9ba984baaa495c):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2653695887
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [rs]/Table/100/"{3886d55e"-808d21ec"}:{0:[1670148097.868893543,0, 1670148108.292556212,1), 1:[1670148097.507007311,0, 1670148108.292556212,1), 2:[1670148097.507007311,0, 1670148108.292556212,1), 3:[1670148099.429375236,1, 1670148108.292556212,1), 4:[1670148099.776146280,0, 1670148108.292556212,1), gap:[<min>, 1670147958.634243175,0),[1670147980.741394194,0, 1670147981.330120665,0),[1670147981.330120665,0, 1670147982.145242771,0),[1670147979.296217319,0, 1670147979.799974608,1),[1670147978.512952311,0, 1670147979.140688903,0),[1670147979.799974608,1, 1670147980.741394194,0),[1670147961.711608261,0, 1670147961.728263678,0),[1670147961.728263678,0, 1670147962.013969150,0),[1670147962.053231659,0, 1670147962.419397787,1),[1670147962.688258493,0, 1670147962.770480354,0),[1670147962.770480354,0, 1670147962.785770777,0),[1670147962.785770777,0, 1670147962.892477039,0),[1670147982.827207112,1, 1670147983.074200793,0),[1670147983.074200793,0, 1670147983.094838233,0),[1670147991.383076089,0, 1670147991.392523301,1),[1670147991.230645641,1, 1670147991.383076089,0),[1670147991.020433869,0, 1670147991.028622202,0),[1670147990.272642515,2, 1670147991.020433869,0),[1670147987.294251012,0, 1670147987.374665461,0),[1670147987.374665461,0, 1670147987.768386922,0),[1670147987.768386922,0, 1670147988.015311004,0),[1670147985.936172256,1, 1670147986.615947416,0),[1670147986.615947416,0, 1670147986.819503272,1),[1670147986.819503272,1, 1670147987.294251012,0),[1670147998.979954611,1, 1670147999.100235186,1),[1670147995.354370625,1, 1670147995.576695326,0),[1670147999.570327398,1, 1670147999.584504961,0),[1670147999.584504961,0, 1670147999.754693576,1),[1670147999.957543489,0, 1670148000.279732373,0),[1670147999.890037204,1, 1670147999.957543489,0),[1670148008.873698848,0, 1670148008.873698848,2),[1670148008.566747977,0, 1670148008.873698848,0),[1670148008.873698848,2, 1670148009.722216068,1),[1670148011.761444359,1, 1670148012.025264941,0),[1670148010.351027555,1, 1670148010.582433939,0),[1670148026.151365776,1, 1670148027.081716385,0),[1670148031.396478150,0, 1670148031.531051180,1),[1670148030.591387021,0, 1670148031.396478150,0),[1670148032.026582095,0, 1670148032.793740236,0),[1670148031.531051180,1, 1670148032.026582095,0),[1670148036.382033410,1, 1670148037.060237416,0),[1670148035.395993545,1, 1670148035.567955022,1),[1670148012.032039306,0, 1670148012.176643107,0),[1670148012.025264941,0, 1670148012.032039306,0),[1670148013.804326024,1, 1670148014.086403514,0),[1670148014.937362559,1, 1670148015.077294696,0),[1670148015.077294696,0, 1670148015.145366950,0),[1670148017.639852256,1, 1670148019.187360253,0),[1670148062.907552427,1, 1670148064.199173492,0),[1670148064.199173492,0, 1670148064.283913010,1),[1670148066.986557005,0, 1670148067.441102525,0),[1670148074.930295173,0, 1670148075.194478735,0),[1670148075.194478735,0, 1670148076.418973216,0),[1670148076.418973216,0, 1670148076.523474648,1),[1670148081.126231453,1, 1670148081.192197464,0),[1670148080.290248203,1, 1670148081.126231453,1),[1670148079.158704797,0, 1670148079.197215931,1),[1670148078.541331850,1, 1670148079.158704797,0),[1670148083.451326843,1, 1670148084.085281752,0),[1670148084.085281752,0, 1670148084.112835815,0),[1670148081.827520045,0, 1670148082.271541228,0),[1670148081.219898625,0, 1670148081.827520045,0),[1670148081.192197464,0, 1670148081.219898625,0),[1670148084.112835815,0, 1670148084.259097903,0),[1670148085.874280727,0, 1670148086.065992466,0),[1670148084.259097903,0, 1670148085.874280727,0),[1670148095.184782199,1, 1670148096.258340878,0),[1670148088.846717770,0, 1670148090.030098093,0),[1670148098.132017989,0, 1670148098.324461750,0),[1670148096.804659908,1, 1670148098.132017989,0),[1670148099.194513886,0, 1670148099.426053819,0),[1670148098.324461750,0, 1670148099.194513886,0),[1670148099.426053819,0, 1670148101.611794866,0),[1670148135.290431596,4, 1670148135.738221414,0),[1670148135.290431596,2, 1670148135.290431596,4),[1670148177.228308860,1, 1670148178.127567673,0),[1670148178.127567673,0, 1670148179.926634797,0),[1670148179.926634797,0, 1670148180.087978431,0),[1670148153.810068778,1, 1670148153.859530513,0),[1670148150.434856856,0, 1670148150.836215114,0),[1670148150.836215114,0, 1670148151.018169300,0),[1670148151.018169300,0, 1670148152.090132339,0),[1670148156.414912563,0, 1670148156.925981841,0),[1670148158.936252170,0, 1670148159.122354602,0),[1670148161.822828677,0, 1670148161.885875607,0),[1670148161.353642065,0, 1670148161.822828677,0),[1670148160.943042194,0, 1670148161.353642065,0),[1670148162.956289507,1, 1670148163.339019412,0),[1670148163.339019412,0, 1670148163.574661854,0),[1670148163.574661854,0, 1670148165.113603299,0),[1670148184.693558406,1, 1670148184.963864371,0),[1670148184.963864371,0, 1670148185.035449251,0),[1670148185.035449251,0, 1670148185.642812258,0),[1670148191.670449125,0, 1670148191.895938468,0),[1670148194.733034699,1, 1670148195.218666450,0),[1670148195.218666450,0, 1670148195.218666450,1),[1670148208.827317110,0, 1670148208.941881657,0),[1670148206.770423291,0, 1670148207.366101773,0),[1670148207.366101773,0, 1670148207.526552897,0),[1670148207.526552897,0, 1670148208.827317110,0),[1670148219.564081886,1, 1670148220.365227769,0),[1670148212.557551106,0, 1670148212.947325130,0),[1670148215.077785886,1, 1670148215.915940721,0),[1670148215.915940721,0, 1670148216.026573215,0),[1670148216.120633697,2, 1670148217.137472893,0),[1670148236.213340603,0, 1670148237.202161836,0),[1670148238.951134663,0, 1670148240.270480960,0),[1670148246.077506399,1, 1670148247.214700401,0),[1670148247.805390317,1, 1670148248.873526156,0),[1670148245.184990881,0, 1670148246.077506399,1),[1670148231.899525698,0, 1670148232.115044601,0),[1670148231.729250558,1, 1670148231.899525698,0),[1670148265.854785507,1, 1670148265.902520866,0),[1670148274.873740372,0, 1670148275.418241003,0),[1670148274.364643506,1, 1670148274.872495264,1),[1670148273.822432626,1, 1670148274.364643506,1),[1670148307.002184828,1, 1670148308.078747401,0),[1670148310.709804968,0, 1670148312.023578464,0),[1670148322.730482970,0, 1670148323.546002332,0),[1670148322.523546626,2, 1670148322.730482970,0),[1670148312.503762176,1, 1670148312.601102664,0),[1670148324.188449116,0, 1670148324.204625021,0),[1670148324.204625021,0, 1670148324.765687448,0),[1670148323.546002332,0, 1670148323.654690418,0),[1670148324.094434017,1, 1670148324.188449116,0),[1670148355.129455254,0, 1670148355.129455254,1),[1670148329.326360881,0, 1670148329.595833726,0),[1670148295.687538091,0, 1670148296.074544915,0),[1670148301.059959363,0, 1670148301.812415664,2),[1670148300.213407134,1, 1670148301.059959363,0)}->[/Table/100/"68667243":v-1011, /Table/100/"639f80fc":v-1008, /Table/100/"61525e01":v-1007, /Table/100/"5f7c6ee6":v-1019, /Table/100/"3a98c5d4":v-1024] [rs]/Table/100/"{28f52639"-d420250d"}:{0:[1670148099.329073824,0, 1670148117.010125492,0), 1:[0,0, 0,0), 2:[1670148099.329073824,0, 1670148102.088706669,1), 3:[1670148099.264749843,0, 1670148119.313480267,1), 4:[1670148097.868893543,0, 1670148108.292556212,1), 5:[1670148097.507007311,0, 1670148108.292556212,1), 6:[1670148097.507007311,0, 1670148108.292556212,1), 7:[1670148099.429375236,1, 1670148108.292556212,1), 8:[1670148099.776146280,0, 1670148108.292556212,1), 9:[1670148099.264749843,0, 1670148108.292556212,1), 10:[1670148099.664517754,0, 1670148120.951531998,0), gap:[<min>, 1670147958.110963382,0),[1670147958.133650657,0, 1670147958.197913885,0),[1670147961.711608261,0, 1670147961.728263678,0),[1670147961.728263678,0, 1670147961.991694692,0),[1670147978.512952311,0, 1670147978.884663308,0),[1670147978.884663308,0, 1670147978.903808293,0),[1670147978.903808293,0, 1670147979.140688903,0),[1670147999.957543489,0, 1670147999.963688624,1),[1670148000.040665548,0, 1670148000.279732373,0),[1670147999.890037204,1, 1670147999.957543489,0),[1670147991.383076089,0, 1670147991.392523301,1),[1670147991.230645641,1, 1670147991.383076089,0),[1670147990.272642515,2, 1670147990.302551836,0),[1670148026.151365776,1, 1670148026.332921792,0),[1670148030.591387021,0, 1670148031.190419173,0),[1670148035.395993545,1, 1670148035.567955022,1),[1670148036.382033410,1, 1670148036.448765002,0),[1670148012.128542874,0, 1670148012.176643107,0),[1670148012.065970824,0, 1670148012.128542874,0),[1670148012.032039306,0, 1670148012.065970824,0),[1670148012.025264941,0, 1670148012.032039306,0),[1670148011.761444359,1, 1670148012.025264941,0),[1670148010.351027555,1, 1670148010.582433939,0),[1670148079.158704797,0, 1670148079.197215931,1),[1670148078.845565980,0, 1670148079.158704797,0),[1670148078.541331850,1, 1670148078.845565980,0),[1670148099.288692240,0, 1670148099.426053819,0),[1670148099.426053819,0, 1670148100.321234822,0),[1670148100.621566531,1, 1670148100.960183552,0),[1670148100.321234822,0, 1670148100.621566531,1),[1670148100.960183552,0, 1670148101.068858522,0),[1670148101.068858522,0, 1670148101.109685338,1),[1670148099.194513886,0, 1670148099.288692240,0),[1670148098.324461750,0, 1670148098.819874811,0),[1670148098.819874811,0, 1670148099.194513886,0),[1670148096.804659908,1, 1670148097.704139080,0),[1670148095.184782199,1, 1670148095.893289081,0),[1670148194.733034699,1, 1670148194.948931925,1),[1670148194.948931925,1, 1670148195.164222288,2),[1670148206.949578611,0, 1670148207.186364350,0),[1670148206.770423291,0, 1670148206.877480326,0),[1670148206.877480326,0, 1670148206.949578611,0),[1670148215.077785886,1, 1670148215.345861707,0),[1670148216.723832860,0, 1670148217.137472893,0),[1670148184.693558406,1, 1670148184.913248764,0),[1670148178.692674105,1, 1670148179.731311433,0),[1670148185.035449251,0, 1670148185.458002443,0),[1670148191.670449125,0, 1670148191.761148417,0),[1670148063.014731697,0, 1670148063.557563124,0),[1670148063.557563124,0, 1670148063.637710337,0),[1670148062.907552427,1, 1670148063.014731697,0),[1670148231.729250558,1, 1670148231.867431344,0),[1670148231.899525698,0, 1670148232.115044601,0),[1670148231.867431344,0, 1670148231.899525698,0),[1670148247.134118827,1, 1670148247.214700401,0),[1670148274.364643506,1, 1670148274.635602321,0),[1670148301.059959363,0, 1670148301.217706497,0),[1670148300.427545067,0, 1670148301.059959363,0)}->[/Table/100/"d2a3a739":v-1017, /Table/100/"ba2aae23":v-1028, /Table/100/"9bda0268":v-1018, /Table/100/"84b61ebd":v-1016, /Table/100/"68667243":v-1011, /Table/100/"639f80fc":v-1008, /Table/100/"61525e01":v-1007, /Table/100/"5f7c6ee6":v-1019, /Table/100/"3a98c5d4":v-1024, /Table/100/"31474658":v-1015, /Table/100/"2a7be558":v-1023] [w]/Table/100/"ba2aae23":1670148100.783251064,1->v-1028 [w]/Table/100/"18bc224d":1670148100.783251064,1->v-1029
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [rs]/Table/100/"{3886d55e"-808d21ec"}:{0:[1670148097.868893543,0, 1670148108.292556212,1), 1:[1670148097.507007311,0, 1670148108.292556212,1), 2:[1670148097.507007311,0, 1670148108.292556212,1), 3:[1670148099.429375236,1, 1670148108.292556212,1), 4:[1670148099.776146280,0, 1670148108.292556212,1), gap:[<min>, 1670147958.634243175,0),[1670147980.741394194,0, 1670147981.330120665,0),[1670147981.330120665,0, 1670147982.145242771,0),[1670147979.296217319,0, 1670147979.799974608,1),[1670147978.512952311,0, 1670147979.140688903,0),[1670147979.799974608,1, 1670147980.741394194,0),[1670147961.711608261,0, 1670147961.728263678,0),[1670147961.728263678,0, 1670147962.013969150,0),[1670147962.053231659,0, 1670147962.419397787,1),[1670147962.688258493,0, 1670147962.770480354,0),[1670147962.770480354,0, 1670147962.785770777,0),[1670147962.785770777,0, 1670147962.892477039,0),[1670147982.827207112,1, 1670147983.074200793,0),[1670147983.074200793,0, 1670147983.094838233,0),[1670147991.383076089,0, 1670147991.392523301,1),[1670147991.230645641,1, 1670147991.383076089,0),[1670147991.020433869,0, 1670147991.028622202,0),[1670147990.272642515,2, 1670147991.020433869,0),[1670147987.294251012,0, 1670147987.374665461,0),[1670147987.374665461,0, 1670147987.768386922,0),[1670147987.768386922,0, 1670147988.015311004,0),[1670147985.936172256,1, 1670147986.615947416,0),[1670147986.615947416,0, 1670147986.819503272,1),[1670147986.819503272,1, 1670147987.294251012,0),[1670147998.979954611,1, 1670147999.100235186,1),[1670147995.354370625,1, 1670147995.576695326,0),[1670147999.570327398,1, 1670147999.584504961,0),[1670147999.584504961,0, 1670147999.754693576,1),[1670147999.957543489,0, 1670148000.279732373,0),[1670147999.890037204,1, 1670147999.957543489,0),[1670148008.873698848,0, 1670148008.873698848,2),[1670148008.566747977,0, 1670148008.873698848,0),[1670148008.873698848,2, 1670148009.722216068,1),[1670148011.761444359,1, 1670148012.025264941,0),[1670148010.351027555,1, 1670148010.582433939,0),[1670148026.151365776,1, 1670148027.081716385,0),[1670148031.396478150,0, 1670148031.531051180,1),[1670148030.591387021,0, 1670148031.396478150,0),[1670148032.026582095,0, 1670148032.793740236,0),[1670148031.531051180,1, 1670148032.026582095,0),[1670148036.382033410,1, 1670148037.060237416,0),[1670148035.395993545,1, 1670148035.567955022,1),[1670148012.032039306,0, 1670148012.176643107,0),[1670148012.025264941,0, 1670148012.032039306,0),[1670148013.804326024,1, 1670148014.086403514,0),[1670148014.937362559,1, 1670148015.077294696,0),[1670148015.077294696,0, 1670148015.145366950,0),[1670148017.639852256,1, 1670148019.187360253,0),[1670148062.907552427,1, 1670148064.199173492,0),[1670148064.199173492,0, 1670148064.283913010,1),[1670148066.986557005,0, 1670148067.441102525,0),[1670148074.930295173,0, 1670148075.194478735,0),[1670148075.194478735,0, 1670148076.418973216,0),[1670148076.418973216,0, 1670148076.523474648,1),[1670148081.126231453,1, 1670148081.192197464,0),[1670148080.290248203,1, 1670148081.126231453,1),[1670148079.158704797,0, 1670148079.197215931,1),[1670148078.541331850,1, 1670148079.158704797,0),[1670148083.451326843,1, 1670148084.085281752,0),[1670148084.085281752,0, 1670148084.112835815,0),[1670148081.827520045,0, 1670148082.271541228,0),[1670148081.219898625,0, 1670148081.827520045,0),[1670148081.192197464,0, 1670148081.219898625,0),[1670148084.112835815,0, 1670148084.259097903,0),[1670148085.874280727,0, 1670148086.065992466,0),[1670148084.259097903,0, 1670148085.874280727,0),[1670148095.184782199,1, 1670148096.258340878,0),[1670148088.846717770,0, 1670148090.030098093,0),[1670148098.132017989,0, 1670148098.324461750,0),[1670148096.804659908,1, 1670148098.132017989,0),[1670148099.194513886,0, 1670148099.426053819,0),[1670148098.324461750,0, 1670148099.194513886,0),[1670148099.426053819,0, 1670148101.611794866,0),[1670148135.290431596,4, 1670148135.738221414,0),[1670148135.290431596,2, 1670148135.290431596,4),[1670148177.228308860,1, 1670148178.127567673,0),[1670148178.127567673,0, 1670148179.926634797,0),[1670148179.926634797,0, 1670148180.087978431,0),[1670148153.810068778,1, 1670148153.859530513,0),[1670148150.434856856,0, 1670148150.836215114,0),[1670148150.836215114,0, 1670148151.018169300,0),[1670148151.018169300,0, 1670148152.090132339,0),[1670148156.414912563,0, 1670148156.925981841,0),[1670148158.936252170,0, 1670148159.122354602,0),[1670148161.822828677,0, 1670148161.885875607,0),[1670148161.353642065,0, 1670148161.822828677,0),[1670148160.943042194,0, 1670148161.353642065,0),[1670148162.956289507,1, 1670148163.339019412,0),[1670148163.339019412,0, 1670148163.574661854,0),[1670148163.574661854,0, 1670148165.113603299,0),[1670148184.693558406,1, 1670148184.963864371,0),[1670148184.963864371,0, 1670148185.035449251,0),[1670148185.035449251,0, 1670148185.642812258,0),[1670148191.670449125,0, 1670148191.895938468,0),[1670148194.733034699,1, 1670148195.218666450,0),[1670148195.218666450,0, 1670148195.218666450,1),[1670148208.827317110,0, 1670148208.941881657,0),[1670148206.770423291,0, 1670148207.366101773,0),[1670148207.366101773,0, 1670148207.526552897,0),[1670148207.526552897,0, 1670148208.827317110,0),[1670148219.564081886,1, 1670148220.365227769,0),[1670148212.557551106,0, 1670148212.947325130,0),[1670148215.077785886,1, 1670148215.915940721,0),[1670148215.915940721,0, 1670148216.026573215,0),[1670148216.120633697,2, 1670148217.137472893,0),[1670148236.213340603,0, 1670148237.202161836,0),[1670148238.951134663,0, 1670148240.270480960,0),[1670148246.077506399,1, 1670148247.214700401,0),[1670148247.805390317,1, 1670148248.873526156,0),[1670148245.184990881,0, 1670148246.077506399,1),[1670148231.899525698,0, 1670148232.115044601,0),[1670148231.729250558,1, 1670148231.899525698,0),[1670148265.854785507,1, 1670148265.902520866,0),[1670148274.873740372,0, 1670148275.418241003,0),[1670148274.364643506,1, 1670148274.872495264,1),[1670148273.822432626,1, 1670148274.364643506,1),[1670148307.002184828,1, 1670148308.078747401,0),[1670148310.709804968,0, 1670148312.023578464,0),[1670148322.730482970,0, 1670148323.546002332,0),[1670148322.523546626,2, 1670148322.730482970,0),[1670148312.503762176,1, 1670148312.601102664,0),[1670148324.188449116,0, 1670148324.204625021,0),[1670148324.204625021,0, 1670148324.765687448,0),[1670148323.546002332,0, 1670148323.654690418,0),[1670148324.094434017,1, 1670148324.188449116,0),[1670148355.129455254,0, 1670148355.129455254,1),[1670148329.326360881,0, 1670148329.595833726,0),[1670148295.687538091,0, 1670148296.074544915,0),[1670148301.059959363,0, 1670148301.812415664,2),[1670148300.213407134,1, 1670148301.059959363,0)}->[/Table/100/"68667243":v-1011, /Table/100/"639f80fc":v-1008, /Table/100/"61525e01":v-1007, /Table/100/"5f7c6ee6":v-1019, /Table/100/"3a98c5d4":v-1024] [rs]/Table/100/"{28f52639"-d420250d"}:{0:[1670148099.329073824,0, 1670148117.010125492,0), 1:[0,0, 0,0), 2:[1670148099.329073824,0, 1670148102.088706669,1), 3:[1670148099.264749843,0, 1670148119.313480267,1), 4:[1670148097.868893543,0, 1670148108.292556212,1), 5:[1670148097.507007311,0, 1670148108.292556212,1), 6:[1670148097.507007311,0, 1670148108.292556212,1), 7:[1670148099.429375236,1, 1670148108.292556212,1), 8:[1670148099.776146280,0, 1670148108.292556212,1), 9:[1670148099.264749843,0, 1670148108.292556212,1), 10:[1670148099.664517754,0, 1670148120.951531998,0), gap:[<min>, 1670147958.110963382,0),[1670147958.133650657,0, 1670147958.197913885,0),[1670147961.711608261,0, 1670147961.728263678,0),[1670147961.728263678,0, 1670147961.991694692,0),[1670147978.512952311,0, 1670147978.884663308,0),[1670147978.884663308,0, 1670147978.903808293,0),[1670147978.903808293,0, 1670147979.140688903,0),[1670147999.957543489,0, 1670147999.963688624,1),[1670148000.040665548,0, 1670148000.279732373,0),[1670147999.890037204,1, 1670147999.957543489,0),[1670147991.383076089,0, 1670147991.392523301,1),[1670147991.230645641,1, 1670147991.383076089,0),[1670147990.272642515,2, 1670147990.302551836,0),[1670148026.151365776,1, 1670148026.332921792,0),[1670148030.591387021,0, 1670148031.190419173,0),[1670148035.395993545,1, 1670148035.567955022,1),[1670148036.382033410,1, 1670148036.448765002,0),[1670148012.128542874,0, 1670148012.176643107,0),[1670148012.065970824,0, 1670148012.128542874,0),[1670148012.032039306,0, 1670148012.065970824,0),[1670148012.025264941,0, 1670148012.032039306,0),[1670148011.761444359,1, 1670148012.025264941,0),[1670148010.351027555,1, 1670148010.582433939,0),[1670148079.158704797,0, 1670148079.197215931,1),[1670148078.845565980,0, 1670148079.158704797,0),[1670148078.541331850,1, 1670148078.845565980,0),[1670148099.288692240,0, 1670148099.426053819,0),[1670148099.426053819,0, 1670148100.321234822,0),[1670148100.621566531,1, 1670148100.960183552,0),[1670148100.321234822,0, 1670148100.621566531,1),[1670148100.960183552,0, 1670148101.068858522,0),[1670148101.068858522,0, 1670148101.109685338,1),[1670148099.194513886,0, 1670148099.288692240,0),[1670148098.324461750,0, 1670148098.819874811,0),[1670148098.819874811,0, 1670148099.194513886,0),[1670148096.804659908,1, 1670148097.704139080,0),[1670148095.184782199,1, 1670148095.893289081,0),[1670148194.733034699,1, 1670148194.948931925,1),[1670148194.948931925,1, 1670148195.164222288,2),[1670148206.949578611,0, 1670148207.186364350,0),[1670148206.770423291,0, 1670148206.877480326,0),[1670148206.877480326,0, 1670148206.949578611,0),[1670148215.077785886,1, 1670148215.345861707,0),[1670148216.723832860,0, 1670148217.137472893,0),[1670148184.693558406,1, 1670148184.913248764,0),[1670148178.692674105,1, 1670148179.731311433,0),[1670148185.035449251,0, 1670148185.458002443,0),[1670148191.670449125,0, 1670148191.761148417,0),[1670148063.014731697,0, 1670148063.557563124,0),[1670148063.557563124,0, 1670148063.637710337,0),[1670148062.907552427,1, 1670148063.014731697,0),[1670148231.729250558,1, 1670148231.867431344,0),[1670148231.899525698,0, 1670148232.115044601,0),[1670148231.867431344,0, 1670148231.899525698,0),[1670148247.134118827,1, 1670148247.214700401,0),[1670148274.364643506,1, 1670148274.635602321,0),[1670148301.059959363,0, 1670148301.217706497,0),[1670148300.427545067,0, 1670148301.059959363,0)}->[/Table/100/"d2a3a739":v-1017, /Table/100/"ba2aae23":v-1028, /Table/100/"9bda0268":v-1018, /Table/100/"84b61ebd":v-1016, /Table/100/"68667243":v-1011, /Table/100/"639f80fc":v-1008, /Table/100/"61525e01":v-1007, /Table/100/"5f7c6ee6":v-1019, /Table/100/"3a98c5d4":v-1024, /Table/100/"31474658":v-1015, /Table/100/"2a7be558":v-1023] [w]/Table/100/"ba2aae23":1670148100.783251064,1->v-1028 [w]/Table/100/"18bc224d":1670148100.783251064,1->v-1029
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [s]/Table/100/"{099753c1"-bc8b85d5"}:{0:[1670148129.696250648,0, 1670148150.434856856,0), 1:[1670148121.993890172,0, 1670148150.434856856,0), 2:[1670148098.620847103,0, 1670148150.434856856,0), 3:[1670148107.054221221,0, 1670148150.434856856,0), 4:[1670148122.177470525,0, 1670148150.434856856,0), 5:[1670148124.067221284,1, 1670148150.434856856,0), 6:[0,0, 0,0), 7:[1670148129.459082469,0, 1670148150.434856856,0), 8:[1670148141.504496713,0, 1670148150.434856856,0), 9:[1670148131.804644089,1, 1670148150.434856856,0), 10:[1670148141.462751015,0, 1670148150.434856856,0), 11:[1670148140.478345673,0, 1670148148.834386168,1), 12:[1670148141.331315891,0, 1670148148.834386168,1), 13:[1670148136.569424502,0, 1670148148.834386168,1), 14:[1670148130.535720891,0, 1670148148.834386168,1), 15:[1670148131.349442172,0, 1670148148.834386168,1), 16:[1670148131.523144547,0, 1670148148.834386168,1), 17:[1670148138.839061386,0, 1670148148.834386168,1), 18:[1670148135.987903243,0, 1670148148.834386168,1), 19:[1670148136.802119701,0, 1670148148.834386168,1), 20:[1670148125.641265036,0, 1670148148.834386168,1), 21:[1670148127.558719868,1, 1670148148.834386168,1), 22:[1670148141.057032913,0, 1670148150.434856856,0), 23:[1670148139.452654762,0, 1670148150.434856856,0), 24:[1670148141.172031637,1, 1670148150.434856856,0), 25:[1670148144.539778614,0, 1670148150.434856856,0), 26:[1670148141.795496490,0, 1670148150.434856856,0), 27:[1670148136.429123474,0, 1670148150.434856856,0), 28:[1670148136.270122559,0, 1670148150.434856856,0), 29:[1670148141.246797925,0, 1670148150.434856856,0), 30:[1670148143.290409450,0, 1670148150.434856856,0), 31:[1670148143.980665098,0, 1670148153.810068778,1), 32:[1670148144.171893273,1, 1670148153.810068778,1), 33:[1670148144.171893273,0, 1670148153.810068778,1), 34:[1670148142.622369159,1, 1670148152.667173402,0), 35:[1670148141.345381735,0, 1670148152.667173402,0), gap:[<min>, 1670147957.925556555,0),[1670147957.925556555,0, 1670147957.963353896,1),[1670147990.792263627,0, 1670147990.793336891,0),[1670147991.230645641,1, 1670147991.383076089,0),[1670147990.793336891,0, 1670147990.859683643,0),[1670147990.272642515,2, 1670147990.792263627,0),[1670147982.827207112,1, 1670147983.074200793,0),[1670147983.074200793,0, 1670147983.094838233,0),[1670147958.192302121,1, 1670147958.197913885,0),[1670148013.804326024,1, 1670148013.963527758,0),[1670148026.151365776,1, 1670148026.332921792,0),[1670148062.907552427,1, 1670148063.557563124,0),[1670148063.557563124,0, 1670148063.801025104,1),[1670148063.801025104,1, 1670148063.869059372,0),[1670148066.986557005,0, 1670148067.441102525,0),[1670148096.804659908,1, 1670148097.507007311,0),[1670148095.184782199,1, 1670148095.727980945,0),[1670148095.727980945,0, 1670148095.791948901,0),[1670148095.791948901,0, 1670148096.258340878,0),[1670148135.290431596,2, 1670148135.290431596,4),[1670148274.364643506,1, 1670148274.371498757,0),[1670148274.371498757,0, 1670148274.635602321,0),[1670148215.947638558,0, 1670148215.967540309,0),[1670148216.723832860,0, 1670148217.137472893,0),[1670148215.754176305,0, 1670148215.915940721,0),[1670148215.915940721,0, 1670148215.947638558,0),[1670148215.077785886,1, 1670148215.754176305,0),[1670148143.793998781,2, 1670148143.965489366,0),[1670148143.343882896,0, 1670148143.793998781,2),[1670148144.168192674,1, 1670148144.586465065,0),[1670148143.965489366,0, 1670148144.168192674,1),[1670148145.280895877,1, 1670148146.244061970,0),[1670148144.586465065,0, 1670148145.280895877,1),[1670148146.244061970,0, 1670148146.996612452,0),[1670148141.172031637,1, 1670148141.246797925,0),[1670148140.679780765,1, 1670148141.172031637,1),[1670148141.861725610,0, 1670148142.423965130,0),[1670148141.246797925,0, 1670148141.861725610,0),[1670148142.423965130,0, 1670148142.795373014,0),[1670148142.896832367,0, 1670148143.343882896,0),[1670148142.795373014,0, 1670148142.896832367,0)}->[/Table/100/"0a888093":v-1178, /Table/100/"0ba8e671":v-1142, /Table/100/"0c7db9b3":v-1014, /Table/100/"0cec8644":v-1059, /Table/100/"1045922c":v-1145, /Table/100/"129872ae":v-1157, /Table/100/"12aff170":v-1247, /Table/100/"155d2b9b":v-1174, /Table/100/"15efe0af":v-1234, /Table/100/"1631728c":v-1197, /Table/100/"169c772e":v-1233, /Table/100/"1a12007c":v-1222, /Table/100/"1ad4bc36":v-1231, /Table/100/"1b8d4209":v-1215, /Table/100/"1c11b7ca":v-1186, /Table/100/"1c30d20a":v-1191, /Table/100/"1f3ee109":v-1194, /Table/100/"27c4be9f":v-1224, /Table/100/"2ac39883":v-1208, /Table/100/"2ba1f50c":v-1217, /Table/100/"2c3ab9b7":v-1163, /Table/100/"2d59df62":v-1168, /Table/100/"34ba0784":v-1226, /Table/100/"39d80113":v-1221, /Table/100/"3ce38789":v-1228, /Table/100/"42c96653":v-1242, /Table/100/"4738c10a":v-1236, /Table/100/"523f7dac":v-1214, /Table/100/"5546433f":v-1212, /Table/100/"60a38245":v-1230, /Table/100/"62f0bb59":v-1238, /Table/100/"89c2f40b":v-1239, /Table/100/"9311df65":v-1229, /Table/100/"95630fc2":v-1240, /Table/100/"b95b006f":v-1237, /Table/100/"bc5dcf3d":v-1232] [w]/Table/100/"a9ec5c23":1670148146.325307961,1->v-1246 [w]/Table/100/"12aff170":1670148146.325307961,1->v-1247
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [s]/Table/100/"{099753c1"-bc8b85d5"}:{0:[1670148129.696250648,0, 1670148150.434856856,0), 1:[1670148121.993890172,0, 1670148150.434856856,0), 2:[1670148098.620847103,0, 1670148150.434856856,0), 3:[1670148107.054221221,0, 1670148150.434856856,0), 4:[1670148122.177470525,0, 1670148150.434856856,0), 5:[1670148124.067221284,1, 1670148150.434856856,0), 6:[0,0, 0,0), 7:[1670148129.459082469,0, 1670148150.434856856,0), 8:[1670148141.504496713,0, 1670148150.434856856,0), 9:[1670148131.804644089,1, 1670148150.434856856,0), 10:[1670148141.462751015,0, 1670148150.434856856,0), 11:[1670148140.478345673,0, 1670148148.834386168,1), 12:[1670148141.331315891,0, 1670148148.834386168,1), 13:[1670148136.569424502,0, 1670148148.834386168,1), 14:[1670148130.535720891,0, 1670148148.834386168,1), 15:[1670148131.349442172,0, 1670148148.834386168,1), 16:[1670148131.523144547,0, 1670148148.834386168,1), 17:[1670148138.839061386,0, 1670148148.834386168,1), 18:[1670148135.987903243,0, 1670148148.834386168,1), 19:[1670148136.802119701,0, 1670148148.834386168,1), 20:[1670148125.641265036,0, 1670148148.834386168,1), 21:[1670148127.558719868,1, 1670148148.834386168,1), 22:[1670148141.057032913,0, 1670148150.434856856,0), 23:[1670148139.452654762,0, 1670148150.434856856,0), 24:[1670148141.172031637,1, 1670148150.434856856,0), 25:[1670148144.539778614,0, 1670148150.434856856,0), 26:[1670148141.795496490,0, 1670148150.434856856,0), 27:[1670148136.429123474,0, 1670148150.434856856,0), 28:[1670148136.270122559,0, 1670148150.434856856,0), 29:[1670148141.246797925,0, 1670148150.434856856,0), 30:[1670148143.290409450,0, 1670148150.434856856,0), 31:[1670148143.980665098,0, 1670148153.810068778,1), 32:[1670148144.171893273,1, 1670148153.810068778,1), 33:[1670148144.171893273,0, 1670148153.810068778,1), 34:[1670148142.622369159,1, 1670148152.667173402,0), 35:[1670148141.345381735,0, 1670148152.667173402,0), gap:[<min>, 1670147957.925556555,0),[1670147957.925556555,0, 1670147957.963353896,1),[1670147990.792263627,0, 1670147990.793336891,0),[1670147991.230645641,1, 1670147991.383076089,0),[1670147990.793336891,0, 1670147990.859683643,0),[1670147990.272642515,2, 1670147990.792263627,0),[1670147982.827207112,1, 1670147983.074200793,0),[1670147983.074200793,0, 1670147983.094838233,0),[1670147958.192302121,1, 1670147958.197913885,0),[1670148013.804326024,1, 1670148013.963527758,0),[1670148026.151365776,1, 1670148026.332921792,0),[1670148062.907552427,1, 1670148063.557563124,0),[1670148063.557563124,0, 1670148063.801025104,1),[1670148063.801025104,1, 1670148063.869059372,0),[1670148066.986557005,0, 1670148067.441102525,0),[1670148096.804659908,1, 1670148097.507007311,0),[1670148095.184782199,1, 1670148095.727980945,0),[1670148095.727980945,0, 1670148095.791948901,0),[1670148095.791948901,0, 1670148096.258340878,0),[1670148135.290431596,2, 1670148135.290431596,4),[1670148274.364643506,1, 1670148274.371498757,0),[1670148274.371498757,0, 1670148274.635602321,0),[1670148215.947638558,0, 1670148215.967540309,0),[1670148216.723832860,0, 1670148217.137472893,0),[1670148215.754176305,0, 1670148215.915940721,0),[1670148215.915940721,0, 1670148215.947638558,0),[1670148215.077785886,1, 1670148215.754176305,0),[1670148143.793998781,2, 1670148143.965489366,0),[1670148143.343882896,0, 1670148143.793998781,2),[1670148144.168192674,1, 1670148144.586465065,0),[1670148143.965489366,0, 1670148144.168192674,1),[1670148145.280895877,1, 1670148146.244061970,0),[1670148144.586465065,0, 1670148145.280895877,1),[1670148146.244061970,0, 1670148146.996612452,0),[1670148141.172031637,1, 1670148141.246797925,0),[1670148140.679780765,1, 1670148141.172031637,1),[1670148141.861725610,0, 1670148142.423965130,0),[1670148141.246797925,0, 1670148141.861725610,0),[1670148142.423965130,0, 1670148142.795373014,0),[1670148142.896832367,0, 1670148143.343882896,0),[1670148142.795373014,0, 1670148142.896832367,0)}->[/Table/100/"0a888093":v-1178, /Table/100/"0ba8e671":v-1142, /Table/100/"0c7db9b3":v-1014, /Table/100/"0cec8644":v-1059, /Table/100/"1045922c":v-1145, /Table/100/"129872ae":v-1157, /Table/100/"12aff170":v-1247, /Table/100/"155d2b9b":v-1174, /Table/100/"15efe0af":v-1234, /Table/100/"1631728c":v-1197, /Table/100/"169c772e":v-1233, /Table/100/"1a12007c":v-1222, /Table/100/"1ad4bc36":v-1231, /Table/100/"1b8d4209":v-1215, /Table/100/"1c11b7ca":v-1186, /Table/100/"1c30d20a":v-1191, /Table/100/"1f3ee109":v-1194, /Table/100/"27c4be9f":v-1224, /Table/100/"2ac39883":v-1208, /Table/100/"2ba1f50c":v-1217, /Table/100/"2c3ab9b7":v-1163, /Table/100/"2d59df62":v-1168, /Table/100/"34ba0784":v-1226, /Table/100/"39d80113":v-1221, /Table/100/"3ce38789":v-1228, /Table/100/"42c96653":v-1242, /Table/100/"4738c10a":v-1236, /Table/100/"523f7dac":v-1214, /Table/100/"5546433f":v-1212, /Table/100/"60a38245":v-1230, /Table/100/"62f0bb59":v-1238, /Table/100/"89c2f40b":v-1239, /Table/100/"9311df65":v-1229, /Table/100/"95630fc2":v-1240, /Table/100/"b95b006f":v-1237, /Table/100/"bc5dcf3d":v-1232] [w]/Table/100/"a9ec5c23":1670148146.325307961,1->v-1246 [w]/Table/100/"12aff170":1670148146.325307961,1->v-1247
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2653695887
--- FAIL: TestKVNemesisSingleNode (466.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

kv/kvnemesis.TestKVNemesisSingleNode [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7854044?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7854044?buildTab=artifacts#/) on master @ [146556e19f5e4fdc8c3e6a623b280cc33aee4d18](https://github.com/cockroachdb/cockroach/commits/146556e19f5e4fdc8c3e6a623b280cc33aee4d18):


```
=== RUN   TestKVNemesisSingleNode
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2881882535
    test_log_scope.go:79: use -show-logs to present logs inline
    kvnemesis_test.go:68: failure:
        committed txn non-atomic timestamps: [r]/Table/100/"321981ca":[<min>, <max>)-><nil> [rs]/Table/100/"{b953df98"-f7793a2d"}:{0:[0,0, 0,0), 1:[1670315463.342700545,0, 1670315466.025414449,1), 2:[1670315464.509236674,0, 1670315466.025414449,1), 3:[1670315464.547097910,0, 1670315466.025414449,1), 4:[1670315465.537219343,0, 1670315466.025414449,1), 5:[1670315464.360178831,1, 1670315466.025414449,1), gap:[<min>, 1670315462.758808576,0),[1670315462.788680168,1, 1670315462.853945445,0),[1670315462.853945445,0, 1670315462.902255789,0),[1670315462.902255789,0, 1670315463.070755585,0),[1670315463.485034668,0, 1670315463.979547476,0),[1670315463.310980153,0, 1670315463.346033668,0),[1670315463.979547476,0, 1670315464.311818377,0),[1670315464.317446872,0, 1670315465.182210096,0),[1670315465.182210096,0, 1670315465.923635268,0),[1670315466.025414449,1, 1670315466.212987521,0),[1670315466.212987521,0, 1670315466.221990663,1),[1670315475.660139875,0, 1670315475.761888046,0),[1670315475.761888046,0, 1670315475.792160006,0),[1670315475.792160006,0, 1670315477.065802722,0),[1670315475.630479017,0, 1670315475.658524954,0),[1670315475.614990208,1, 1670315475.630479017,0),[1670315475.658524954,0, 1670315475.660139875,0),[1670315533.519881941,1, 1670315534.277714656,0),[1670315530.016563858,0, 1670315530.020067341,0),[1670315497.778085634,1, 1670315498.006555393,0),[1670315500.782607213,1, 1670315500.856994727,2),[1670315500.856994727,2, 1670315500.938661883,0),[1670315511.685317629,1, 1670315511.691613353,0),[1670315510.686734066,1, 1670315510.798116280,0),[1670315510.798116280,0, 1670315510.800503116,0),[1670315510.633260797,0, 1670315510.686734066,1),[1670315510.557514915,1, 1670315510.633260797,0),[1670315510.376482043,0, 1670315510.498372953,0),[1670315510.498372953,0, 1670315510.557514915,1),[1670315506.914024735,0, 1670315507.019281001,0),[1670315529.807612749,0, 1670315530.016563858,0),[1670315529.568400645,0, 1670315529.807612749,0),[1670315522.269789292,1, 1670315522.452235590,0),[1670315600.637637229,0, 1670315600.658187401,0),[1670315615.440306499,0, 1670315615.983959697,0),[1670315615.206431470,1, 1670315615.440306499,0),[1670315610.077228415,0, 1670315610.313223809,1),[1670315609.995723010,0, 1670315610.077228415,0),[1670315610.313223809,1, 1670315610.384185555,0),[1670315610.384185555,0, 1670315610.409417066,0),[1670315563.805135262,1, 1670315563.888302323,0),[1670315563.563749956,1, 1670315563.805135262,1),[1670315563.888302323,0, 1670315564.391733719,0),[1670315617.003616515,1, <max>),[1670315616.226447504,1, 1670315616.327920352,0),[1670315615.983959697,0, 1670315616.008535238,0)}->[/Table/100/"f3cd7c41":v-86, /Table/100/"ed90c728":v-53, /Table/100/"e47e5ac3":v-76, /Table/100/"df4acc12":v-79, /Table/100/"da97bcbf":v-83, /Table/100/"c3617c8f":v-66] [w]/Table/100/"80c7a199":1670315465.537219343,1->v-85 [w]/Table/100/"f3cd7c41":1670315465.537219343,1->v-86
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomicCommitted
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:845
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).checkAtomic
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:600
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:569
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/validator.go:72
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisSingleNode
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/pkg/kv/kvnemesis/kvnemesis_test.go:64
          | testing.tRunner
          | 	GOROOT/src/testing/testing.go:1446
          | runtime.goexit
          | 	GOROOT/src/runtime/asm_amd64.s:1594
        Wraps: (2) committed txn non-atomic timestamps: [r]/Table/100/"321981ca":[<min>, <max>)-><nil> [rs]/Table/100/"{b953df98"-f7793a2d"}:{0:[0,0, 0,0), 1:[1670315463.342700545,0, 1670315466.025414449,1), 2:[1670315464.509236674,0, 1670315466.025414449,1), 3:[1670315464.547097910,0, 1670315466.025414449,1), 4:[1670315465.537219343,0, 1670315466.025414449,1), 5:[1670315464.360178831,1, 1670315466.025414449,1), gap:[<min>, 1670315462.758808576,0),[1670315462.788680168,1, 1670315462.853945445,0),[1670315462.853945445,0, 1670315462.902255789,0),[1670315462.902255789,0, 1670315463.070755585,0),[1670315463.485034668,0, 1670315463.979547476,0),[1670315463.310980153,0, 1670315463.346033668,0),[1670315463.979547476,0, 1670315464.311818377,0),[1670315464.317446872,0, 1670315465.182210096,0),[1670315465.182210096,0, 1670315465.923635268,0),[1670315466.025414449,1, 1670315466.212987521,0),[1670315466.212987521,0, 1670315466.221990663,1),[1670315475.660139875,0, 1670315475.761888046,0),[1670315475.761888046,0, 1670315475.792160006,0),[1670315475.792160006,0, 1670315477.065802722,0),[1670315475.630479017,0, 1670315475.658524954,0),[1670315475.614990208,1, 1670315475.630479017,0),[1670315475.658524954,0, 1670315475.660139875,0),[1670315533.519881941,1, 1670315534.277714656,0),[1670315530.016563858,0, 1670315530.020067341,0),[1670315497.778085634,1, 1670315498.006555393,0),[1670315500.782607213,1, 1670315500.856994727,2),[1670315500.856994727,2, 1670315500.938661883,0),[1670315511.685317629,1, 1670315511.691613353,0),[1670315510.686734066,1, 1670315510.798116280,0),[1670315510.798116280,0, 1670315510.800503116,0),[1670315510.633260797,0, 1670315510.686734066,1),[1670315510.557514915,1, 1670315510.633260797,0),[1670315510.376482043,0, 1670315510.498372953,0),[1670315510.498372953,0, 1670315510.557514915,1),[1670315506.914024735,0, 1670315507.019281001,0),[1670315529.807612749,0, 1670315530.016563858,0),[1670315529.568400645,0, 1670315529.807612749,0),[1670315522.269789292,1, 1670315522.452235590,0),[1670315600.637637229,0, 1670315600.658187401,0),[1670315615.440306499,0, 1670315615.983959697,0),[1670315615.206431470,1, 1670315615.440306499,0),[1670315610.077228415,0, 1670315610.313223809,1),[1670315609.995723010,0, 1670315610.077228415,0),[1670315610.313223809,1, 1670315610.384185555,0),[1670315610.384185555,0, 1670315610.409417066,0),[1670315563.805135262,1, 1670315563.888302323,0),[1670315563.563749956,1, 1670315563.805135262,1),[1670315563.888302323,0, 1670315564.391733719,0),[1670315617.003616515,1, <max>),[1670315616.226447504,1, 1670315616.327920352,0),[1670315615.983959697,0, 1670315616.008535238,0)}->[/Table/100/"f3cd7c41":v-86, /Table/100/"ed90c728":v-53, /Table/100/"e47e5ac3":v-76, /Table/100/"df4acc12":v-79, /Table/100/"da97bcbf":v-83, /Table/100/"c3617c8f":v-66] [w]/Table/100/"80c7a199":1670315465.537219343,1->v-85 [w]/Table/100/"f3cd7c41":1670315465.537219343,1->v-86
        Error types: (1) *withstack.withStack (2) *errutil.leafError
    kvnemesis_test.go:70: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1f42cf5be2fc021646bf9b2daf5eaef3/logTestKVNemesisSingleNode2881882535
--- FAIL: TestKVNemesisSingleNode (207.23s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestKVNemesisSingleNode.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvclient/kvcoord/txn_test.go
+++ b/pkg/kv/kvclient/kvcoord/txn_test.go
@@ -979,3 +979,55 @@ func TestRetrySerializableBumpsToNow(t *testing.T) {
 	}))
 	require.Greater(t, attempt, 1, "Transaction is expected to retry once")
 }
+
+// TestTxnRetryWithLatchesDroppedEarly serves as a regression test for
+// https://github.com/cockroachdb/cockroach/issues/92189. It constructs a batch
+// like:
+// b.Scan(a, e)
+// b.Put(b, "value2")
+// which is forced to retry at a higher timestamp. It ensures that the scan
+// request does not see the intent at key b, even when the retry happens.
+func TestTxnRetryWithLatchesDroppedEarly(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	s := createTestDB(t)
+	defer s.Stop()
+
+	keyA := "a"
+	keyB := "b"
+	keyE := "e"
+	keyF := "f"
+
+	err := s.DB.Txn(context.Background(), func(ctx context.Context, txn *kv.Txn) error {
+		s.Manual.Advance(1 * time.Second)
+
+		{
+			// Attempt to write to keyF in another txn.
+			conflictTxn := kv.NewTxn(ctx, s.DB, 0 /* gatewayNodeID */)
+			conflictTxn.TestingSetPriority(enginepb.MaxTxnPriority)
+			if err := conflictTxn.Put(ctx, keyF, "valueF"); err != nil {
+				return err
+			}
+			if err := conflictTxn.Commit(ctx); err != nil {
+				return err
+			}
+		}
+
+		b := txn.NewBatch()
+		b.Scan(keyA, keyE)
+		b.Put(keyB, "value2")
+		b.Put(keyF, "value3") // bumps the transaction and causes a server side retry.
+
+		err := txn.Run(ctx, b)
+		if err != nil {
+			return err
+		}
+
+		// Ensure no rows were returned as part of the scan.
+		require.Equal(t, 0, len(b.RawResponse().Responses[0].GetInner().(*roachpb.ScanResponse).Rows))
+		return nil
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+}

--- a/pkg/kv/kvnemesis/kvnemesis_test.go
+++ b/pkg/kv/kvnemesis/kvnemesis_test.go
@@ -181,10 +181,6 @@ func testKVNemesisImpl(t *testing.T, cfg kvnemesisTestCfg) {
 		sqlDBs[i] = tc.ServerConn(i)
 	}
 	sqlutils.MakeSQLRunner(sqlDBs[0]).Exec(t, `SET CLUSTER SETTING kv.rangefeed.enabled = true`)
-	// TODO(arul): remove this line when #92189 is addressed.
-	//
-	// See: https://github.com/cockroachdb/cockroach/issues/93164.
-	sqlutils.MakeSQLRunner(sqlDBs[0]).Exec(t, `SET CLUSTER SETTING kv.transaction.dropping_latches_before_eval.enabled = false`)
 	// Turn net/trace on, which results in real trace spans created throughout.
 	// This gives kvnemesis a chance to hit NPEs related to tracing.
 	sqlutils.MakeSQLRunner(sqlDBs[0]).Exec(t, `SET CLUSTER SETTING trace.debug.enable = true`)

--- a/pkg/kv/kvserver/replica_read.go
+++ b/pkg/kv/kvserver/replica_read.go
@@ -31,6 +31,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/mon"
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/cockroachdb/cockroach/pkg/util/uuid"
 	"github.com/cockroachdb/errors"
 	"github.com/kr/pretty"
 )
@@ -300,9 +301,14 @@ func (r *Replica) canDropLatchesBeforeEval(
 	// Check if any of the requests within the batch need to resolve any intents
 	// or if any of them need to use an intent interleaving iterator.
 	for _, req := range ba.Requests {
-		start, end := req.GetInner().Header().Key, req.GetInner().Header().EndKey
-		needsIntentInterleavingForThisRequest, err := storage.ScanConflictingIntents(
-			ctx, rw, ba.Txn, ba.Header.Timestamp, start, end, &intents, maxIntents,
+		reqHeader := req.GetInner().Header()
+		start, end, seq := reqHeader.Key, reqHeader.EndKey, reqHeader.Sequence
+		var txnID uuid.UUID
+		if ba.Txn != nil {
+			txnID = ba.Txn.ID
+		}
+		needsIntentInterleavingForThisRequest, err := storage.ScanConflictingIntentsForDroppingLatchesEarly(
+			ctx, rw, txnID, ba.Header.Timestamp, start, end, seq, &intents, maxIntents,
 		)
 		if err != nil {
 			return false /* ok */, true /* stillNeedsIntentInterleaving */, roachpb.NewError(

--- a/pkg/storage/engine.go
+++ b/pkg/storage/engine.go
@@ -1677,21 +1677,31 @@ func assertMVCCIteratorInvariants(iter MVCCIterator) error {
 	return nil
 }
 
-// ScanConflictingIntents scans intents using only the separated intents lock
-// table. The result set is added to the given `intents` slice. It ignores
-// intents that do not conflict with `txn`. If it encounters intents that were
-// written by `txn` that are either at a higher sequence number than txn's or at
-// a lower sequence number but at a higher timestamp, `needIntentHistory` is set
-// to true. This flag is used to signal to the caller that a subsequent scan
-// over the MVCC key space (for the batch in question) will need to be performed
-// using an intent interleaving iterator in order to be able to read the correct
-// provisional value.
-func ScanConflictingIntents(
+// ScanConflictingIntentsForDroppingLatchesEarly scans intents using only the
+// separated intents lock table on behalf of a batch request trying to drop its
+// latches early. If found, conflicting intents are added to the supplied
+// `intents` slice, which indicates to the caller that evaluation should not
+// proceed until the intents are resolved. Intents that don't conflict with the
+// transaction referenced by txnID[1] at the supplied `ts` are ignored.
+//
+// The caller must supply the sequence number of the request on behalf of which
+// the intents are being scanned. This is used to determine if the caller needs
+// to consult intent history when performing a scan over the MVCC keyspace
+// (indicated by the `needIntentHistory` return parameter). Intent history is
+// required to read the correct provisional value when scanning if we encounter
+// an intent written by the `txn` at a higher sequence number than the one
+// supplied or at a higher timestamp than the `ts` supplied (regardless of the
+// sequence number of the intent).
+//
+// [1] The supplied txnID may be empty (uuid.Nil) if the request on behalf of
+// which the scan is being performed is non-transactional.
+func ScanConflictingIntentsForDroppingLatchesEarly(
 	ctx context.Context,
 	reader Reader,
-	txn *roachpb.Transaction,
+	txnID uuid.UUID,
 	ts hlc.Timestamp,
 	start, end roachpb.Key,
+	seq enginepb.TxnSeq,
 	intents *[]roachpb.Intent,
 	maxIntents int64,
 ) (needIntentHistory bool, err error) {
@@ -1718,7 +1728,9 @@ func ScanConflictingIntents(
 	var ok bool
 	for ok, err = iter.SeekEngineKeyGE(EngineKey{Key: ltStart}); ok; ok, err = iter.NextEngineKey() {
 		if maxIntents != 0 && int64(len(*intents)) >= maxIntents {
-			break
+			// Return early if we're done accumulating intents; make no claims about
+			// not needing intent history.
+			return true /* needsIntentHistory */, nil
 		}
 		if err = protoutil.Unmarshal(iter.UnsafeValue(), &meta); err != nil {
 			return false, err
@@ -1726,14 +1738,14 @@ func ScanConflictingIntents(
 		if meta.Txn == nil {
 			return false, errors.Errorf("intent without transaction")
 		}
-		ownIntent := txn != nil && txn.ID == meta.Txn.ID
+		ownIntent := txnID != uuid.Nil && txnID == meta.Txn.ID
 		if ownIntent {
 			// If we ran into one of our own intents, check whether the intent has a
 			// higher (or equal) sequence number or a higher (or equal) timestamp. If
 			// either of these conditions is true, a corresponding scan over the MVCC
 			// key space will need access to the key's intent history in order to read
 			// the correct provisional value. So we set `needIntentHistory` to true.
-			if txn.Sequence <= meta.Txn.Sequence || ts.LessEq(meta.Timestamp.ToTimestamp()) {
+			if seq <= meta.Txn.Sequence || ts.LessEq(meta.Timestamp.ToTimestamp()) {
 				needIntentHistory = true
 			}
 			continue

